---
output: 
    thesisdown::thesis_pdf: default
bibliography: bib/thesis.bib
csl: csl/nature.csl
space_betwee_paragraphs: true
fig_caption: true
always_allow_html: yes
link-citations: true
toc-depth: 2
lot: true
lof: true
indent: true
header-includes: # include other LaTeX packages here
    \usepackage{booktabs}
    \usepackage{longtable}
    \usepackage{siunitx}
    \usepackage[left]{lineno}
    \usepackage{float}
    \floatplacement{figure}{H}
    \linenumbers
    \pagestyle{plain}
    \raggedbottom 
    \usepackage{indentfirst}
editor_options: 
  chunk_output_type: console
---

```{r include=FALSE}
# word count = 
doc.type <- knitr::opts_knit$get('rmarkdown.pandoc.to') # this will tell the .Rmd file what output you are knitting to (word, pdf, html) so that you cna use if else statements when making tables - html/pdf output tables do not work well in word. For word we need to use the 'flextable' packaged to make tables.
## packages
library(dplyr)
library(kableExtra)
library(flextable)
options(tinytex.verbose = TRUE)
```

# Associations between multiple measures of adiposity and metabolites: Mendelian randomization analysis {#MR} 

<style>
body {
text-align: justify}
</style>

## Chapter summary {-}
Chapter \@ref(observational) explored the association between multiple measures of adiposity (body mass index (BMI), waist hip ratio (WHR), body fat percent (BF)) and up to 230 metabolites in the Avon Longitudinal Study of Parents and Children (ALSPAC). Across four time points (children (~7 years), adolescents (~15 years), young adults (~24 years), adults (~50 years)) three linear models, adjusting for age and sex (models 1, 2, 3), education, smoking, alcohol, and diet (models 2, 3) and physical activity (model 3), found a large number of associations across metabolite subclasses. Poor measurement, residual confounding, reverse causation, and missing data are all limitations of observational analyses which could bias estimates. Perhaps most pertinent to analysis in Chapter \@ref(observational) is residual confounding and reverse causation. As discussed in \@ref(introduction-summary) and \@ref(mendelian-randomization), methods which try to account for these issues, such as Mendelian randomization (MR), provide an opportunity to further interrogate relationships. MR analyses hold a different set of assumptions and limitations; triangulation of evidence across multiple study designs with different sources of bias can strengthen confidence in results[@Lawlor2016]. In this chapter, MR is employed to investigate the association between multiple measures of adiposity (BMI, WHR, BF) and metabolites. Consistent results between observational and Mendelian randomization analyses strengthens evidence of association between adiposity and metabolites and may help to better characterize the changing metabolic profile as a result of adiposity which may later inform disease analysis. \par

\newpage

## Introduction {#MR-intro}
Evidence from observational analyses, including Chapter \@ref(observational), highlights the broad effect of adiposity on metabolites[@Wurtz2014; @Moore2014; @SantosFerreira2017; @Cirulli2019; @Lau2020; @Brachem2020; @Stevens2020; @OKeeffe2020; @Wulaningsih2019; @Frigerio2021; @Neeland2019; @Bachlechner2016; @Rangel-Huerta2019]. These effects are consistent across overall body composition and site-specific adiposity, and persist over time, and after adjustment for covariables. However, observational analyses are subject to a number of limitations including measurement error and unmeasured confounding (Section \@ref(mendelian-randomization)). These limitations could have resulted in biased estimates of the effect of adiposity on metabolites XXX. \par

Mendelian randomization (MR), as discussed in Section \@ref(mendelian-randomization) is able to reduce the issues traditionally observed in observational analyses (namely confounding and reverse causation)[@DaveySmith2003]. MR has previously been used to investigate the causal effect between adiposity and metabolic profile[@Wurtz2014; @Bull2020; @Bell2021a]. These studies used body mass index (BMI)[@Wurtz2014] alongside waist hip ratio (WHR)[@Bull2020; @Bell2021a] with 82[@Wurtz2014], 123[@Bull2020], and 249[@Bell2021a] metabolic measures in up to 12,664[@Wurtz2014], 24,925[@Bull2020], and 109,532[@Bell2021a] individuals. They show similar evidence of association to observational analyses with systemic changes across the metabolome. However, these studies either used sole measures of adiposity or did not include a measure that accurately capture the total body fat content of the body, such ads body fat percentage (BF). The studies are therefore unable to comment on the similarities or differences between adiposity measures. \par

Here, a parallel MR analysis using multiple measures of adiposity (BMI, WHR, BF) with 123 metabolites from Kettunen et al. (2016)[@Kettunen2016] (N = 24,925) and 230 metabolites from the INTERVAL trial[@Moore2014b] (Efficiency and safety of varying the frequency of whole blood donation; unpublished; N = 40,849) is performed. For all metabolites that were shared between these two studies, follow-up meta-analyses were conducted in up to 65,774 individuals of European ancestries. Triangulation of these results with observational analyses in adults (Chapter \@ref(observational)), were used to identify persistent effects across studies and methods to take into analyses presented later in this thesis. Metabolites investigated here include directly measured metabolites such as the amino acids tyrosine and phenylalanine, as well as derived (not-directly measured) metabolite measures such as the ratio of saturated fatty acids to total fatty acid. \par

## Methods {#MR-methods}
This chapter details hypothesis-free, two-sample summary-level MR analyses and subsequent meta-analyses (Figure \@ref(fig:MR-figure-analysis-flow)). Genetic variants, single nucleotide polymorphisms (SNPs), associated with variation in three measures of adiposity (BMI[@Yengo2018], WHR[@Pulit2019], and BF[@Lu2016]) measured in individuals of European ancestries were used as genetic instrumental variables for the three adiposity exposures. Each of the exposures (BMI, WHR, and BF) capture different components of adiposity, and the associated genetic variants explain different amounts of variation in the respective trait. \par

A parallel MR analysis was performed on the following outcomes: (1) 123 nuclear magnetic resonance (NMR) derived metabolites measured in 24,925 individuals of European ancestries from Kettunen et al. (2016)[@Kettunen2016]; (2) 230 NMR derived metabolites from 40,849 individuals of European ancestries from INTERVAL (unpublished). The main MR analysis consisted of an inverse variance weighted multiplicative random effects (IVW-MRE) model and sensitivity analyses using additional models (MR-Egger, weighted median, and weighted mode). Meta-analyses of all metabolites shared between the Kettunen and INTERVAL studies was also performed. Meta-analyses included up to 65,774 individuals of European ancestries. These datasets, including the adiposity data detailed later on, were inverse rank normally transformed prior to genome-wide analysis. Traditionally, and throughout this Chapter, units from these GWAS are given as standard deviations (SD). However, due to this transformation the data are in essence dimensionless and have no units[@McCaw2020]. The benefit of this transformation is that estimates from these GWAS can be easily compared across other GWAS using the same transformation. \par

All data manipulation and analyses were performed using `R`[@r2019] (version 3.5.3). MR analysis was performed using the `TwoSampleMR`[@Hemani2018] (version 0.4.22) `R` package. Data for exposures were obtained from published summary statistics (Table \@ref(tab:appendix-MR-table-exposures)). Summary statistics for the Kettunen et al. (2016)[@Kettunen2016]) metabolites were obtained from MR Base[@Hemani2018] (accessed 26/07/2019), while those for INTERVAL were unpublished and obtained from collaborators (Adam Butterworth, University of Cambridge). A list of all metabolites is available in the Appendix (Table \@ref(tab:appendix-MR-table-metabolites)) and on [GitHub](https://github.com/mattlee821/000_thesis/blob/master/index/data/MR/tables/metabolites.txt). A list of all genetic variants used as instrumental variables are available on [GitHub](https://github.com/mattlee821/000_thesis/blob/master/index/data/MR/tables/exposure_SNPs.txt). Meta analyses were conducted using the `meta` (version 4.18-0) `R` package. Results were visualised using the `EpiViz` (version 0.0.0.9; detailed in Chapter \@ref(visualisation)) and `ggforestlot` (version 0.1.0) `R` packages. \par

(ref:MR-figure-overview-cap) **Analysis overview: Effect of adiposity on metabolites**. The main analysis consisted of two parallel two-sample Mendelian randomization (MR) analyses using three exposures (BMI (body mass index), WHR (waist hip ratio), and BF (body fat percentage)). Two outcome metabolomics data sets were used: (1) 123 nuclear magnetic resonance (NMR) derived metabolites measured in 24,925 individuals of European ancestries from Kettunen et al. (2016)[@Kettunen2016]; (2) 230 NMR derived metabolites from 40,849 individuals of European ancestries from INTERVAL[@Lotta2021]. Four models were used: inverse variance weighted, multiplicative random effects (IVW-MRE) model was the main analysis; MR-Egger, weighted median, and weighted mode were used as sensitivity analyses. The number of SNPs used for each exposure along with the N for the exposure and outcome data sets are given. A meta-analysis of the IVW-MRE results of 110 metabolites that were shared across the two studies was conducted.

(ref:MR-figure-overview-scap) Analysis overview: Effect of adiposity on metabolites

```{r MR-figure-analysis-flow, echo=FALSE, warning=FALSE, error=FALSE, out.width='100%',fig.cap='(ref:MR-figure-overview-cap)', fig.scap='(ref:MR-figure-overview-scap)'}
knitr::include_graphics("../index/data/MR/figures/analysis_flow.pdf")
```

### Instrumentation
Instrumentation of exposures is primarily achieved using either a single genetic instrumental variable or multiple genetic instrumental variables. A limitation of using a single genetic variant to instrument an exposure is that the proportion of variance explained by that variant on the exposure will likely be low (although some cases exist where cis-variants near or in the protein coding region of the gene lead to changes in protein levels, such as CRP). This will result in a weak instrument which can lead to biased estimates towards the confounded observational effect[@Davies2018]. Using multiple genetic instruments which, collectively, explain a greater proportion of the trait variance than any one individual variant can mitigate weak instrument bias. \par

In Chapter \@ref(systematic-review), 173 studies which used a measure of adiposity as an exposure in 2,214 MR analyses were reviewed. The majority of these 2,214 MR analyses used BMI (N = 1,509) as the exposure; 112 analyses used WHR and 45 used BF, the remaining analyses used differing measures of fat mass and depositions such as hepatic fat. The majority of analyses used multiple instruments. In many instances, it was not possible to identify the instruments used due to absence of information or miss-reporting. On the whole however, the main instrumentation strategy was to obtain instruments from the most recent GWAS with the largest sample size. This resulted in most analyses using a genome-wide significance threshold of 5 x 10^-8^. Few studies discussed the independence of their genetic variants, and those that did mostly reported that the original GWAS declared the SNPs to be independent from one another - very few analyses performed clumping of their obtained instruments. In order to allow comparison between results here and those previously reported, the instrumentation approach used most commonly in previous analyses was used here. For the main analysis here, instruments were obtained directly from the most recently published GWAS with the largest sample size. Additional analyses used instruments obtained from GWAS which did not use UK Biobank, due to potential concerns over population structure[@Haworth2019; @Lawson2020; @Sarmanova2020; @Sanderson2021], and performed clumping of identified instruments to ensure independence of instruments in the lists. \par

The majority of MR analyses identified in Chapter \@ref(SR) investigated the effects of adiposity on health outcomes using sex combined exposure and outcome data. Where the outcome was sex specific, the majority of analyses used sex specific exposure data. In the analyses detailed here in, exposure and outcome data were sex combined. However, this does not necessarily mean that results can not be used to inform subsequent analyses which may involve sex specific outcomes. Many studies identified in Chapter \@ref(SR) used sex combined exposure data to investigate sex specific outcomes such as ovarian [@Dixon2016a], breast[@Shu2019], and endometrial cancer[@Nead2015a; @Yarmolinsky2019], as well as cardiovascular disease[@Hagg2015]. \par

### Data
The following details a number of GWAS and meta-analyses of previously published studies which were used in this chapter to perform MR analyses and are required reporting as per STROBE-MR guidelines[@DaveySmith2019]. I did not perform these analyses. The total N and sex specific N does not always tally, this is because of variation in the N for each SNP. Where this is the case, 'N up to' is used. \par

<!--Ancestries (as opposed to ancestry) is used throughout as multiple ancestries are encompassed within the 'European ancestry' classification (see: [https://twitter.com/ewanbirney/status/1148574135284633600?s=20](https://twitter.com/ewanbirney/status/1148574135284633600?s=20)). -->

#### Exposures: measures of adiposity {#MR-exposures-adiposity}
##### Body mass index
Robustly associated genetic variants associated with BMI were extracted from Yengo et al. (2018)[@Yengo2018], who analysed data from 515,509--795,624 individuals of European ancestries from two studies, the Genetic Investigation of Anthropometric Traits (GIANT) consortium[@Locke2015] and UK Biobank. In both studies BMI was calculated as $\displaystyle \frac{weight\ (kg)}{height\ (m^2)}$. Yengo et al. (2018)[@Yengo2018] performed a fixed effect inverse variance weighted meta-analysis of BMI using GWAS results generated from UK Biobank (N = 456,426) and results from The Genetic Investigation of ANthropometric Traits (GIANT) consortium, Locke et al. (2015)[@Locke2015] (N = 322,154). These genetic variants were used as genetic instrumental variables in MR analyses. \par

In UK Biobank, BMI was adjusted for age, sex, recruitment centre, genotyping batch, and the first 10 principal components (PC) calculated from 132,102 (out of 147,604) genotyped SNPs pre-selected by the UK Biobank quality control team[@Bycroft2018]. In GIANT, BMI was adjusted for age, age squared, and study specific covariables (e.g. genotype-derived principal components). The residuals from both UK Biobank and GIANT were inverse normally transformed and therefore represent normalised standard deviation (SD) units. In total, 681,275 individuals of European ancestry and ~2.4 million HapMap-2 imputed SNPs were included in the meta-analysis. The intercept for linkage disequilibrium score regression (^I^LDSC 1.03; SE = 0.02) suggested population stratification however, LDSC can rise above 1 as sample sizes and heterogeneity increase[@Loh2018]. \par

In the combined meta-analysis, a total of 656 primary associations reaching a genome wide significance threshold of 5 x 10^-8^ were identified. Approximate conditional and joint multiple-SNP (COJO) analysis identified a further 285 independent SNPs reaching an adjusted genome-wide significance threshold of 1 x 10^-8^. Together, these 941 associations explain 6% (SE = 0.8%) and 22.4% (SE = 3.7%) of the variance and heritability of BMI respectively. COJO-specific summary statistics for all 941 SNPs were used in the main analysis. \par

##### Waist hip ratio
Robustly associated genetic variants associated with WHR were extracted from Pulit et al. (2019)[@Pulit2019], who analysed data from 485,486–-697,702 individuals of European ancestries from two studies: UK Biobank and the GIANT consortium[@Shungin2015]. In both studies, WHR was calculated as $\displaystyle \frac{waist\ circumference\ (cm)} {hip\ circumference\ (cm)}$. Pulit et al. (2019)[@Pulit2019] performed a fixed effects inverse variance weighted meta-analysis of WHR using GWAS results generated from UK Biobank (N up to = 485,486 (men up to = 263,148; women up to= 222,338)) and results from the GIANT consortium (N = up to212,248 (women up to= 118,004; men up to= 94,434))[@Shungin2015] using METAL[@Willer2010]. SNPs with a frequency difference > 15% between the two studies were removed prior to meta-analysis. In both studies, WHR was adjusted for sex (UK Biobank only), age at assessment, age at assessment squared, and assessment centre (UK Biobank only); study specific covariables were included in the GIANT GWAS where appropriate. The residuals from both UK Biobank and GIANT were inverse normally transformed and therefore represent normalised SD units. In GIANT residuals were calculated for men and women separately. In total, 316 associations reaching a genome-wide significance threshold of 5 x 10^-9^ were identified. The p-value was adjusted to account for denser imputation data[@Pulit2017]. These associations explain 3% of the phenotypic variance as calculated in an independent dataset (N = 7,721). SNP based heritability across all SNPs was estimated to be 22.7%. These genetic variants were used as genetic instrumental variables in MR analyses. \par

For the UK Biobank GWAS, the second release (June 2017) of UK Biobank data, which did not have corrected imputation at non-HRC sites, was used. Pulit et al performed a linear mixed model (LMM) using BOLT-LMM[@Loh2015] on SNPs with imputation quality score > 0.3, MAF > 0.01%, and SNPs present in the HRC imputation reference panel. The LMM was adjusted for the SNP array that was used to genotype each sample, i.e. genome-wide or metabochip array. No other covariables were included in the mode. They used an infinitesimal model in BOLT-LMM. In GIANT, individual studies recruited participants and undertook sample and SNP quality control. In studies using genome-wide arrays, imputation was performed with CEU (Utah residents of northern and western European ancestry) haplotypes from HapMap. Assuming an additive genetic model, each study ran a linear regression GWAS. Sex-specific summary statistics were corrected for population structure using the genomic control inflation factor. Prior to meta-analysis of studies using genome-wide array or Metabochip, SNPs were removed if they had a minor allele count $\leq$ 3, were not in Hardy-Weinberg equilibrium (p-value < 10^-6^), had a call rate < 95% or an imputation quality score < 0.3 for MACH, < 0.4 for IMPUTE, and < 0.8 for PLINK. In step 1, a meta-analysis of each array was performed using a fixed effects inverse variance weighted model and corrected for genomic control to account for structure between cohorts. In step 2, the meta-analysed genome wide array studies and meta-analysed Metabochip studies were meta-analysed using a fixed effects inverse variance weighted model using METAL. In this second step genomic correction was not performed. \par

##### Body fat percentage
Robustly associated genetic variants associated with BF were extracted from Lu et al. (2016)[@Lu2016]. In this GWAS individuals with dual-energy X-ray abosrptiometry (DXA) or bioelectrical impedance measured BF were included. The number of individuals with DXA and impedance measured BF was not reported, instead a total N was reported (N up to = 89,297). Lu et al. (2016)[@Lu2016] performed a fixed effects inverse variance weighted meta-analysis of BF using two meta-analyses generated from genome-wide array (N = 65,831) and Metabochip array GWAS (N = 23,468) using METAL[@Willer2010]. In total, up to 89,297 (men up to= 44,429; women up to= 45,525) individuals of European ancestries were included in the inverse variance weighted fixed effects meta-analysis. In total 7 SNPs reached a genome wide significance threshold (p-value < 5 x 10^-8^) and were considered independent (± 500kb of the most significant SNP). Estimation of variance explained was not available in the European ancestries meta-analysis. In a meta-analysis of individuals of all ancestries, which included up to 11,419 additional individuals of non-European ancestries, these 7 SNPs explained 0.416% of the variance in BF. The additional 5 SNPs identified in this all ancestries meta-analysis explained 0.58% of the variance in BF. These 7 genetic variants, identified in the European ancestries meta-analysis, were used as genetic instrumental variables in MR analyses. \par

In each cohort, BF was measured via a bioelectrical impedance device or DXA as described previously[@Kilpelainen2011]. For each study, BF was adjusted for age, age squared, and study-specific covariables (e.g. genotype-based principle components and study centre) if necessary. These residuals, for studies of unrelated individuals, were calculated separately in men and women, and separately in cases and controls. For studies of family-based design, the residuals were calculated in men and women together, and sex was additionally included in the model. The residuals were then inverse rank normally transformed prior to association testing, and therefore represent normalised SD units. For studies of family-based design, the family relatedness was additionally included in the association testing. \par

Lu et al performed their meta-analysis in two stages. First, two parallel meta-analyses were conducted; one meta-analysis combined summary statistics from genome-wide array GWAS, totalling up to 65,831 individuals of European ancestries and the other meta-analysis combined summary statistics from Metabochip array GWAS, totalling up to 23,469 individuals of European ancestries. Each study performed study specific quality control. Imputation was performed in each study using the European ancestries HapMap Phase II (Release 22) reference panel. Associations between individual SNPs and inverse normally transformed BF residuals were identified using linear regression assuming an additive genetic model. All SNPs with low imputation scores (MACH r2-hat < 0.3, IMPUTE proper_info < 0.4, or PLINK info < 0.8) and a MAC $\leq$ 3 were removed.  In the second stage, meta-analysis of these two meta-analyses was performed using an inverse variance-weighted fixed-effect model using METAL. \par

Each unique locus was defined as ±500*kb* on either side of the most significant SNP that reached the genome wide significance threshold (p-value < 5 x 10^-8^) in the meta-analysis. Genotype data for genome-wide significant SNPs were of high quality with a median imputation score of >= 0.95. \par

<!--
Inflation before genomic control correction was generally low (lambda men + women = 1.13; lambda men = 1.07; lambda women = 1.10). To reduce inflation of test statistics from potential population structure, individual GWAS results and GWAS meta-analysis results were corrected for genomic control using all SNPs. Individual Metabochip results and Metabochip meta-analysis results were genomic control corrected using 4,425 SNPs, which were derived from pruning of QT-interval replication SNPs within 500 kb of an anthropometry replication SNP on the Metabochip. The genomic control corrected GWAS and Metabochip meta-analysis results were then meta-analysed. 

LDSC suggested observed inflation was not due to population substructure; the regression intercept was 1.0045 (lambda genomic control = 1.136 and mean X2 = 1.16) for sex-combined, 0.999 (lambda genomic control = 1.062 and mean X2 = 1.079) for men-only and 1.014 (lambda genomic control = 1.105 and mean X2 = 1.112) for women-only analyses. The authors used these regression intercepts, rather than the lambda genomic control, to correct the meta-analysis results in more significant associations (for example, for the rs1558902-*FTO* SNP, p-value = 3.24 x 10-27 in the modified European sex-combined meta-analysis compared with p-value = 1.1 x 10-25). Overall, the less stringent correction did not result in the identification of novel loci. \par
-->

#### Outcomes: metabolites
##### Kettunen et al. (2016)[@Kettunen2016]
Genome-wide summary level data for 123 NMR derived metabolites, which includes derived metabolite measures (Appendix Table \@ref(tab:appendix-MR-table-metabolites)), were available from Kettunen et al. (2016)[@Kettunen2016]. Kettunen et al (2016)[@Kettunen2016] performed a fixed effects meta-analysis of 123 serum and EDTA (Ethylenediaminetetraacetic acid) serum NMR quantified metabolites from 14 GWAS. In total, up to 24,925 individuals of European ancestries were included in the meta-analysis. \par

In each of the 14 cohorts 123 metabolites were quantified from human blood. Four cohorts used EDTA-plasma, the other 10 used serum. The majority of blood samples were fasted. Where a study did not have over night fasted samples, correction for fasting time effect was performed using the `gam` `R` package and fitting a smoothed spline to adjust for fasting. The NMR analysis was performed with the comprehensive quantitative serum/plasma platform described by Soininen et al. (2009)[@Soininen2009]. All metabolites were adjusted for age, sex, time from last meal if applicable, and the first ten PCs. The residuals for each adjusted metabolite were inverse rank normally transformed and therefore represent normalised SD units. Each of the 14 cohorts performed a univariate GWAS assuming an additive genetic model. SNPs were imputed up to 39 million markers using the 1000 Genomes Project, March 2012 version. \par

For the meta-analysis, SNPs with accurate imputation (info > 0.4) and minor allele count > 0.3 were combined using double genomic control correction, that is, both individual cohort results and meta-analysis results were corrected for the genomic inflation factor as implemented in `GWAMA`. Up to 12,133,295 SNPs were included in the meta-analysis. Variants present in more than seven studies after filtering and meta-analysis were considered for the final results. All traits gave genomic inflation factors in the meta-analysis < 1.034 showing little evidence of systematic bias in the test statistic. A genome-wide significance threshold (p-value < 2.27 x 10^-9^) after correcting for 22 independent tests (22 being the number of principal components explaining over 95% of variation in the metabolite data) was used. \par

<!--
A total of 62 SNPs reached a genome wide significance threshold (p-value < 2.27 x 10^-9^) after correcting for 22 independent tests (22 being the number of principal components explaining over 95% of variation in the metabolite data). After performing conditional analysis, a further 12 SNPs were identified as independent after reaching the genome-wide significance threshold. The 12 additional SNPs were identified in 9 of the 62 main loci (*PCSK9*, *LPL*, *PPM1K*, *HAL*, *CETP*, *CILP*, *PLTP*, *APOB* and *LIPC*), with a third independent SNP identified in two of these loci (*APOB* and *LIPC*) and a fourth independent SNP identified in one of these (*LIPC*). Overall, 74 independent SNPs were associated with one or more of the 123 metabolites, with the variance explained ranging from 0.2% for acetoacetate to 12.5% for glycine, with a median of 5%. \par
-->

##### INTERVAL
Genome-wide summary level data for 230 NMR derived metabolites, which includes derived metabolites measures (Appendix Table \@ref(tab:appendix-MR-table-metabolites)), were available from the INTERVAL study. INTERVAL is a randomised trial of ~50,000 individuals recruited from June 2012--June 2014 in the United Kingdom to investigate the safety of different blood donation intervals[@Moore2014b]. In INTERVAL, a LMM GWAS of 230 serum NMR quantified metabolites was performed. In total, up to 40,849 individuals of European ancestries were included. The INTERVAL GWAS data were unpublished and provided by collaborators (Adam Butterworth, University of Cambridge). \par

SNPs were imputed using a combined 1000 Genomes + UK10K panel, which captured 87,696,910 variants. A LMM was fit using BOLT-LMM. Variants that were poorly imputed (info score < 0.3 or R^2^ < 0.3), variants with unrealistic results (e.g. standard error < 0, standard error > 10, beta = infinity), and variants with minor allele count < 5 were excluded. A genome-wide significance threshold was not set. A total of 28 PC explained over 95% of variation in the metabolite data. \par

The NMR analysis was performed with the comprehensive quantitative serum/plasma platform described by Soininen et al. (2009)[@Soininen2009]. Samples or analytes with potentially unreliable data were flagged. Flagged metabolites included acetate, pyruvate, glucose, and lactate. Individuals with flagged values for any of these metabolites had those values set to missing. In addition, metabolites with values of 0 or values > 10 SD from the mean after log transformation were set to missing. Individuals were excluded if they had > 30% analyte missingness, and one NMR PC outlier. All metabolites were log transformed and then adjusted for age, sex, recruitment centre, time between blood draw and sample processing, and the first 10 PCs of genetic ancestry. Residuals were then inverse normal rank transformed prior to GWAS and therefore represent log normalised SD units. \par

### Two-sample Mendelian randomization: effect of adiposity measures on metabolites
For all exposures, the following data were obtained from the original GWAS publications: rsID, effect allele, other/non-effect allele, effect allele frequency, effect estimate, standard error of the effect estimate, p-value, N, and units. The same data for these genetic instrumental variables were obtained from each metabolite for each exposure separately. Clumping of SNPs was not performed as the studies from which they were obtained stated they were independent or near-independent and had already been clumped. For BMI, near-independent SNPs were identified by Yengo et al. as those with the smallest p-value within a 1 mega-base (Mb) window, as well as those additional SNPs within that same window that reached the p-value threshold (1 x 10^-8^) after COJO analysis[@Yengo2018]. For WHR, lead SNPs (p-value < 5 x 10^-9^), and all SNPs in LD (r^2^ > 0.05) with the lead SNPs, within a 10Mb window were identified by Pulit et al. using the PLINK clumping algorithm. The genomic span of each LD-based clump was identified and a 1kb buffer was added up- and down-stream. Where windows overlapped, they were merged into a single locus[@Pulit2019]. For BF, Lu et al.[@Lu2016] defined each locus by the SNP with the smallest p-value that reached the GWS threshold (p-value < 5 x 10^-8^) and a 1Mb window centered around that SNP. F-statistics for each SNP and an average for each exposure were calculated. \par

Genetic instrumental variables were extracted from the metabolite GWAS and where these were not present, proxy SNPs were included if linkage disequilibrium was $\geq$ 0.8. For proxy SNPs, the inclusion of SNPs where the reference strand was ambiguous (strand flips) was allowed and the direction was inferred using a minor allele frequency threshold. That is, the direction was inferred using a minor allele frequency, so long as that frequency was not $\geq$ 0.3 or $\leq$ 0.7, in which case it was excluded. Exposure and outcome SNPs were harmonized in reference to the exposure effect allele being on the increasing scale. For included alleles where the reference strand was ambiguous, the positive strand was inferred using effect allele frequency. \par

An inverse variance weighted (IVW), multiplicative random effects (IVW-MRE) model was used to investigate the effect of each exposure on each metabolite. The model assumes that the strength of the association of the genetic instruments with the exposure is not correlated with the magnitude of the pleiotropic effects and that the pleiotropic effects have an average value of zero[@Bowden2017]. A multiple testing threshold of p-value < `r round(0.05/22,4)` was applied to the Kettunen et al. (2016) metabolomics data. This is based on the number of principal components (22) in the Kettunen et al. (2016) meta-analysis that explained 95% of the variation in metabolite data. A multiple testing threshold of p-value < `r round(0.05/28,4)` was applied to the INTERVAL metabolomics data. This is based on the number of principal components (28) in the INTERVAL GWAS that explained 95% of the variation in metabolite data. \par

The metabolic profile of adiposity was visualised using Circos plots. Directions of effect were compared across exposures for analysis using Kettunen data and analysis using INTERVAL data. Tests which reached a multiple testing threshold within each analysis are presented and the effect of adiposity on subclasses were explored in regard to the multiple testing threshold. \par

#### Sensitivity analysis {#MR-sensitivity}
Where possible, the assumptions of no pleiotropy among genetic instruments and outcomes were explored using: MR-Egger[@Bowden2015], weighted median[@Burgess2017a], and weighted mode[@Hartwig2017] based estimators. Sensitivity analysis was performed for all exposure-outcome pairs, but focus was given to those pairs which met a multiple testing threshold in the main analysis. For these sensitivity models, no threshold requirements were set, instead, consistency between the IVW-MRE model and these methods was investigated. \par

MR-Egger provides an estimate of unbalanced/directional horizontal pleiotropy via the intercept of a linear regression of the SNP-exposure and SNP-outcome association. In the presence of pleiotropy the intercept will bias away from the origin. MR-Egger gives consistent estimates when 100% of genetic instruments are invalid[@Bowden2015]. The weighted median is complimentary to MR-Egger but does not rely on the "instrument strength independent of direct effect" (InSIDE) assumption. It calculates the median of an empirical distribution of the causal effect estimates weighted for precision. It provides consistent estimates when at least 50% of the weight comes from valid genetic instruments and as long as no one genetic instrument contributes > 50% of the weight[@Burgess2017a]. The weighted mode assumes the true causal effect is the most common effect, it is robust when the majority of effect estimates are derived from valid instruments[@Hartwig2017]. \par

In these analyses, it is assumed that the instruments influence the exposure first. Their effect on the outcome through the exposure is secondary. Given the large number of genetic instruments used, as well as the feed-back and feed-forward loops present in the metabolome, the directionality of association may be difficult to evaluate. For examlpe, a SNP which is strongly associated with adiposity may also have an effect on the metabolite under investigation via that SNPs association with another metabolite which is up- or down-stream of that metabolite. The Steiger test[@Hemani2017], applied here using the `TwoSampleMR` package, can be used to estimate whether the test under investigation is the "true" causal direction. The Steiger test calculates the variance explained in the exposure and the variance explained in the outcome by the exposure related instruments. If the variance explained in the outcome is less than that explained in the exposure, then the direction of effect can be considered to be from the exposure to the outcome. In order to estimate the variance explained the N is required. For the Kettunen dataset, the N for each individual SNP was available. For the INTERVAL dataset, the overall sample size (N = 40,849) was used as the individual SNP sample sizes were not available. \par

#### Additional sensitivity analysis
The effects of heterogeneity in the exposure instruments was investigated using Cochran's Q statistic for IVW-MRE and MR-Egger models. A single SNP MR, whereby the association of each SNP individually is estimated using the IVW and MR-Egger models, and Wald ratio where appropriate, was also used to investigate heterogeneity. A l"eave-one-out" MR analysis was performed whereby each SNP was sequentially left out of the MR analysis and the causal effect estimated absent of that SNP. If the estimated effect is substantially altered after the removal of a single SNP, this may imply that SNP is driving the association between the exposure and outcome. Finally, each of these additional sensitivity analyses were visualised using a funnel plot and using forest plots of the single SNP and leave-one-out MR analysis to identify potential pleiotropic effects. These additional sensitivity analysis are presented as a summary for each analysis. \par

#### Additional analyses
There were a number of potential limitations with the genetic instrumental variables used in the main analysis. In order to test whether these limitations produced biased results the following post-hoc additional analyses were performed. Firstly, both BMI and WHR genetic instrumental variables were obtained from studies using UK Biobank, which has shown evidence of latent population structure[@Haworth2019; @Berg2019]. BF instruments were obtained from a study which used different measures of BF; two of the included SNPs have also been associated previously with ‘favourable adiposity’ and may therefore not be reflections of total body fat[@Yaghootkar2014; @Yaghootkar2016]. Additional genetic instrumental variables for BMI, WHR and BF were obtained from additional published summary statistics (Appendix Table \@ref(tab:appendix-MR-table-exposures)). Briefly, SNPs for BMI were obtained from the initial non-COJO analysis by Yengo et al 2018[@Yengo2018], and a separate set of SNPs were obtained from Locke et al. (2015)[@Locke2015] - which did not use UK Biobank; for WHR, SNPs were obtained from Shungin et al. (2015)[@Shungin2015] - which did not use UK Biobank; for BF, the SNPs associated with ‘favourable adiposity’ were removed and an additional SNP set identified in a GWAS using a single measure of BF was obtained from Hubel et al. (2019)[@Hubel2019]. Outcome data extraction and harmonization was performed as for the main analysis. F-statistics (Figure \@ref(fig:appendix-MR-figure-fstatistics)) and detailed information on each additional exposure is available in the Appendix, section \@ref(appendix-MR-post-hoc). The main analysis (IVW-MRE, MR-Egger, weighted median, weighted mode and additional sensitivity analysis) was re-run using these additional exposures. \par

In addition, studies used a variety of methods and definitions of independence for SNPs. In order to ensure SNPs were independent to the same degree across all studies, genetic instrumental variables for all exposures (those used in the main analysis and in additional analyses described here) were clumped and the main analysis (IVW-MRE, MR-Egger, weighted median, weighted mode and additional sensitivity analysis) was re-run using these clumped instruments. Clumping was performed using the `R` package `TwoSampleMR`[@Hemani2018] setting an LD R^2^ threshold of $0.001$ for SNPs within a 10,000 base window of each other. Spearman's correlation between MR results from the non-clumped and clumped SNP lists was performed. \par

### Meta-analysis of two-sample Mendelian randomization results
Metabolomics data from Kettunen et al. (2016)[@Kettunen2016] were inverse rank normally transformed prior to GWAS. For INTERVAL, metabolomics data were log transformed and then inverse rank transformed prior to GWAS. As these transformations use different scales, meta-analysis using MR effect estimates was not appropriate. Instead, a meta-analysis of each exposure-outcome pair was performed with p-values using the `metap` (version 1.4) `R` package using Fisher's method for combining p-values. Meta-analysis of the IVW-MRE results from the main analysis was performed. \par

Fisher's method \@ref(eq:fishers-method) combines p-values from each test into one test statistic ($X^2$), where $p_i$ is the p-value for the $i$^th^ test, $k$ is the number of tests, and $2k$ is the degrees of freedom. When all the null hypotheses are true, and the $p_i$ are independent, $X^2$ has a chi-squared distribution with $2k$ degrees of freedom. \par

\begin{equation}
  X^2_{2k} \sim -2 \sum_{i=1}^{k}ln(p_i), 
  (\#eq:fishers-method)
\end{equation}

The chi-squared statistic will follow a chi-squared distribution with $2k$  degrees of freedom if there is no effect in every study, where $k$ is the number of tests. A large chi-squared statistic compared to the degrees of freedom (with a corresponding low p-value) provides evidence of an effect in at least one study. The null hypothesis is that each tests null hypothesis is true; the alternative is that at least one of the tests null hypotheses is true. Acceptance of the null hypothesis is not interpreted as evidence of no effect in all studies. Given only two studies are included in any one exposure-outcome pair in the current work, the potential effects of heterogeneity are difficult to interpret and are therefore not focused on here. Meta-analysis results are presented as directions of effect and p-values. A Bonferroni corrected p-value threshold was used to identify evidence of association. \par

<!--
Given a null hypothesis for $i$, the p-value $p_i$ follows a uniform distribution on the interval [0,1]. The negative natural logarithm of a uniformly distributed value follows an exponential distribution. Scaling a value that follows an exponential distribution by a factor of two yields a quantity that follows a chi-squared distribution with two degrees of freedom. Finally, the sum of $k$ independent chi-squared values, each with two degrees of freedom, follows a chi-squared distribution with $2k$ degrees of freedom. \par
-->

### Comparison of two-sample Mendelian randomization and observational results from Chapter \@ref(systematic-review)
Metabolites included in the meta-analysis, which showed consistent directions of effect across the Kettunen and INTERVAL data and which reached a Bonferroni corrected meta-analysis p-value threshold, were compared with observational results from Chapter \@ref(observational). Direction of effect in the meta-analysis was compared with direction of effect in the observational analysis to triangulate evidence, where consistency across methods improved confidence in causal effects. \par

## Results {#MR-results}
```{r eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
# data 
data <- read.table("../../000_thesis/index/data/MR/tables/exposure_SNPs.txt", header = T, sep = "\t")
bmi <- subset(data, Exposure == "BMI")
bmi <- subset(bmi, Ref == "Yengo")
bmi <- subset(bmi, Note == "COJO")
bmi_f <- bmi$mean_fstat[1]
whr <- subset(data, Exposure == "WHR")
whr <- subset(whr, Ref == "Pulit")
whr_f <- whr$mean_fstat[1]
bf <- subset(data, Exposure == "BF")
bf <- subset(bf, Ref == "Lu")
bf_f <- bf$mean_fstat[1]
```
The effects of BMI, WHR and BF on a total of 123 metabolites from Kettunen et al. (2016)[@Kettunen2016] and 230 metabolites from INTERVAL were investigated using an IVW-MRE model. F-statistics for all SNPs used as genetic instruments for each exposure were above 10 (mean F-statistics: BMI = `r round(bmi_f)`, WHR = `r round(whr_f)`, BF = `r round(bf_f)`; Figure \@ref(fig:appendix-MR-figure-fstatistics); [GitHub](https://github.com/mattlee821/000_thesis/blob/master/index/data/MR/tables/exposure_SNPs.txt)). \par 

For MR analyses using the Kettunen et al. (2016) metabolites, effect estimates are given in SD units per SD higher BMI/WHR/BF. For MR analyses using the INTERVAL metabolites, effect estimates are given in log SD units per SD higher BMI/WHR/BF. Metabolites were grouped into subclasses by the metabolomics analytic platform. The Kettunen data included two subclasses (Metabolites ratio and Protein) which were not present in the INTERVAL data In the INTERVAL data, 16 subclasses, which were all derived metabolic measures (i.e. ratios of one measured metabolite to another), were not available in the Kettunen data \par

### Two-sample Mendelian randomization: effect of adiposity measures on metabolites
#### Metabolic profile
```{r eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
# packages ====
library(tidyr)
library(dplyr)
# data ==== 
data <- read.table("../../002_adiposity_metabolites/analysis/step1/combined/001_combined_mr_results_kettunen.txt", header = T, sep ="\t")
data <- subset(data, method == "Inverse variance weighted (multiplicative random effects)")
main_exposures <- c("BMI_Yengo_941", "WHR_Pulit_316", "BF_Lu_7")
data <- data[data$exposure %in% main_exposures, ]
data <- data[,c("exposure","b","se","pval","raw.label","subclass")]
kettunen <- data[order(data$b),]

data <- read.table("../../006_interval_metabolites/analysis/combined/001_combined_mr_results.txt", header = T, sep ="\t")
data <- subset(data, method == "Inverse variance weighted (multiplicative random effects)")
main_exposures <- c("BMI_Yengo_941", "WHR_Pulit_316", "BF_Lu_7")
data <- data[data$exposure %in% main_exposures, ]
data <- data[,c("exposure","b","se","pval","raw.label","subclass")]
interval <- data[order(data$b),]
```
Effect estimates from the IVW-MRE model revealed evidence for a broad effect of adiposity on the metabolic profile (Figure \@ref(fig:MR-figure-circosplot-kettunen-main-analysis) and \@ref(fig:MR-figure-circosplot-interval-main-analysis)). The pattern of association was visually similar when using the Kettunen data for BMI and WHR, and when using the INTERVAL data for BMI and WHR. When using the Kettunen data, the effect of BF on metabolites was visually distinct to that of BMI and WHR effects, with many effect estimates appearing to be inverse to those for BMI and WHR. However, when using the INTERVAL data, the effect of BF looked more similar in regards to the direction of effect to that of BMI and WHR compared to effects in the Kettunen data. However, these effects appear to cross the null much more often. Effects for WHR were generally larger with wider confidence intervals, whereas those for BMI were smaller with tighter confidence intervals. Effects for BF were much larger across both metabolite datasets, with wide confidence intervals which spanned the null. The wide confidence intervals observed for BF and the tighter confidence intervals for BMI are perhaps expected given the variance explained by instruments for each measure of adiposity varies from 0.4% for BF, 3% for WHR, and 6% for BMI. \par

\newpage
\thispagestyle{empty}

(ref:MR-figure-circosplot-kettunen-main-analysis-cap) **Mendelian randomization analysis: Effect of adiposity measures on 123 NMR derived metabolites using Kettunen data**. Circos plot shows each track as one measures of adiposity; the outer track is body mass index (BMI), the middle track is waist hip ratio (WHR), the inner track is body fat percentage (BF). Solid points indicate a multiple testing threshold (`r round(0.05/22,4)`) has been reached. Effect estimates are given in SD units per SD higher BMI/WHR/BF. 95% confidence intervals shown. Metabolites are grouped by subclass and arranged alphabetically within each subclass.

(ref:MR-figure-circosplot-kettunen-main-analysis-scap) Mendelian randomization analysis: Effect of adiposity measures on 123 NMR derived metabolites using Kettunen data

```{r MR-figure-circosplot-kettunen-main-analysis, echo=FALSE, out.width='120%', fig.align='center', fig.cap='(ref:MR-figure-circosplot-kettunen-main-analysis-cap)', fig.scap='(ref:MR-figure-circosplot-kettunen-main-analysis-scap)'}
knitr::include_graphics("data/MR/figures/circosplot_kettunen_main_analysis.pdf")
```

\newpage
\thispagestyle{empty}

(ref:MR-figure-circosplot-interval-main-analysis-cap) **Mendelian randomization analysis: Effect of adiposity measures on 230 NMR derived metabolites using INTERVAL data**. Circos plot shows each track as one measures of adiposity; the outer track is body mass index (BMI), the middle track is waist hip ratio (WHR), the inner track is body fat percentage (BF). Solid points indicate a multiple testing threshold (`r round(0.05/28,4)`) has been reached. effect estimates are given in log SD units per SD higher BMI/WHR/BF. 95% confidence intervals shown. Metabolites are grouped by subclass and arranged alphabetically within each subclass.

(ref:MR-figure-circosplot-interval-main-analysis-scap) Mendelian randomization analysis: Effect of adiposity measures on 230 NMR derived metabolites using INTERVAL data

```{r MR-figure-circosplot-interval-main-analysis, echo=FALSE, out.width='120%', fig.align='center', fig.cap='(ref:MR-figure-circosplot-interval-main-analysis-cap)', fig.scap='(ref:MR-figure-circosplot-interval-main-analysis-scap)'}
knitr::include_graphics("data/MR/figures/circosplot_interval_main_analysis.pdf")
```

#### Consistency of effect direction
```{r eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE, out.width='100%'}
# packages
library(tidyr)
library(dplyr)
library(ggplot2)
library(cowplot)
source("data/index/colour_palette.R")

# kettunen ====
# data ==== 
data <- read.table("../../002_adiposity_metabolites/analysis/step1/combined/001_combined_mr_results_kettunen.txt", header = T, sep ="\t")
data <- data[,c("id.outcome","exposure","method","b")]
ivw <- subset(data, method == "Inverse variance weighted (multiplicative random effects)")
mr_egger <- subset(data, method == "MR-Egger")
weighted_median <- subset(data, method == "Weighted median")
weighted_mode <- subset(data, method == "Weighted mode")

## seperate data into exposures ====
main_exposures <- c("BMI_Yengo_941", "WHR_Pulit_316", "BF_Lu_7")
main <- ivw[ivw$exposure %in% main_exposures, ]

## convert to wide based on model ====
main <- main[,c(1,2,4)]
main <- spread(main, exposure, b)

## assign values for directions consistent across non-clumped and clumped data ====
###  ====
data <- main[,c(2:ncol(main))]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect"))
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data1 <- data[,c(4,5)]
data1$exposure <- "All"

###  ====
data <- main[,c(3,4)]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect"))
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data2 <- data[,c(3,4)]
data2$exposure <- "BMI & WHR"

###  ====
data <- main[,c(3,2)]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect"))
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data3 <- data[,c(3,4)]
data3$exposure <- "BMI & BF"

###  ====
data <- main[,c(2,4)]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect"))
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data4 <- data[,c(3,4)]
data4$exposure <- "BF & WHR"


# plot ====
plot_data <- rbind(data1,data2,data3,data4)
kettunen <- plot_data

# interval ====
# data ==== 
data <- read.table("../../006_interval_metabolites/analysis/combined/001_combined_mr_results.txt", header = T, sep ="\t")
data <- data[,c("metabolite","exposure","method","b")]
ivw <- subset(data, method == "Inverse variance weighted (multiplicative random effects)")
mr_egger <- subset(data, method == "MR-Egger")
weighted_median <- subset(data, method == "Weighted median")
weighted_mode <- subset(data, method == "Weighted mode")

## seperate data into exposures ====
main_exposures <- c("BMI_Yengo_941", "WHR_Pulit_316", "BF_Lu_7")
main <- ivw[ivw$exposure %in% main_exposures, ]

## convert to wide based on model ====
main <- main[,c(1,2,4)]
main <- spread(main, exposure, b)

## assign values for directions consistent across non-clumped and clumped data ====
###  ====
data <- main[,c(2:ncol(main))]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect"))
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data1 <- data[,c(4,5)]
data1$exposure <- "All"

###  ====
data <- main[,c(3,4)]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect"))
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data2 <- data[,c(3,4)]
data2$exposure <- "BMI & WHR"

###  ====
data <- main[,c(3,2)]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect"))
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data3 <- data[,c(3,4)]
data3$exposure <- "BMI & BF"

###  ====
data <- main[,c(2,4)]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect"))
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data4 <- data[,c(3,4)]
data4$exposure <- "BF & WHR"


# plot ====
plot_data <- rbind(data1,data2,data3,data4)
interval <- plot_data
```
Given adiposity measures used here are highly correlated, and evidence from Chapter \@ref(SR) highlighted broad consistency in the direction of effect estimates across adiposity measures, consistency of effect direction across exposures within datasets was investigated. Directional concordance or discordance may help to understand the underlying mechanisms of the adiposity-metabolite relationship, for example, if a metabolite is decreased by BMI but increased by WHR the effect of deposition over composition will be more important in downstream analyses. Here, a positive effect is identified if there is a positive effect for all exposures being assessed, i.e. if BMI, WHR, and BF all had a positive effect on metabolite A this would be recorded as a positive effect. If however, the effect of BF on metabolite A was negative this would be recorded as an 'opposite effect'. \par

Across all exposures, directional consistency of effect estimates from the IVW-MRE model was low for both metabolite sources from the Kettunen (N opposite effect = `r table(kettunen$direction_group, kettunen$exposure)[3,1]`) and INTERVAL (N = `r table(interval$direction_group, interval$exposure)[3,1]`) data (Figure \@ref(fig:MR-figure-directional-consistency)). This was the same for BF and WHR and for BMI and BF. Directional consistency was much higher for BMI and WHR for both analyses using the Kettunen (N = `r table(kettunen$direction_group, kettunen$exposure)[3,4]`) and INTERVAL (`r table(interval$direction_group, interval$exposure)[3,4]`) data \par

(ref:MR-figure-directional-consistency-cap) **Directional consistency of two-sample MR effect estimates**. A positive effect reflects the effect estimates being either all positive or both positive, depending on whether the comparison is with all three exposure (All) or between just two exposures; a negative effect reflects effect estimates being in a negative direction; opposite effect reflects different directions for the effect estimates across the comparisons. **A:** Two-sample MR IVW-MRE for 123 metabolites from Kettunen et al. (2016)[@Kettunen2016]; **B:** Two-sample MR IVW-MRE for 230 metabolites from INTERVAL. BMI = body mass index; WHR = waist hip ratio; BF = body fat percentage.

(ref:MR-figure-directional-consistency-scap) Directional consistency of two-sample MR effect estimates

```{r MR-figure-directional-consistency, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE, out.width='100%', fig.cap='(ref:MR-figure-directional-consistency-cap)', fig.scap='(ref:MR-figure-directional-consistency-scap)'}
# packages
library(tidyr)
library(dplyr)
library(ggplot2)
library(cowplot)
source("data/index/colour_palette.R")

# kettunen ====
# data ==== 
data <- read.table("../../002_adiposity_metabolites/analysis/step1/combined/001_combined_mr_results_kettunen.txt", header = T, sep ="\t")
data <- data[,c("id.outcome","exposure","method","b")]
ivw <- subset(data, method == "Inverse variance weighted (multiplicative random effects)")
mr_egger <- subset(data, method == "MR-Egger")
weighted_median <- subset(data, method == "Weighted median")
weighted_mode <- subset(data, method == "Weighted mode")

## seperate data into exposures ====
main_exposures <- c("BMI_Yengo_941", "WHR_Pulit_316", "BF_Lu_7")
main <- ivw[ivw$exposure %in% main_exposures, ]

## convert to wide based on model ====
main <- main[,c(1,2,4)]
main <- spread(main, exposure, b)

## assign values for directions consistent across non-clumped and clumped data ====
###  ====
data <- main[,c(2:ncol(main))]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect"))
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data1 <- data[,c(4,5)]
data1$exposure <- "All"

###  ====
data <- main[,c(3,4)]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect"))
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data2 <- data[,c(3,4)]
data2$exposure <- "BMI & WHR"

###  ====
data <- main[,c(3,2)]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect"))
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data3 <- data[,c(3,4)]
data3$exposure <- "BMI & BF"

###  ====
data <- main[,c(2,4)]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect"))
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data4 <- data[,c(3,4)]
data4$exposure <- "BF & WHR"


# plot ====
plot_data <- rbind(data1,data2,data3,data4)
kettunen <- plot_data
p_kettunen <- ggplot(data = plot_data,
       aes(x = exposure)) +
  geom_bar(aes(fill = direction_group))  +
  scale_fill_manual(values = discrete_wes_pal) +
  guides(fill = guide_legend(override.aes = list(size = 5),
                             title = "",
                             label.hjust = 0,
                             label.vjust = 0.5)) +
  xlab("") +
  theme_cowplot() +
  theme(axis.text.x = element_text(angle = 90),
        axis.ticks.x = element_blank(),
        legend.position = "none")

# interval ====
# data ==== 
data <- read.table("../../006_interval_metabolites/analysis/combined/001_combined_mr_results.txt", header = T, sep ="\t")
data <- data[,c("metabolite","exposure","method","b")]
ivw <- subset(data, method == "Inverse variance weighted (multiplicative random effects)")
mr_egger <- subset(data, method == "MR-Egger")
weighted_median <- subset(data, method == "Weighted median")
weighted_mode <- subset(data, method == "Weighted mode")

## seperate data into exposures ====
main_exposures <- c("BMI_Yengo_941", "WHR_Pulit_316", "BF_Lu_7")
main <- ivw[ivw$exposure %in% main_exposures, ]

## convert to wide based on model ====
main <- main[,c(1,2,4)]
main <- spread(main, exposure, b)

## assign values for directions consistent across non-clumped and clumped data ====
###  ====
data <- main[,c(2:ncol(main))]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect"))
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data1 <- data[,c(4,5)]
data1$exposure <- "All"

###  ====
data <- main[,c(3,4)]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect"))
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data2 <- data[,c(3,4)]
data2$exposure <- "BMI & WHR"

###  ====
data <- main[,c(3,2)]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect"))
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data3 <- data[,c(3,4)]
data3$exposure <- "BMI & BF"

###  ====
data <- main[,c(2,4)]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect"))
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data4 <- data[,c(3,4)]
data4$exposure <- "BF & WHR"


# plot ====
plot_data <- rbind(data1,data2,data3,data4)
interval <- plot_data
p_interval <- ggplot(data = plot_data,
       aes(x = exposure)) +
  geom_bar(aes(fill = direction_group))  +
  scale_fill_manual(values = discrete_wes_pal) +
  guides(fill = guide_legend(override.aes = list(size = 5),
                             title = "",
                             label.hjust = 0,
                             label.vjust = 0.5)) +
  xlab("") +
  theme_cowplot() +
  theme(axis.text.x = element_text(angle = 90),
        axis.ticks.x = element_blank(),
        legend.position = "none")

# legend ====
legend <- ggplot(data = plot_data,
       aes(x = exposure)) +
  geom_bar(aes(fill = direction_group))  +
  scale_fill_manual(values = discrete_wes_pal) +
  guides(fill = guide_legend(override.aes = list(size = 5),
                             title = "",
                             label.hjust = 0,
                             label.vjust = 0.5)) +
  xlab("") +
  theme_cowplot() +
  theme(axis.text.x = element_text(angle = 90),
        axis.ticks.x = element_blank())

legend <- get_legend(legend)

# plot ====
plot_grid(p_kettunen, p_interval, legend, nrow = 1, labels = c("A", "B"))

```

#### Multiple testing threshold
```{r eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
# packages
library(tidyr)
library(dplyr)
# data ==== 
data <- read.table("../../002_adiposity_metabolites/analysis/step1/combined/001_combined_mr_results_kettunen.txt", header = T, sep ="\t")
data <- data[,c("raw.label", "exposure", "method", "pval", "b")]
main <- subset(data, method == "Inverse variance weighted (multiplicative random effects)")
## seperate data into exposures ====
bmi <- subset(main, exposure == "BMI_Yengo_941")
whr <- subset(main, exposure == "WHR_Pulit_316")
bf <- subset(main, exposure == "BF_Lu_7")
## identify associations ====
kettunen_bmi <- subset(bmi, pval <= 0.05/22)
kettunen_whr <- subset(whr, pval <= 0.05/22)
kettunen_bf <- subset(bf, pval <= 0.05/22)
## overlapping associations ====
kettunen_bmi_whr <- inner_join(kettunen_bmi,kettunen_whr, by = "raw.label")
kettunen_bmi_only <- anti_join(kettunen_bmi,kettunen_whr, by = "raw.label")
kettunen_whr_only <- anti_join(kettunen_whr,kettunen_bmi, by = "raw.label")

data <- read.table("../../006_interval_metabolites/analysis/combined/001_combined_mr_results.txt", header = T, sep ="\t")
data <- data[,c("raw.label", "exposure", "method", "pval", "b")]
main <- subset(data, method == "Inverse variance weighted (multiplicative random effects)")
## seperate data into exposures ====
bmi <- subset(main, exposure == "BMI_Yengo_941")
whr <- subset(main, exposure == "WHR_Pulit_316")
bf <- subset(main, exposure == "BF_Lu_7")
## identify associations ====
interval_bmi <- subset(bmi, pval <= 0.05/28)
interval_whr <- subset(whr, pval <= 0.05/28)
interval_bf <- subset(bf, pval <= 0.05/28)
## overlapping associations ====
interval_bmi_whr <- inner_join(interval_bmi,interval_whr, by = "raw.label")
interval_bmi_only <- anti_join(interval_bmi,interval_whr, by = "raw.label")
interval_whr_only <- anti_join(interval_whr,interval_bmi, by = "raw.label")

```
Multiple testing thresholds of `r round(0.05/22,4)` and `r round(0.05/28,4)` were used for the analyses using the Kettunen and INTERVAL data, respectively. For the 123 metabolites in the Kettunen data, a total of `r nrow(kettunen_bmi)`, `r nrow(kettunen_whr)`, and `r nrow(kettunen_bf)` tests reached a multiple testing threshold for BMI, WHR, and BF respectively. Of these tests, `r nrow(kettunen_bmi_whr)` reached a multiple testing threshold across both BMI and WHR. A total of `r nrow(kettunen_bmi_only)` and `r nrow(kettunen_whr_only)` tests reached a multiple testing threshold for BMI only and WHR only respectively. For the 230 INTERVAL metabolites, a larger proportion of metabolites reached the multiple testing threshold. A total of `r nrow(interval_bmi)`, `r nrow(interval_whr)`, and `r nrow(interval_bf)` tests reached a multiple testing threshold for BMI, WHR, and BF respectively. Of these tests, `r nrow(interval_bmi_whr)` reached a multiple testing threshold across both BMI and WHR. A total of `r nrow(interval_bmi_only)` and `r nrow(interval_whr_only)` tests reached a multiple testing threshold for BMI only and WHR only, respectively. None of the `r nrow(interval_bf)` tests for BF overlapped with BMI or WHR tests. \par

For the `r nrow(kettunen_bmi_whr)` tests reaching a multiple testing threshold for BMI and WHR in analysis using the Kettunen data, the strongest positive and negative effects for BMI were for Total lipids in chylomicrons and extremely large VLDL (beta = 0.12) and Cholesterol esters in large HDL (beta = -0.12) respectively. For WHR the strongest positive and negative effects were found for Triglycerides in small VLDL (beta = 0.31) and Free cholesterol in large HDL (beta = -0.38) respectively. For the `r nrow(interval_bmi_whr)` tests reaching a multiple testing threshold for BMI and WHR in analysis using the INTERVAL data, the strongest positive and negative effects for BMI were found for Phenylalanine (beta = 0.10) and Mean diameter for HDL particles (beta = -0.10) respectively. For WHR the strongest positive and negative effects were found for Ratio of triglycerides to phosphoglycerides ratio (beta = 0.40) and Free cholesterol in large HDL to total lipids in large HDL ratio (beta = -0.41) respectively. For BF, all `r nrow(interval_bf)` tests reaching the multiple testing threshold were in the subclass Very small VLDL. Three of these (Total cholesterol in very small VLDL, Cholesterol esters in very small VLDL, and the ratio of these two) had the same effect sizes and p-value (beta = 0.2241230; p-value = 5.27 x 10^-7^). The fourth metabolite, Total cholesterol in very small VLDL to total lipids in very small VLDL ratio had a similar effect size (beta = 0.27). \par

#### Subclass results
When using the Kettunen data, tests reaching the multiple testing threshold (p-value < `r 0.05/22`) were observed for at least one exposure in 23 of 28 subclasses across the three exposures. No tests reached the multiple testing threshold for subclasses: small LDL, medium LDL, large LDL, protein, and fluid balance. However, a number of metabolites in these subclasses had confidence intervals which did not span the null. For subclasses IDL, metabolites ratio, ketone bodies, glycolysis related metabolites, glycerides and phospholipids, cholesterol, and amino acids, only a small number of metabolites within each subclass reached the multiple testing threshold. Whereas, for subclasses small VLDL, medium VLDL, large VLDL, very large VLDL, extremely large VLDL, large HDL, very large HDL, lipoprotein particle size, branched-chain amino acids, and aromatic amino acids a majority of metabolites reached the multiple testing threshold.

The INTERVAL data grouped metabolites into 42 subclasses. The additional subclasses to that in the Kettunen data were comprised of ratios (e.g., small HDL ratios). A protein subclass, available in the Kettunen data, was not available in the INTERVAL data. Of the 42 subclasses, tests reached a multiple testing threshold ($0.05/28$) in 39 subclasses. No tests reached the multiple testing threshold for subclasses fluid balance, glycolysis related metabolites, and ketone bodies. A majority of tests did not reach the multiple testing threshold in a majority of subclasses. Only 12 subclasses (amino acids, apolipoproteins, aromatic amino acids, branched chain amino acids, glycerides and phospholipids ratios, inflammation, large HDL, large HDL ratios, medium HDL, medium HDL ratios, very large HDL, and very large HDL ratios) had a majority of tests reaching the multiple testing threshold. \par

#### Sensitivity analysis
```{r eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
# packages
library(tibble)
library(tidyr)
library(dplyr)
library(ggplot2)
library(cowplot)
source("data/index/colour_palette.R")
# kettunen ====
# data ==== 
data <- read.table("../../002_adiposity_metabolites/analysis/step1/combined/001_combined_mr_results_kettunen.txt", header = T, sep ="\t")
data <- data[,c("name", "exposure", "method", "b")]
bmi <- subset(data, exposure == "BMI_Yengo_941")
whr <- subset(data, exposure == "WHR_Pulit_316")
bf <- subset(data, exposure == "BF_Lu_7")

bmi <- spread(bmi, method, b)
rownames(bmi) <- bmi[,1]
whr <- spread(whr, method, b)
rownames(whr) <- whr[,1]
bf <- spread(bf, method, b)
rownames(bf) <- bf[,1]

## assign values for directions ====
###  ====
data <- bmi[,c(3:ncol(bmi))]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  rownames_to_column() %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect")) %>%
  column_to_rownames()
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data1 <- data[,c(5,6)]
data1$exposure <- "BMI"

###  ====
data <- whr[,c(3:ncol(whr))]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  rownames_to_column() %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect")) %>%
  column_to_rownames()
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data2 <- data[,c(5,6)]
data2$exposure <- "WHR"

###  ====
data <- bf[,c(3:ncol(bf))]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  rownames_to_column() %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect")) %>%
  column_to_rownames()
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data3 <- data[,c(5,6)]
data3$exposure <- "BF"

# ====
data1 <- cbind(rownames(data1), data.frame(data1, row.names=NULL))
colnames(data1)[1] <- "name"
data2 <- cbind(rownames(data2), data.frame(data2, row.names=NULL))
colnames(data2)[1] <- "name"
data3 <- cbind(rownames(data3), data.frame(data3, row.names=NULL))
colnames(data3)[1] <- "name"
plot_data <- rbind(data1,data2,data3)
plot_data$exposure <- as.factor(plot_data$exposure)
plot_data$exposure <- factor(plot_data$exposure,levels(plot_data$exposure)[c(2,3,1)])
plot_data_kettunen <- plot_data
data <- subset(plot_data, direction != 0)
kettunen_bmi_list <- subset(data, exposure == "BMI")
kettunen_whr_list <- subset(data, exposure == "WHR")
kettunen_bf_list <- subset(data, exposure == "BF")

# shared metabs ====
data1 <- kettunen_bmi_list
data2 <- kettunen_whr_list
data3 <- kettunen_bf_list
data4 <- inner_join(data2,data1, by = "name")
data5 <- inner_join(data4,data3, by = "name")
kettunen_shared_list <- as.character(data5$name)

# interval ====
# data ==== 
data <- read.table("../../006_interval_metabolites/analysis/combined/001_combined_mr_results.txt", header = T, sep ="\t")
data <- data[,c("label.no.units", "exposure", "method", "b")]
bmi <- subset(data, exposure == "BMI_Yengo_941")
whr <- subset(data, exposure == "WHR_Pulit_316")
bf <- subset(data, exposure == "BF_Lu_7")

bmi <- spread(bmi, method, b)
rownames(bmi) <- bmi[,1]
whr <- spread(whr, method, b)
rownames(whr) <- whr[,1]
bf <- spread(bf, method, b)
rownames(bf) <- bf[,1]

## assign values for directions ====
###  ====
data <- bmi[,c(3:ncol(bmi))]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  rownames_to_column() %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect")) %>%
  column_to_rownames()
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data1 <- data[,c(5,6)]
data1$exposure <- "BMI"

###  ====
data <- whr[,c(3:ncol(whr))]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  rownames_to_column() %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect")) %>%
  column_to_rownames()
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data2 <- data[,c(5,6)]
data2$exposure <- "WHR"

###  ====
data <- bf[,c(3:ncol(bf))]
data$direction <- sapply(1:nrow(data), function(x) ifelse(all(sign(data[x,]) == 1), 1, ifelse(all(sign(data[x,]) == -1), 2, 0)))
data <- data %>%
  rownames_to_column() %>%
  mutate(direction_group = case_when(direction == 1 ~ "Positive effect",
                               direction == 2 ~ "Negative effect",
                               direction == 0 ~ "Opposite effect")) %>%
  column_to_rownames()
data$direction_group <- factor(data$direction_group,
                            levels = c("Positive effect", "Negative effect", "Opposite effect"),ordered = TRUE)
data3 <- data[,c(5,6)]
data3$exposure <- "BF"

# ====
data1 <- cbind(rownames(data1), data.frame(data1, row.names=NULL))
colnames(data1)[1] <- "name"
data2 <- cbind(rownames(data2), data.frame(data2, row.names=NULL))
colnames(data2)[1] <- "name"
data3 <- cbind(rownames(data3), data.frame(data3, row.names=NULL))
colnames(data3)[1] <- "name"
plot_data <- rbind(data1,data2,data3)
plot_data$exposure <- as.factor(plot_data$exposure)
plot_data$exposure <- factor(plot_data$exposure,levels(plot_data$exposure)[c(2,3,1)])
plot_data_interval <- plot_data
data <- subset(plot_data, direction != 0)
interval_bmi_list <- subset(data, exposure == "BMI")
interval_whr_list <- subset(data, exposure == "WHR")
interval_bf_list <- subset(data, exposure == "BF")

# shared metabs ====
data1 <- interval_bmi_list
data2 <- interval_whr_list
data3 <- interval_bf_list
data4 <- inner_join(data2,data1, by = "name")
data5 <- inner_join(data4,data3, by = "name")
interval_shared_list <- as.character(data5$name)
```
Assumptions of no pleiotropy were explored using MR-Egger[@Bowden2015], weighted median[@Burgess2017a] and weighted mode[@Hartwig2017] based estimators. Globally, sensitivity analysis was similar to that of the main analysis for each exposure, though with wider confidence intervals (Appendix \@ref(appendix-MR-sensitivity-analysis)). Confidence intervals for sensitivity analyses tended to cross the null and were widest for MR-Egger, which is unsurprising given the lower power afforded with this model. Sensitivity results for WHR appeared to show most consistency with the main analysis for analyses using both Kettunen and INTERVAL data; confidence intervals for weighted median and mode models did not cross the null in a majority of results for subclasses. When looking at concordance in effect direction across all models (sensitvity and main analysis), consistency in the direction of effect was highest for WHR across both datasets (Figure \@ref(fig:MR-figure-kettunen-sensitivity-directional-consistency). For both datasets, effects for BF were the least consistent. \par

(ref:MR-figure-kettunen-sensitivity-directional-consistency-cap) **Directional consistency of main and sensitivity analyses from two-sample Mendelian randomization results using Kettunen and INTERVAL data**. Plot shows the directional consistency across all 4 models (IVW-MRE, MR-Egger, Weighted Mode, and Weighted median) within each exposure. A positive effect reflects the effect estimate from all four models being in the positive direction; a negative effect reflects betas being in a negative direction; opposite effect reflects different directions for the effect estimates. **A:** Two-sample MR IVW-MRE for 123 metabolites from Kettunen et al. (2016)[@Kettunen2016]; **B:** Two-sample MR IVW-MRE for 230 metabolites from INTERVAL. BMI = body mass index; WHR = waist hip ratio; BF = body fat percentage.

(ref:MR-figure-kettunen-sensitivity-directional-consistency-scap) Directional consistency of main and sensitivity analyses from two-sample Mendelian randomization results using Kettunen and INTERVAL data

```{r MR-figure-kettunen-sensitivity-directional-consistency, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE, out.width='100%', fig.cap='(ref:MR-figure-kettunen-sensitivity-directional-consistency-cap)', fig.scap='(ref:MR-figure-kettunen-sensitivity-directional-consistency-scap)'}
p_kettunen <- ggplot(data = plot_data_kettunen,
       aes(x = exposure)) +
  geom_bar(aes(fill = direction_group))  +
  scale_fill_manual(values = discrete_wes_pal) +
  guides(fill = guide_legend(override.aes = list(size = 5),
                             title = "",
                             label.hjust = 0,
                             label.vjust = 0.5)) +
  xlab("") +
  theme_cowplot() +
  theme(axis.text.x = element_text(angle = 90),
        axis.ticks.x = element_blank(),
        legend.position = "none")

p_interval <- ggplot(data = plot_data_interval,
       aes(x = exposure)) +
  geom_bar(aes(fill = direction_group))  +
  scale_fill_manual(values = discrete_wes_pal) +
  guides(fill = guide_legend(override.aes = list(size = 5),
                             title = "",
                             label.hjust = 0,
                             label.vjust = 0.5)) +
  xlab("") +
  theme_cowplot() +
  theme(axis.text.x = element_text(angle = 90),
        axis.ticks.x = element_blank(),
        legend.position = "none")

legend <- ggplot(data = plot_data_kettunen,
       aes(x = exposure)) +
  geom_bar(aes(fill = direction_group))  +
  scale_fill_manual(values = discrete_wes_pal) +
  guides(fill = guide_legend(override.aes = list(size = 5),
                             title = "",
                             label.hjust = 0,
                             label.vjust = 0.5)) +
  xlab("") +
  theme_cowplot() +
  theme(axis.text.x = element_text(angle = 90),
        axis.ticks.x = element_blank())

legend <- get_legend(legend)

# plot ====
plot_grid(p_kettunen, p_interval, legend, nrow = 1, labels = c("A", "B"))

```

Of these directionally consistent tests, a total of `r length(kettunen_shared_list)` tests were directionally consistent across methods for all three exposures when using the Kettunen data (Figure \@ref(fig:MR-figure-forestplot-kettunen-sensitivity-directional-consistency)). The direction of effect for BF was on the whole opposite to BMI and WHR for all methods. Of these `r length(kettunen_shared_list)` tests, only valine was also found to reach the multiple testing threshold ($0.05/22$) for both BMI and WHR in the main analysis. Sensitivity analysis showed a consistent positive direction of effect with the main analysis for the effect of BMI, WHR, and BF on valine The effect of WHR appeared to show the strongest evidence for an association with valine, with confidence intervals for all models away from the null. When using the INTERVAL data, of the directionally consistent results, a total of `r length(interval_shared_list)` tests were directionally consistent across methods for all three exposures (Figure \@ref(fig:MR-figure-forestplot-interval-sensitivity-directional-consistency)). Of these `r length(interval_shared_list)` tests, valine and Tyrosine were also found to reach the multiple testing threshold ($0.05/28$) for both BMI and WHR in the main analysis where models were consistent. Sensitivity analysis showed a consistent positive direction of effect with the main analysis for the effect of BMI, WHR, and BF on valine and tyrosine. The effect of WHR appeared to show the strongest evidence for an association with valine, with confidence intervals for all models away from the null. \par

(ref:MR-figure-forestplot-kettunen-sensitivity-directional-consistency-cap) **Directionally consistent effects across all two-sample Mendelian randomization models using the Kettunen data**. Effect estimates and 95% confidence intervals for metabolites which showed directionally consistent results across the main (IVW-MRE = inverse variance weighted multiplicative random effects) and sensitivity analysis (MR-Egger, weighted median, and weighted mode) within each exposure. Solid points indicate a multiple testing threshold (`r round(0.05/22,4)`) has been reached. BMI = body mass index; WHR = waist hip ratio; BF = body fat percentage. Names for metabolites are available in Appendix Table \@ref(tab:appendix-MR-table-metabolites).

(ref:MR-figure-forestplot-kettunen-sensitivity-directional-consistency-scap) Directionally consistent effects across all two-sample Mendelian randomization models using the Kettunen data

```{r MR-figure-forestplot-kettunen-sensitivity-directional-consistency, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE, out.width='100%', fig.cap='(ref:MR-figure-forestplot-kettunen-sensitivity-directional-consistency-cap)', fig.scap='(ref:MR-figure-forestplot-kettunen-sensitivity-directional-consistency-scap)'}
knitr::include_graphics("data/MR/figures/forestplot_kettunen_sensitivity_directional_consistency.pdf")
```

(ref:MR-figure-forestplot-interval-sensitivity-directional-consistency-cap) **Directionally consistent effects across all two-sample Mendelian randomization models using the INTERVAL data**. Effect estimates and 95% confidence intervals for metabolites which showed directionally consistent results across the main (IVW-MRE = inverse variance weighted multiplicative random effects) and sensitivity analysis (MR-Egger, weighted median, and weighted mode) within each exposure. Solid points indicate a multiple testing threshold (`r round(0.05/28,4)`) has been reached. BMI = body mass index; WHR = waist hip ratio; BF = body fat percentage. Names for metabolites are available in Appendix Table \@ref(tab:appendix-MR-table-metabolites)

(ref:MR-figure-forestplot-interval-sensitivity-directional-consistency-scap) Directionally consistent effects across all two-sample Mendelian randomization models in the INTERVAL analysis

```{r MR-figure-forestplot-interval-sensitivity-directional-consistency, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE, out.width='100%', fig.cap='(ref:MR-figure-forestplot-interval-sensitivity-directional-consistency-cap)', fig.scap='(ref:MR-figure-forestplot-interval-sensitivity-directional-consistency-scap)'}
knitr::include_graphics("data/MR/figures/forestplot_interval_sensitivity_directional_consistency.pdf")
```

```{r eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
steiger_kettunen <- read.table("../../002_adiposity_metabolites/analysis/step1/combined/001_combined_steiger_kettunen.txt", header = T, sep = "\t")
steiger_interval <- read.table("../../006_interval_metabolites/analysis/combined/001_combined_steiger.txt", header = T, sep = "\t")
```

The causal direction between the exposure and outcomes were assessed using the Steiger test. In the Kettunen data, a total of 123 tests were performed for each exposure, the causal direction of effect from the exposure to the outcome was "true" for `r table(steiger_kettunen$exposure, steiger_kettunen$correct_causal_direction)[15,2]` tests for BMI (i.e., the Steiger test results reflect that a change in the outcome is a consequence of the exposure), and `r table(steiger_kettunen$exposure, steiger_kettunen$correct_causal_direction)[9,2]` tests for WHR. In contrast, the causal direction of effect from the exposure to the outcome was "true" for `r table(steiger_interval$exposure, steiger_interval$correct_causal_direction)[6,2]` tests for BF. For INTERVAL data, a majority of test directions were found to be "true." A total of 230 tests were performed for each exposure, the causal direction of effect from the exposure to the outcome was "true" for `r table(steiger_interval$exposure, steiger_interval$correct_causal_direction)[15,2]` tests for BMI, `r table(steiger_interval$exposure, steiger_interval$correct_causal_direction)[9,2]` tests for WHR, and `r table(steiger_interval$exposure, steiger_interval$correct_causal_direction)[6,2]` tests for BF. \par

#### Additional sensitivity analysis
The effects of heterogeneity in each expopsures instruments were investigated using Cochran's Q statistic for IVW-MRE and MR-Egger models. Heterogeneity of the genetic instruments for each exposure was greater than the degrees of freedom for a majority of metabolites for all three exposures across both the Kettunen and INTERVAL datasets (Table \@ref(tab:MR-table-heterogeneity-N)). When using the Kettunen data, for BMI and WHR all IVW and MR-Egger tests with Q greater than the degrees of freedom overlapped. That is, for each exposure-metabolite pair, Q was greater than the degrees of freedom for the same exposure-metabolite pair in the IVW and MR-Egger tests. For BF, there were 5 tests which did not overlap. When using the INTERVAL data, for BMI 1 test did not overlap, for BF 7 tests did not overlap, and for WHR all tests overlapped. The remaining sensitivity analyses are presented individually for each ouctome dataset. \par 

```{r MR-table-heterogeneity-N, echo=FALSE}
# kettunen ====
## heterogenity ====
bmi_heterogenity <- read.table("../../002_adiposity_metabolites/analysis/step1/BMI_Yengo_941/mr_hetrogeneity_kettunen.txt", header = T, sep = "\t")
colnames(bmi_heterogenity)[6] <- "Q_bmi"
colnames(bmi_heterogenity)[7] <- "Qdf_bmi"
colnames(bmi_heterogenity)[8] <- "Qp_bmi"
whr_heterogenity <- read.table("../../002_adiposity_metabolites/analysis/step1/WHR_Pulit_316/mr_hetrogeneity_kettunen.txt", header = T, sep = "\t")
colnames(whr_heterogenity)[6] <- "Q_whr"
colnames(whr_heterogenity)[7] <- "Qdf_whr"
colnames(whr_heterogenity)[8] <- "Qp_whr"
bf_heterogenity <- read.table("../../002_adiposity_metabolites/analysis/step1/BF_Lu_7/mr_hetrogeneity_kettunen.txt", header = T, sep = "\t")
colnames(bf_heterogenity)[6] <- "Q_bf"
colnames(bf_heterogenity)[7] <- "Qdf_bf"
colnames(bf_heterogenity)[8] <- "Qp_bf"

data <- left_join(bmi_heterogenity, whr_heterogenity, by = c("id.outcome", "method"))
data <- left_join(data, bf_heterogenity, by = c("id.outcome", "method"))
data <- data[,c(3,4,5,6,7,8,12,13,14,18,19,20)]

data$bmi_result <- ifelse(data$Q_bmi > data$Qdf_bmi, 1, 0)
data$whr_result <- ifelse(data$Q_whr > data$Qdf_whr, 1, 0)
data$bf_result <- ifelse(data$Q_bf > data$Qdf_bf, 1, 0)

### methods ====
ivw <- subset(data, method == "Inverse variance weighted")
mregger <- subset(data, method == "MR Egger")
ml <- subset(data, method == "Maximum likelihood")

### table ====
table <- data.frame(exposure = c("BMI", "WHR", "BF"),
                    analysis = c("Kettunen", "Kettunen", "Kettunen"),
                    ML = c(sum(ivw$bmi_result), sum(ivw$whr_result), sum(ivw$bf_result)),
                    IVW = c(sum(mregger$bmi_result), sum(mregger$whr_result), sum(mregger$bf_result)),
                    MR_Egger = c(sum(ml$bmi_result), sum(ml$whr_result), sum(ml$bf_result)))

### overlap ====
overlap <- left_join(ivw,mregger, by = "outcome.x")
overlap_bmi <- overlap[,c("outcome.x", "bmi_result.x", "bmi_result.y")]
overlap_bmi$overlap <- rowSums( overlap_bmi[,2:3] )
overlap_whr <- overlap[,c("outcome.x", "whr_result.x", "whr_result.y")]
overlap_whr$overlap <- rowSums( overlap_whr[,2:3] )
overlap_bf <- overlap[,c("outcome.x", "bf_result.x", "bf_result.y")]
overlap_bf$overlap <- rowSums( overlap_bf[,2:3] )

# interval ====
## heterogenity ====
bmi_heterogenity <- read.table("../../006_interval_metabolites/analysis/BMI_Yengo_941/mr_hetrogeneity.txt", header = T, sep = "\t")
colnames(bmi_heterogenity)[6] <- "Q_bmi"
colnames(bmi_heterogenity)[7] <- "Qdf_bmi"
colnames(bmi_heterogenity)[8] <- "Qp_bmi"
whr_heterogenity <- read.table("../../006_interval_metabolites/analysis/WHR_Pulit_316/mr_hetrogeneity.txt", header = T, sep = "\t")
colnames(whr_heterogenity)[6] <- "Q_whr"
colnames(whr_heterogenity)[7] <- "Qdf_whr"
colnames(whr_heterogenity)[8] <- "Qp_whr"
bf_heterogenity <- read.table("../../006_interval_metabolites/analysis/BF_Lu_7/mr_hetrogeneity.txt", header = T, sep = "\t")
colnames(bf_heterogenity)[6] <- "Q_bf"
colnames(bf_heterogenity)[7] <- "Qdf_bf"
colnames(bf_heterogenity)[8] <- "Qp_bf"

data <- left_join(bmi_heterogenity, whr_heterogenity, by = c("outcome", "method"))
data <- left_join(data, bf_heterogenity, by = c("outcome", "method"))
data <- data[,c(3,4,5,6,7,8,12,13,14,18,19,20)]

data$bmi_result <- ifelse(data$Q_bmi > data$Qdf_bmi, 1, 0)
data$whr_result <- ifelse(data$Q_whr > data$Qdf_whr, 1, 0)
data$bf_result <- ifelse(data$Q_bf > data$Qdf_bf, 1, 0)

### methods ====
ivw <- subset(data, method == "Inverse variance weighted")
mregger <- subset(data, method == "MR Egger")
ml <- subset(data, method == "Maximum likelihood")

### table ====
table1 <- data.frame(exposure = c("BMI", "WHR", "BF"),
                     analysis = c("INTERVAL", "INTERVAL", "INTERVAL"),
                    ML = c(sum(ivw$bmi_result), sum(ivw$whr_result), sum(ivw$bf_result)),
                    IVW = c(sum(mregger$bmi_result), sum(mregger$whr_result), sum(mregger$bf_result)),
                    MR_Egger = c(sum(ml$bmi_result), sum(ml$whr_result), sum(ml$bf_result)))
data <- rbind(table,table1)
data <- data[,c("exposure", "analysis", "IVW", "MR_Egger")]

### overlap ====
overlap <- left_join(ivw,mregger, by = "outcome")
overlap_bmi <- overlap[,c("outcome", "bmi_result.x", "bmi_result.y")]
overlap_bmi$overlap <- rowSums( overlap_bmi[,2:3] )
overlap_whr <- overlap[,c("outcome", "whr_result.x", "whr_result.y")]
overlap_whr$overlap <- rowSums( overlap_whr[,2:3] )
overlap_bf <- overlap[,c("outcome", "bf_result.x", "bf_result.y")]
overlap_bf$overlap <- rowSums( overlap_bf[,2:3] )

# table ====
knitr::kable(data, longtable = T, booktabs = T, format = "latex", escape = F,
    caption = 'Number of tests in which Cochran`s Q exceeds genetic instrumental variable degrees of freedom', row.names = F,
    col.names = c("", "", "IVW", "MR-Egger")) %>%
    collapse_rows(columns = 2) %>%
  kable_styling(latex_options = c("striped", "HOLD_position"), position = "center") %>%
  kableExtra::footnote(general = "Table gives the number of tests for each exposure in which heteregoneity, measured by Cochrans Q, was greater than the degrees of freedom (number of SNPs - 1) for each exposure. That is, if BMI has 941 SNPs then the degrees of freedom is 940, and if Cochrans Q for the effect of BMI on metabolite 1 is 1000 then there is evidence of heterogeneity in the genetic instruments in relation to that test. Total number of exposure-outcome tests for Kettunen and INTERVAL data was 123 and 230 respectively. IVW = Inverse variance weighted method; BMI = body mass idnex; WHR = waist hip ratio; BF = body fat percentage", threeparttable = T, general_title = "", footnote_as_chunk = T)
```

##### Analysis using Kettunen data:
```{r eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
library(data.table)
## single snp ====
### bmi
data <- read.table("../../002_adiposity_metabolites/analysis/step1/BMI_Yengo_941/singlesnp_summary_kettunen.txt", header = T, sep = "\t")
bmi_high_kettunen <- subset(data, median_b >= quantile(abs(data$median_b), probs = 0.95)[[1]])
bmi_low_kettunen <- subset(data, median_b <= quantile(abs(data$median_b), probs = 0.05)[[1]])

### whr
data <- read.table("../../002_adiposity_metabolites/analysis/step1/WHR_Pulit_316/singlesnp_summary_kettunen.txt", header = T, sep = "\t")
whr_high_kettunen <- subset(data, median_b >= quantile(abs(data$median_b), probs = 0.95)[[1]])
whr_low_kettunen <- subset(data, median_b <= quantile(abs(data$median_b), probs = 0.05)[[1]])

### bf
data <- read.table("../../002_adiposity_metabolites/analysis/step1/BF_Lu_7/singlesnp_summary_kettunen.txt", header = T, sep = "\t")
bf_high_kettunen <- subset(data, median_b >= quantile(abs(data$median_b), probs = 0.95)[[1]])
bf_low_kettunen <- subset(data, median_b <= quantile(abs(data$median_b), probs = 0.05)[[1]])
```
In single SNP MR using Kettunen data, visual inspection of forest plots showed $S$ shaped distributions of effect estimates for all tests (Schematic representative Figure \@ref(fig:MR-figure-kettunen-singlesnp-representative-figure)). Effect estimates for some SNPs in the single SNP MR analysis appeared to be outliers. For example, for BMI on glycoproteins, rs4673553 showed a disproportionately larger effect estimate of 22 SD units increase per normalised SD higher BMI (Figure \@ref(fig:MR-figure-kettunen-singlesnp-outlier); standard error = 0.85; p-value = 5.66 x 10^-148^) when compared to other SNPs. \par

(ref:MR-figure-kettunen-singlesnp-representative-figure-cap) **Schematic representative figure: Single-SNP MR analysis of the effect of adiposity on a metabolite**. In single-SNP MR analysis, the effect of each SNP on the outcome is investigated. In these analyses, SNP effects are organised from largest to smallest. Here, the figure shows an $S$ shaped distribution of effects which is representative of the single-SNP MR analyses performed for Kettunen and INTERVAL data.

(ref:MR-figure-kettunen-singlesnp-representative-figure-scap) Schematic representative figure: Single-SNP MR analysis of the effect of adiposity on a metabolite

```{r MR-figure-kettunen-singlesnp-representative-figure, echo=FALSE, out.width='50%', fig.align='center',  fig.cap='(ref:MR-figure-kettunen-singlesnp-representative-figure-cap)', fig.scap='(ref:MR-figure-kettunen-singlesnp-representative-figure-scap)'}
knitr::include_graphics("data/MR/figures/schemeatic_single_snp.pdf")
```

(ref:MR-figure-kettunen-singlesnp-outlier-cap) **Schematic representative figure: Single-SNP MR analysis of the effect of adiposity on a metabolite**. In single-SNP MR analysis, the effect of each SNP on the outcome is investigated. In these analyses, SNP effects are organised from largest to smallest. Here, the figure shows an $S$ shaped distribution of effects which is representative of the single-SNP MR analyses performed for Kettunen and INTERVAL data. In addition, a single SNP is shown to have a much larger effect than the others.

(ref:MR-figure-kettunen-singlesnp-outlier-scap) Schematic representative figure: Leave-one-out MR analysis of the effect of adiposity on a metabolite

```{r MR-figure-kettunen-singlesnp-outlier, echo=FALSE, out.width='50%', fig.align='center', fig.cap='(ref:MR-figure-kettunen-singlesnp-outlier-cap)', fig.scap='(ref:MR-figure-kettunen-singlesnp-outlier-scap)'}
knitr::include_graphics("data/MR/figures/schemeatic_loo.pdf")
```

When looking at the median effect size across all metabolites for each SNP, a number of SNPs showed larger median effect sizes across a majority of metabolites. The total number of SNPs with the largest positive (effect sizes in the top 95%) and largest negative (effect sizes in the bottom 5%) effect sizes were: BMI = `r nrow(bmi_low_kettunen)` (5%) and `r nrow(bmi_high_kettunen)` (95%); WHR = `r nrow(whr_low_kettunen)` (5%) and `r nrow(whr_high_kettunen)` (95%); BF = `r nrow(bf_low_kettunen)` (5%) and `r nrow(bf_high_kettunen)` (95%). Funnel plots did not however highlight outlying SNPs. Instead, funnel plots were reflective of some SNPs having larger effect estimates more broadly (Representative Figure \@ref(fig:MR-figure-kettunen-funnelplot-BMI-representative-figure)). The low number of SNPs used for BF did not result in meaningfully interpretable funnel plots (Representative Figure \@ref(fig:MR-figure-kettunen-funnelplot-BF-representative-figure)). \par

(ref:MR-figure-kettunen-funnelplot-BMI-representative-figure-cap) **Representative figure: Funnel plot of of BMI on one metabolite using Kettunen data**. Funnel plot shows the results of a single SNP MR with effect estimate and standard error. Asymmetry in the funnel indicates unreliable effect estimates. The representative plot illustrates a SNP (bottom left) with a larger effect estimate.

(ref:MR-figure-kettunen-funnelplot-BMI-representative-figure-scap) Representative figure: Funnel plot of of BMI on one metabolite using Kettunen data

```{r MR-figure-kettunen-funnelplot-BMI-representative-figure, echo=FALSE, out.width='100%', fig.cap='(ref:MR-figure-kettunen-funnelplot-BMI-representative-figure-cap)', fig.scap='(ref:MR-figure-kettunen-funnelplot-BMI-representative-figure-scap)'}
knitr::include_graphics("data/MR/figures/funnelplot_BMI_representative_figure_kettunen.pdf")
```

(ref:MR-figure-kettunen-funnelplot-BF-representative-figure-cap) **Representative figure: Funnel plot of of BF on one metabolite using Kettunen data**. Funnel plot shows the results of a single SNP MR with effect estimate and standard error. Asymmetry in the funnel indicates unreliable effect estimates.

(ref:MR-figure-kettunen-funnelplot-BF-representative-figure-scap) Representative figure: Funnel plot of of BF on one metabolite using Kettunen data

```{r MR-figure-kettunen-funnelplot-BF-representative-figure, echo=FALSE, out.width='100%', fig.cap='(ref:MR-figure-kettunen-funnelplot-BF-representative-figure-cap)', fig.scap='(ref:MR-figure-kettunen-funnelplot-BF-representative-figure-scap)'}
knitr::include_graphics("data/MR/figures/funnelplot_BF_representative_figure_kettunen.pdf")
```

Although a number of SNPs showed disproportionately larger effect sizes, in leave-one-out analysis, visual inspection of forest plots showed that no single SNP altered the direction of effect for any metabolite across exposures. For BF, confidence intervals for one or more SNPs crossed the null for every metabolite tested (Representative Figure \@ref(fig:MR-figure-kettunen-leaveoneout-BF-representative-figure)). This was not the case for BMI and WHR, where for many metabolites confidence intervals did not cross the null for any SNPs (Representative Figure \@ref(fig:appendix-MR-figure-kettunen-leaveoneout-BMI-representative-figure)). \par

(ref:MR-figure-kettunen-leaveoneout-BF-representative-figure-cap) **Representative figure: Leave-one-out MR analysis of BF on acetoacetate using Kettunen data**. A leave-one-out analysis performs an MR of exposure on outcome for all SNPs excluding a different SNP each time. Forest plot shows the effect estimate and 95% confidence interval for each SNP exclusion on acetoacetate. Effect estimates represent the change in the inverse rank position of each metabolite per change in the inverse rank position of the exposure.

(ref:MR-figure-kettunen-leaveoneout-BF-representative-figure-scap) Representative figure: Leave-one-out MR analysis of BF on acetoacetate using Kettunen data

```{r MR-figure-kettunen-leaveoneout-BF-representative-figure, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE, out.width='100%', fig.cap='(ref:MR-figure-kettunen-leaveoneout-BF-representative-figure-cap)', fig.scap='(ref:MR-figure-kettunen-leaveoneout-BF-representative-figure-scap)'}
knitr::include_graphics("data/MR/figures/leaveoneout_BF_representative_figure_kettunen.pdf")
```


##### Analysis using INTERVAL data:
```{r eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
library(data.table)
## single snp ====
### bmi
data <- read.table("../../006_interval_metabolites/analysis/BMI_Yengo_941/singlesnp_summary.txt", header = T, sep = "\t")
bmi_high_interval <- subset(data, median_b >= quantile(abs(data$median_b), probs = 0.95)[[1]])
bmi_low_interval <- subset(data, median_b <= quantile(abs(data$median_b), probs = 0.05)[[1]])

### whr
data <- read.table("../../006_interval_metabolites/analysis/WHR_Pulit_316/singlesnp_summary.txt", header = T, sep = "\t")
whr_high_interval <- subset(data, median_b >= quantile(abs(data$median_b), probs = 0.95)[[1]])
whr_low_interval <- subset(data, median_b <= quantile(abs(data$median_b), probs = 0.05)[[1]])

### bf
data <- read.table("../../006_interval_metabolites/analysis/BF_Lu_7/singlesnp_summary.txt", header = T, sep = "\t")
bf_high_interval <- subset(data, median_b >= quantile(abs(data$median_b), probs = 0.95)[[1]])
bf_low_interval <- subset(data, median_b <= quantile(abs(data$median_b), probs = 0.05)[[1]])

# comparison ====
bmi_high <- inner_join(bmi_high_kettunen, bmi_high_interval, by = "SNP")
bmi_low <- inner_join(bmi_low_kettunen, bmi_low_interval, by = "SNP")

whr_high <- inner_join(whr_high_kettunen, whr_high_interval, by = "SNP")
whr_low <- inner_join(whr_low_kettunen, whr_low_interval, by = "SNP")

bf_high <- inner_join(bf_high_kettunen, bf_high_interval, by = "SNP")
bf_low <- inner_join(bf_low_kettunen, bf_low_interval, by = "SNP")


```
Broadly speaking, additional sensitivity analyses using INTERVAL data were similar to that of the additional sensitivity analyses using the Kettunen data. In single SNP MR visual inspection of forest plots showed $S$ shaped distributions of effect estimates for all tests (Representative Figure \@ref(fig:appendix-MR-figure-interval-singlesnp-representative-figure), figure also shows outlier SNP with effect estimate close to -6). As with the Kettunen data, effect estimates for some SNPs in the single SNP MR analysis were much greater than others (See Representative Figure \@ref(fig:appendix-MR-figure-interval-singlesnp-representative-figure)). \par

When looking at the median effect size across all metabolites for each SNP, a number of SNPs showed larger median effect sizes across a majority of metabolites. The total number of SNPs with the largest positive (effect sizes in the top 95%) and largest negative (effect sizes in the bottom 5%) effect sizes were: BMI = `r nrow(bmi_low_interval)` (5%) and `r nrow(bmi_high_interval)` (95%); WHR = `r nrow(whr_low_interval)` (5%) and `r nrow(whr_high_interval)` (95%); BF = `r nrow(bf_low_interval)` (5%) and `r nrow(bf_high_interval)` (95%). Many of the SNPs identified as having larger effects are shared across both the Kettunen and INTERVAL datasets: BMI = `r nrow(bmi_low)` (5%) and `r nrow(bmi_high)` (95%); WHR = `r nrow(whr_low)` (5%) and `r nrow(whr_high)` (95%); BF = `r nrow(bf_low)` (5%) and `r nrow(bf_high)` (95%). As an example, for BF, more often than not, rs6857 showed an effect estimate greater than the 6 other SNPs and did not overlap confidence intervals with them or the null. rs6857 was found in both the Kettunen and INTERVAL data to have a disproportionately larger effect estimate than other SNPs. For BMI and WHR, SNPs with disproportionally larger effect sizes tended to have confidence intervals which overlapped other SNPs. The degree of overlap was minimal however and mostly at the tail-end of the confidence interval. \par

Funnel plots did not highlight outlying SNPs, but did reflect some SNPs having larger effect estimates across the board (Representative Figure \@ref(fig:appendix-MR-figure-interval-funnelplot-BMI-representative-figure)). The low number of SNPs used for BF did not result in meaningfully interpretable funnel plots (Representative Figure \@ref(fig:appendix-MR-figure-interval-funnelplot-Bf-representative-figure)). In leave-one-out analysis, visual inspection of forest plots showed that no single SNP altered the direction of effect for any metabolite across exposures. For BF, confidence intervals for one or more SNPs crossed the null for a majority of metabolite tested (Representative Figure \@ref(fig:appendix-MR-figure-interval-leaveoneout-BF-representative-figure)). This was not the case for BMI and WHR, where for many metabolites confidence intervals did not cross the null for any SNPs (Representative Figure \@ref(fig:appendix-MR-figure-interval-leaveoneout-BMI-representative-figure)). \par


#### Additional analyses
##### Additional exposures
A number of additional SNP lists were used to instrument BMI, WHR, and BF to explore the validity of the instruments used in the main analyses. Additional SNP lists for BMI and WHR were selected on the basis that they did not contain UK Biobank individuals, which were included in SNP lists for the main analysis. For BF, an additional SNP list which did contain UK Biobank individuals was chosen. For BMI and WHR, additional SNP lists contained fewer SNPs than the SNP list used in the main analysis. For BF, the additional SNP list contained more SNPs than the main analysis. Each SNP list explained a different amount of variation in its respective trait in comparison the the SNP list used in the main analysis. All details on the additional SNP lists are presented in the Appendix (\@ref(appendix-MR-exposures)). \par

All analyses, including sensitivity analyses, were repeated for these additional SNP lists. Focus here is on the Kettunen data as results were similar for the INTERVAL data. A table of all SNP lists can be found on [GitHub](https://github.com/mattlee821/000_thesis/blob/master/index/data/MR/tables/exposure_SNPs.txt). \par

```{r eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
# packages ====
library(tidyr)
library(dplyr)
# data ==== 
data <- read.table("../../002_adiposity_metabolites/analysis/step1/combined/001_combined_mr_results_kettunen.txt", header = T, sep ="\t")
data <- subset(data, method == "Inverse variance weighted (multiplicative random effects)")

## bmi
a <- c("BMI_Yengo_941", "BMI_Yengo_656", "BMI_Locke_77")
data1 <- data[data$exposure %in% a, ]
data1 <- data1[,c(3,4,7)]
data1 <- spread(data1, exposure, b)
data1 <- data1[,c(2:4)]
bmi_cor <- cor(data1, use = "complete.obs", method = "spearman")

## whr
a <- c("WHR_Pulit_316", "WHR_Shungin_26")
data1 <- data[data$exposure %in% a, ]
data1 <- data1[,c(3,4,7)]
data1 <- spread(data1, exposure, b)
data1 <- data1[,c(2:3)]
whr_cor <- cor(data1, use = "complete.obs", method = "spearman")

## bf
a <- c("BF_Lu_7", "BF_Lu_5", "BF_Hubel_76")
data1 <- data[data$exposure %in% a, ]
data1 <- data1[,c(3,4,7)]
data1 <- spread(data1, exposure, b)
data1 <- data1[,c(2:4)]
bf_cor <- cor(data1, use = "complete.obs", method = "spearman")
```

Results from additional instruments for BMI showed broadly larger effect estimates but consistent directions of effect across metabolites (Appendix Figure \@ref(fig:appendix-MR-figure-kettunen-circosplot-additional-BMI)). For the BMI SNPs obtained from a non-UK Biobank GWAS, effect estimates had much wider confidence intervals. Spearman's Rho correlation of MR results was highest between the two SNP lists (N SNP = 941 and 656) from Yengo et al. (2018) (`r round(bmi_cor[[2,3]],2)`). Correlation between the Locke et al (2014) SNP list (N SNP = 77) and the COJO SNP list from Yengo et al (N SNP = 941) (`r round(bmi_cor[[1,3]],2)`) and the non-COJO SNP list from Yengo et al. (N SNP = 656) (`r round(bmi_cor[[1,2]],2)`) were also high. For WHR the pattern of association was similar between both the main analysis (N SNP = 316) and analysis using the additional SNP list from Shungin et al. (2014; N SNP = 26) (Figure \@ref(fig:appendix-MR-kettunen-figure-circosplot-additional-BMI)) with high correlation between MR results (`r round(whr_cor[[1,2]],2)`). Effect estimates were larger, however confidence intervals were wider and crossed the null more often when using the additional SNP list from Shungin et al compare to the main analysis. \par

For BF, there was considerable similarity between the main analysis and the additional analysis when using SNPs from Lu et al (2016) which did not include two SNPs previously identified as being associated with 'favourable adiposity' (Figure \@ref(fig:appendix-MR-kettunen-figure-circosplot-additional-BF)). More tests reached the multiple testing threshold when using the 5 SNPs from Lu et al., as opposed to the full 7 SNPs, this included associations with Apolipoprotein A1, Phenylalanine, Tyrosine, Glucose, and Cholesterol esters in very large HDL. For the additional analysis, which used 76 SNPs from Hubel et al (2016), MR results were considerably smaller and appeared to show conflicting directions of effect with that of the Lu et al. (2016) SNPs (both using 7 and 5 SNPs). Confidence intervals were much tighter and two  metabolites (phenylalanine and glycoprotein acetyls) reached the multiple testing threshold. Correlation between the two Lu et al (2016) SNP lists was high (`r round(bf_cor[[3,2]],2)`), however both the 5 (`r round(bf_cor[[1,3]],2)`) and 7 (`r round(bf_cor[[1,2]],2)`) SNP lists from Lu et al. (2016) showed weaker inverse correlations with the SNP list from Hubel et al (2016). \par

Steiger directionality tests were variable across analyses (Kettunen vs. INTERVAL) and instrument lists. Variability between instrument lists is perhaps unsurprising given that a "true" causal direction from the exposure to the outcome is identified when the SNP variance explained is greater in the exposure than in the outcome. When using large instrument lists, as the number of SNPs increases the proportion of variance explained by each SNP will decrease. A result of this will be that many SNPs will explain a very low amount of the variance in the exposure, this will increase the likelihood of that SNP explaining a larger proportion of variance in the outcome phenotype thus resulting in the "true" causal direction being from the outcome to the exposure. \par

In the main analysis using BMI with 941 SNPs and the Kettunen data, 0 tests were shown to reflect the "true" causal direction of exposure to outcome. This did not change after clumping. However, when using the non-COJO SNPs (N SNP = 656) from the same study, 4 tests were found to reflect the "true" causal direction. This increased to 5 tests after clumping. Additionally, when using 77 SNPs from Locke et al. (2014)[@Locke2015] to instrument BMI a total of 76 tests were shown to reflect the "true" causal direction. This increased to 79 tests after clumping. There was no difference in the non-clumped (N SNP = 316) and clumped SNP list for WHR used in the main analysis (N test that reflect the "true" causal direction = 4). However, when using the 26 SNP instrument from Shungin et al. (2015)[@Shungin2015], 102 and 112 tests were found to reflect the "true" causal direction for the non-clumped and clumped SNP lists. For BF, the 7 SNP instrument resulted in 80 true tests while the 5 SNP instrument resulted in 76 true tests. The BF GWAS from Hubel et al. (2019)[@Hubel2019] did not have an N available for the summary statistics, instead the total N for the GWAS (155,961) was used. All 123 tests were found to reflect the "true" causal direction when using both the non-clumped (N SNP = 76) and clumped instruments from Hubel et al. (2016) for BF. \par

When using the INTERVAL data, a different picture was found, with a majority of tests shown to reflect the "true" causal direction across all instrument lists. The exception was when instrumenting WHR with SNPs from Pulit et al. (2019)[@Pulit2019]. After clumping, the number of tests that reflected the "true" causal direction for WHR reduced from 91 to 75. There was also little difference in the number tests that reflected the "true" causal direction when instrumenting BMI using 77 SNPs from Locke et al. compared to the 941 SNPs used in the main analysis. As with the Kettunen data, all tests reflected a "true" causal direction when instrumenting BF using the 76 SNPs from Hubel et al. \par

##### Clumped exposures
```{r eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
# packages ====
library(tidyr)
library(dplyr)

# data ==== 
data <- read.table("../../002_adiposity_metabolites/data/exposure_SNPs.txt", header = T, sep ="\t")
data1 <- subset(data, Ref == "Yengo")
```
All SNP lists used as instruments, including those described above, underwent clumping and all analyses were repeated. Clumping resulted in the removal of the following number of SNPs due to LD (R^2^ $\geq$ 0.001) with other variants or absence from the LD reference panel: BMI Locke et al. (2014) = `r table(data$Clumped, data$Ref)[[2,2]]`, BMI Yengo et al. (2018) using COJO SNPs = `r table(data1$Clumped, data1$Note)[[2,1]]`, BMI Yengo et al. (2018) using non-COJO SNPs = `r table(data1$Clumped, data1$Note)[[2,2]]`, WHR Pulit et al. (2018) = `r table(data$Clumped, data$Ref)[[2,4]]`, WHR Shungin et al. (2014) = `r table(data$Clumped, data$Ref)[[2,5]]`, BF Hubel et al. (2018) = `r table(data$Clumped, data$Ref)[[2,1]]`. No SNPs were removed due to clumping for BF from Lu et al. (2016). All SNPs, including whether they were removed due to clumping, are presented in the Appendix (Table \@ref(tab:appendix-MR-table-exposures)). \par

```{r eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
# packages ====
library(tidyr)
library(dplyr)

# data ==== 
data <- read.table("../../002_adiposity_metabolites/analysis/step1/combined/001_combined_mr_results_kettunen.txt", header = T, sep ="\t")
data <- subset(data, method == "Inverse variance weighted (multiplicative random effects)")

# bmi
a <- c("BMI_Yengo_941", "BMI_Yengo_941_clump", "BMI_Yengo_656", "BMI_Yengo_656_clump", "BMI_Locke_77", "BMI_Locke_77_clump")
data1 <- data[data$exposure %in% a, ]
data1 <- data1[,c("outcome", "exposure", "b")]
data1 <- spread(data1, exposure, b)
data1 <- data1[,c(2:7)]
bmi_cor <- cor(data1, use = "complete.obs", method = "spearman")

# whr
a <- c("WHR_Pulit_316", "WHR_Pulit_316_clump", "WHR_Shungin_26", "WHR_Shungin_26_clump")
data1 <- data[data$exposure %in% a, ]
data1 <- data1[,c("outcome", "exposure", "b")]
data1 <- spread(data1, exposure, b)
data1 <- data1[,c(2:5)]
whr_cor <- cor(data1, use = "complete.obs", method = "spearman")

# bf
a <- c("BF_Hubel_76", "BF_Hubel_76_clump")
data1 <- data[data$exposure %in% a, ]
data1 <- data1[,c("outcome", "exposure", "b")]
data1 <- spread(data1, exposure, b)
data1 <- data1[,c(2:3)]
bf_cor <- cor(data1, use = "complete.obs", method = "spearman")
```
When using the Kettunen data, correlation for BMI results between the Yengo COJO (`r round(bmi_cor[[5,6]],2)`), non-COJO (`r round(bmi_cor[[3,4]],2)`), and Locke (`r round(bmi_cor[[1,2]],2)`) non-clumped and clumped MR results was high. Similarly, for WHR MR results from non-clumped and clumped analyses correlation was high for the main exposure (Pulit et al. (2018) = `r round(whr_cor[[1,2]],2)`) and for the additional exposure (`r round(whr_cor[[3,4]],2)`). For BF, clumping was not possible for the main exposure, however correlation between the non-clumped and clumped SNP list from Hubel et al. (2018) was high (`r round(bf_cor[[1,2]],2)`)

```{r eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
# packages ====
library(tidyr)
library(dplyr)

# data ==== 
data <- read.table("../../006_interval_metabolites/analysis/combined/001_combined_mr_results.txt", header = T, sep ="\t")
data <- subset(data, method == "Inverse variance weighted (multiplicative random effects)")

# bmi
a <- c("BMI_Yengo_941", "BMI_Yengo_941_clump", "BMI_Yengo_656", "BMI_Yengo_656_clump", "BMI_Locke_77", "BMI_Locke_77_clump")
data1 <- data[data$exposure %in% a, ]
data1 <- data1[,c("outcome", "exposure", "b")]
data1 <- spread(data1, exposure, b)
data1 <- data1[,c(2:7)]
bmi_cor <- cor(data1, use = "complete.obs", method = "spearman")

# whr
a <- c("WHR_Pulit_316", "WHR_Pulit_316_clump", "WHR_Shungin_26", "WHR_Shungin_26_clump")
data1 <- data[data$exposure %in% a, ]
data1 <- data1[,c("outcome", "exposure", "b")]
data1 <- spread(data1, exposure, b)
data1 <- data1[,c(2:5)]
whr_cor <- cor(data1, use = "complete.obs", method = "spearman")

# bf
a <- c("BF_Hubel_76", "BF_Hubel_76_clump")
data1 <- data[data$exposure %in% a, ]
data1 <- data1[,c("outcome", "exposure", "b")]
data1 <- spread(data1, exposure, b)
data1 <- data1[,c(2:3)]
bf_cor <- cor(data1, use = "complete.obs", method = "spearman")
```
When using the INTERVAL data, correlation for BMI results between the Yengo COJO (`r round(bmi_cor[[5,6]],2)`), non-COJO (`r round(bmi_cor[[3,4]],2)`), and Locke (`r round(bmi_cor[[1,2]],2)`) non-clumped and clumped MR results was high. Similarly, for WHR MR results from non-clumped and clumped analyses correlation was high for the main exposure (Pulit et al. (2018) = `r round(whr_cor[[1,2]],2)`) and for the additional exposure (`r round(whr_cor[[3,4]],2)`). For BF, clumping was not possible for the main exposure, however correlation between the non-clumped and clumped SNP list from Hubel et al. (2018) was high (`r round(bf_cor[[1,2]],3)`). \par

### Meta-analysis of two-sample Mendelian randomization results
```{r eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
data <- read.table("../../006_interval_metabolites/data/meta_analysis_p_results.txt", header = T, sep = "\t")
# directions ====
pos <- subset(data, meta_direction == "+ +")
neg <- subset(data, meta_direction == "- -")
pos_bmi <- subset(pos, exposure == "BMI_Yengo_941")
neg_bmi <- subset(neg, exposure == "BMI_Yengo_941")
pos_whr <- subset(pos, exposure == "WHR_Pulit_316")
neg_whr <- subset(neg, exposure == "WHR_Pulit_316")
pos_bf <- subset(pos, exposure == "BF_Lu_7")
neg_bf <- subset(neg, exposure == "BF_Lu_7")
# sig and directions ====
data_sig <- subset(data, p <= 0.05/110)
sig_pos <- subset(data_sig, meta_direction == "+ +")
sig_neg <- subset(data_sig, meta_direction == "- -")
sig_bmi <- subset(data_sig, exposure == "BMI_Yengo_941")
sig_whr <- subset(data_sig, exposure == "WHR_Pulit_316")
sig_bf <- subset(data_sig, exposure == "BF_Lu_7")
sig_pos_bmi <- subset(sig_pos, exposure == "BMI_Yengo_941")
sig_neg_bmi <- subset(sig_neg, exposure == "BMI_Yengo_941")
sig_pos_whr <- subset(sig_pos, exposure == "WHR_Pulit_316")
sig_neg_whr <- subset(sig_neg, exposure == "WHR_Pulit_316")
sig_pos_bf <- subset(sig_pos, exposure == "BF_Lu_7")
sig_neg_bf <- subset(sig_neg, exposure == "BF_Lu_7")
sig_pos_bmi_whr <- rbind(sig_pos_bmi, sig_pos_whr)
sig_pos_bmi_whr <- unique(sig_pos_bmi_whr$outcome)
sig_neg_bmi_whr <- rbind(sig_neg_bmi, sig_neg_whr)
sig_neg_bmi_whr <- unique(sig_neg_bmi_whr$outcome)
# table ====
table <- data[data$outcome %in% sig_pos_bmi_whr, ]
table$direction <- "+"
table1 <- data[data$outcome %in% sig_neg_bmi_whr, ]
table1$direction <- "-"
table <- rbind(table,table1)
table <- table[!duplicated(table$outcome),]
table <- table[,c("raw.label", "class", "subclass", "direction")]
table <- table[order(table$class, table$subclass, table$raw.label),]
```

In total, 110 metabolites were shared across the Kettunen and INTERVAL metabolite data Meta-analysis of p-value was performed using Fisher's method (Equation \@ref(eq:fishers-method)). Across all 3 exposures (330 tests), a total of `r nrow(pos)` tests had a positive direction of effect when using the Kettunen and INTERVAL data, `r nrow(neg)` tests had a negative direction of effect. For BMI, `r nrow(pos_bmi)` metabolites had positive directions of effect and `r nrow(neg_bmi)` metabolites had negative directions of effect. Similar results were found for WHR (positive = `r nrow(pos_whr)`; negative = `r nrow(neg_whr)`). For BF, a larger number of metabolites had consistent negative directions of effect (N = `r nrow(neg_bf)`) than positive (N = `r nrow(pos_bf)`). \par

Across the 330 tests, a total of `r nrow(data_sig)` tests reached a Bonferroni ($0.05/110$) multiple testing threshold. Of these, `r nrow(sig_pos)` tests had positive directions of effect and `r nrow(sig_neg)` tests had negative directions of effect when using the Kettunen and INTERVAL data. For BMI, `r nrow(sig_pos_bmi)` metabolites had consistent positive directions of effect and met the multiple testing threshold, while `r nrow(sig_neg_bmi)` metabolites had consistent negative directions of effect and met the multiple testing threshold. Similar results were found for WHR (positive = `r nrow(sig_pos_whr)`; negative = `r nrow(sig_neg_whr)`). For BF, no metabolites reached the multiple testing threshold. \par

<!--
\newpage
\thispagestyle{empty}

(ref:MR-figure-circosplot-meta-analysis-pval-cap) **Meta-analysis -log10 p-value results for 110 NMR derived metabolites shared between Kettunen and INTERVAL datasets.**. Circos plot shows each track as one measures of adiposity; the outer track is body mass index (BMI), the middle track is waist hip ratio (WHR), the inner track is body fat percentage (BF). The lines give the -log10 p-value for the meta-analysis results for each adiposity measure on each metabolite. Metabolites are grouped by subclass and arranged alphabetically within each subclass.

(ref:MR-figure-circosplot-meta-analysis-pval-scap) Meta-analysis -log10 p-value results for 110 NMR derived metabolites shared between Kettunen and INTERVAL datasets

```{r MR-figure-circosplot-meta-analysis-pval, echo=FALSE, out.width='120%', fig.align='center', fig.cap='(ref:MR-figure-circosplot-meta-analysis-pval-cap)', fig.scap='(ref:MR-figure-circosplot-meta-analysis-pval-scap)'}
knitr::include_graphics("data/MR/figures/circosplot_metaanalysis_pval.pdf")
```
-->

Across both BMI and WHR, a total of `r length(sig_pos_bmi_whr)` metabolites had a consistent positive direction of effect and met the multiple testing threshold, while `r length(sig_neg_bmi_whr)` metabolites had a consistent negative direction of effect and reached the multiple testing threshold. As such, a total of `r nrow(table)` metabolites were associated with BMI or WHR in meta-analysis. Table \@ref(tab:MR-table-meta-analysis-sig-direction) gives a list of all `r nrow(table)` metabolites associated with BMI and WHR along with their directions of effect, metabolites are ordered alphabetically by class, subclass, and then metabolite label, and are presented with their originally measured units. \par

<!--
```{r MR-table-meta-analysis-associations-N, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
table1 <- data.frame(group = c("Consistent total", "Consistent +", " Consistent -", "Associated total", "Associated +", "Associated -"),
                     Total = c(nrow(pos) + nrow(neg), nrow(pos), nrow(neg), 
                     nrow(data_sig), nrow(sig_pos), nrow(sig_neg)),
                     BMI = c(nrow(pos_bmi) + nrow(neg_bmi), nrow(pos_bmi), nrow(neg_bmi),
                     nrow(sig_bmi), nrow(sig_pos_bmi), nrow(sig_neg_bmi)),
                     WHR = c(nrow(pos_whr) + nrow(neg_whr), nrow(pos_whr), nrow(neg_whr),
                     nrow(sig_whr), nrow(sig_pos_whr), nrow(sig_neg_whr)),
                     BF = c(nrow(pos_bf) + nrow(neg_bf), nrow(pos_bf), nrow(neg_bf),
                     nrow(sig_bf), nrow(sig_pos_bf), nrow(sig_neg_bf)))
knitr::kable(table1, longtable = T, booktabs = T, format = "latex", escape = F,
    caption = 'Meta analysis: Tests with consistent directions of effect and reaching a multiple testing threshold', row.names = F,
    col.names = c("", "Total", "BMI", "WHR", "BF")) %>%
  kable_styling(latex_options = c("striped", "HOLD_position", "repeat_header"), position = "center", font_size = 8) %>%
  kableExtra::footnote(general = "+ = positive direction of effect; - = negative direction of effect; Associated = consistent direction of effect in the meta-analysis and meet the multiple testing threshold for each exposure = 0.05/110", threeparttable = T, general_title = "", footnote_as_chunk = T)
```
-->

```{r MR-table-meta-analysis-sig-direction, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
knitr::kable(table, longtable = T, booktabs = T, format = "latex", escape = F,
    caption = 'Metabolites with a consistent direction of effect between BMI and WHR which reached a multiple testing threshold (p-value < 0.00045) in meta-analysis', row.names = F,
    col.names = c("Metabolite", "Class", "Subclass", "")) %>%
  kable_styling(latex_options = c("striped", "HOLD_position", "repeat_header"), position = "center", font_size = 8) %>%
  kableExtra::footnote(general = "", threeparttable = T, general_title = "", footnote_as_chunk = T)
```

### Comparison of two-sample Mendelian randomization and observational results from Chapter \@ref(systematic-review)
```{r eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
# meta-analysis associated metabolites 
data <- read.table("../../006_interval_metabolites/data/meta_analysis_p_results.txt", header = T, sep = "\t")
# directions
pos <- subset(data, meta_direction == "+ +")
neg <- subset(data, meta_direction == "- -")
pos_bmi <- subset(pos, exposure == "BMI_Yengo_941")
neg_bmi <- subset(neg, exposure == "BMI_Yengo_941")
pos_whr <- subset(pos, exposure == "WHR_Pulit_316")
neg_whr <- subset(neg, exposure == "WHR_Pulit_316")
pos_bf <- subset(pos, exposure == "BF_Lu_7")
neg_bf <- subset(neg, exposure == "BF_Lu_7")
# sig and directions 
data_sig <- subset(data, p <= 0.05/110)
sig_pos <- subset(data_sig, meta_direction == "+ +")
sig_neg <- subset(data_sig, meta_direction == "- -")
sig_pos_bmi <- subset(sig_pos, exposure == "BMI_Yengo_941")
sig_neg_bmi <- subset(sig_neg, exposure == "BMI_Yengo_941")
sig_pos_whr <- subset(sig_pos, exposure == "WHR_Pulit_316")
sig_neg_whr <- subset(sig_neg, exposure == "WHR_Pulit_316")
sig_pos_bf <- subset(sig_pos, exposure == "BF_Lu_7")
sig_neg_bf <- subset(sig_neg, exposure == "BF_Lu_7")
sig_pos_bmi_whr <- rbind(sig_pos_bmi, sig_pos_whr)
sig_pos_bmi_whr <- unique(sig_pos_bmi_whr$outcome)
sig_neg_bmi_whr <- rbind(sig_neg_bmi, sig_neg_whr)
sig_neg_bmi_whr <- unique(sig_neg_bmi_whr$outcome)
table <- data[data$outcome %in% sig_pos_bmi_whr, ]
table$direction <- "+"
table1 <- data[data$outcome %in% sig_neg_bmi_whr, ]
table1$direction <- "-"
table <- rbind(table,table1)
table <- table[!duplicated(table$outcome),]
metabolites <- table$outcome
table <- table[,c("raw.label", "class", "subclass", "direction")]
table <- table[order(table$class, table$subclass, table$raw.label),]

# get list of metabolites for BMI, WHR, BF from meta-analysis
data <- read.table("../../006_interval_metabolites/data/meta_analysis_p_results.txt", header = T, sep = "\t")
# directions ====
pos <- subset(data, meta_direction == "+ +")
neg <- subset(data, meta_direction == "- -")
pos_bmi <- subset(pos, exposure == "BMI_Yengo_941")
neg_bmi <- subset(neg, exposure == "BMI_Yengo_941")
pos_whr <- subset(pos, exposure == "WHR_Pulit_316")
neg_whr <- subset(neg, exposure == "WHR_Pulit_316")
pos_bf <- subset(pos, exposure == "BF_Lu_7")
neg_bf <- subset(neg, exposure == "BF_Lu_7")
# sig and directions ====
data_sig <- subset(data, p <= 0.05/110)
sig_pos <- subset(data_sig, meta_direction == "+ +")
sig_neg <- subset(data_sig, meta_direction == "- -")
sig_pos_bmi <- subset(sig_pos, exposure == "BMI_Yengo_941")
sig_neg_bmi <- subset(sig_neg, exposure == "BMI_Yengo_941")
sig_pos_whr <- subset(sig_pos, exposure == "WHR_Pulit_316")
sig_neg_whr <- subset(sig_neg, exposure == "WHR_Pulit_316")
sig_pos_bf <- subset(sig_pos, exposure == "BF_Lu_7")
sig_neg_bf <- subset(sig_neg, exposure == "BF_Lu_7")
sig_pos_bmi_whr <- rbind(sig_pos_bmi, sig_pos_whr)
sig_pos_bmi_whr <- unique(sig_pos_bmi_whr$outcome)
sig_neg_bmi_whr <- rbind(sig_neg_bmi, sig_neg_whr)
sig_neg_bmi_whr <- unique(sig_neg_bmi_whr$outcome)
# MR metabolite lists
mr_bmi_metabolites <- sig_pos_bmi$outcome
mr_bmi_metabolites <- c(mr_bmi_metabolites, sig_neg_bmi$outcome)
mr_whr_metabolites <- sig_pos_whr$outcome
mr_whr_metabolites <- c(mr_whr_metabolites, neg_whr$outcome)
mr_bf_metabolites <- pos_bf$outcome
mr_bf_metabolites <- c(mr_bf_metabolites, neg_bf$outcome)

# get metabolites from observational data ====
obs <- read.table("data/observational/data/analysis/results/combined/combined.txt", header = T, sep = "\t")
obs <- subset(obs, model == "model2" & group == "adults")
obs <- obs[,c("exposure", "metabolite", "raw.label", "b","se","p")]
colnames(obs) <- c("exposure", "outcome", "raw.label", "observational_b","observational_se","observational_p")
obs$direction[obs$observational_b > 0] <- "+"
obs$direction[obs$observational_b < 0] <- "-"
obs_bmi <- subset(obs, exposure == "bmi")
obs_whr <- subset(obs, exposure == "whr")
obs_bf <- subset(obs, exposure == "bf")
obs_bmi <- obs_bmi[obs_bmi$outcome %in% mr_bmi_metabolites, ]
obs_whr <- obs_whr[obs_whr$outcome %in% mr_whr_metabolites, ]
obs_bf <- obs_bf[obs_bf$outcome %in% mr_bf_metabolites, ]

# get metabolites from MR data ====
mr <- read.table("../../006_interval_metabolites/data/meta_analysis_p_results.txt", header = T, sep = "\t")
mr <- mr[,c("exposure", "outcome", "raw.label", "kettunen_b","kettunen_se","kettunen_p","interval_b","interval_se","interval_p","p","meta_direction","class","subclass")]
colnames(mr) <- c("exposure", "outcome", "raw.label", "kettunen_b","kettunen_se","kettunen_p","interval_b","interval_se","interval_p","meta_p","meta_direction","class","subclass")
mr$exposure[mr$exposure == "BMI_Yengo_941"] <- "bmi"
mr$exposure[mr$exposure == "WHR_Pulit_316"] <- "whr"
mr$exposure[mr$exposure == "BF_Lu_7"] <- "bf"
mr_bmi <- subset(mr, exposure == "bmi")
mr_whr <- subset(mr, exposure == "whr")
mr_bf <- subset(mr, exposure == "bf")
mr_bmi <- mr_bmi[mr_bmi$outcome %in% mr_bmi_metabolites, ]
mr_whr <- mr_whr[mr_whr$outcome %in% mr_whr_metabolites, ]
mr_bf <- mr_bf[mr_bf$outcome %in% mr_bf_metabolites, ]

# identify associated metabolites ====
bmi <- left_join(mr_bmi, obs_bmi, by = c("exposure", "outcome"))
whr <- left_join(mr_whr, obs_whr, by = c("exposure", "outcome"))
bf <- left_join(mr_bf, obs_bf, by = c("exposure", "outcome"))
bf <- bf[complete.cases(bf[,]),]

bmi$meta_direction = paste(bmi$meta_direction, bmi$direction)
whr$meta_direction = paste(whr$meta_direction, whr$direction)
bf$meta_direction = paste(bf$meta_direction, bf$direction)

bmi_obs_sig <- subset(bmi, observational_p < 0.05/53)
whr_obs_sig <- subset(whr, observational_p < 0.05/53)
bf_obs_sig <- subset(bf, observational_p < 0.05/53)
bf_obs_sig_pos <- subset(bf_obs_sig, meta_direction ==  "+ + +")
bf_obs_sig_neg <- subset(bf_obs_sig, meta_direction ==  "- - -")
bf_obs_sig <- rbind(bf_obs_sig_pos, bf_obs_sig_neg)

# associated metabolites
table <- read.table("../../006_interval_metabolites/analysis/associated_metabolites.txt", header = T, sep = "\t")
associated_metabolites <- unique(table$metabolite)
```

All `r nrow(table)` metabolites identified in the meta-analysis as associated with adiposity were analysed in the observational analysis in Chapter \@ref(observational). In the observational analysis, a multiple testing threshold of $0.05/53$ (`r round(0.05/53,4)`) was used. For the `r nrow(sig_neg_bmi) + nrow(sig_pos_bmi)` metabolites associated with BMI in the meta-analysis, all showed consistent directions of effect in the observational analysis. Only one metabolite (Total lipids in medium HDL, p-value = 0.1) did not reach the multiple testing threshold in the observational analysis for BMI. All `r nrow(sig_neg_whr) + nrow(sig_pos_whr)` metabolites associated with WHR in the meta-analysis had consistent directions of effect in the observational analysis and met the multiple testing threshold in the observational analyses (Figure \@ref(fig:MR-figure-associated-metabolites-comparison)). \par

As no associations reached the multiple testing threshold for BF in the meta-analysis, all metabolites with consistent directions of effect in the meta-analysis for BF were compared for consistent directions of effect with the observational analysis. A total of `r nrow(bf)` metabolites directionally consistent in the MR analyses for BF were available in the observational analysis. Of these, `r table(bf$meta_direction)[1]` had consistent negative directions across MR and observational analyses and `r table(bf$meta_direction)[4]` had consistent positive directions. All `r table(bf_obs_sig$meta_direction)[1]` metabolites with a negative direction of effect reached the multiple testing threshold, while `r table(bf_obs_sig$meta_direction)[2]` of the `r table(bf$meta_direction)[4]` metabolites with positive directions of effect reached the multiple testing threshold. \par

Across the 45 BMI, 48 WHR, and 9 BF associated metabolites, a total of `r length(associated_metabolites)` metabolites were associated with at least one measure of adiposity. For BMI and WHR, associations were identified through directional consistency and a multiple testing threshold in the MR meta-analysis followed by whether these metabolites had a consistent direction of effect and reached a multiple testing threshold in the observational analysis. For BF, associated metabolites were identified by consistent directions of effect across the MR meta-analysis and observational analysis, and which reached a multiple testing threshold in the observational analysis. Figure \@ref(fig:MR-figure-associated-metabolites-comparison) shows all `r length(associated_metabolites)` adiposity associated metabolites and there directions of effect in each analysis. Associations were identified as: having consistent directions of effect within exposures across MR and observational analyses and which reached a multiple testing threshold (BMI and WHR) in the meta-analysis (p-value $\leq$ `r round(0.05/110,4)`) and observational analysis (p-value $\leq$ `r round(0.05/53,4)`; BMI and WHR) and just in the observational analysis (BF). Yellow tiles indicate a negative direction of effect; blue tiles indicate a positive direction of effect.

\newpage
\thispagestyle{empty}
\vspace*{-3cm}

(ref:MR-figure-associated-metabolites-comparison-cap) **Metabolites associated with measures of adiposity in observational, MR, and MR meta-analysis**.

(ref:MR-figure-associated-metabolites-comparison-scap) Metabolites associated with measures of adiposity in observational, MR, and MR meta-analysis

```{r MR-figure-associated-metabolites-comparison, echo=FALSE, out.width='60%', fig.align='center', fig.cap='(ref:MR-figure-associated-metabolites-comparison-cap)', fig.scap='(ref:MR-figure-associated-metabolites-comparison-scap)'}
knitr::include_graphics("data/MR/figures/associated_metabolites_comparison.pdf")
```

\noindent Tile plot shows all metabolites with consistent directions of effect within exposures across MR and observational analyses and which reached a multiple testing threshold (BMI and WHR) in the meta-analysis and observational analysis (BMI and WHR) and just in the observational analysis (BF). Yellow tiles indicate a negative direction of effect; blue tiles indicate a positive direction of effect. * = multiple testing threshold reached: observational = `r round(0.05/53,4)`; Kettunen = `r round(0.05/22,4)`; INTERVAL = `r round(0.05/28,4)`; meta-analysis = `r round(0.05/110,4)`. \par

## Discussion {#MR-discussion}
In this chapter, the influence of adiposity on the metabolic profile is demonstrated in an MR framework. The use of MR allowed the interrogation of causality of various measures of adiposity on the metabolic profile, while accounting for limitations in observational analyses (discussed in Chapter \@ref(introduction) and \@ref(observational)). Data on adiposity measures were available for: BMI from up to 795,624 individuals of European ancestries from GIANT[Yengo2018], WHR from up to 697,702 individuals of European ancestries from GIANT[@Pulit2019], BF from up to 89,297 individuals European ancestries from Lu et a. (2016)[@Lu2016]. Two parallel MR analyses of 123 NMR derived metabolites measured in up-to 24,925 individuals of European ancestries from Kettunen et al. (2016)[@Kettunen2016] and 230 NMR derived metabolites measured in up-to 40,905 individuals of European ancestries from INTERVAL (unpublished) were conducted. Meta-analysis of 110 metabolites measured in both the Kettunen and INTERVAL datasets and comparison with observational analyses from Chapter \@ref(observational) identified 56 associations between adiposity and metabolite measures that were consistent in direction of effect across MR and observational analyses and passed multiple testing thresholds for both analyses. Positive associations were found for metabolites in VLDL (small, very small, medium, large, and very large), as well as aromatic and branched chain amino acid subclasses. While negative associations were found for HDL (medium, large and very large) subclasses. \par

In MR analyses, there was evidence for a broad effect of BMI and WHR on the metabolic profile. Both adiposity measures showed positive and negative effects on whole subclasses of metabolites, such as small and very small VLDL, branched chain amino acids (positive), and large and very large HDL (negative). However, there were many effect estimates across the Kettunen and INTERVAL datasets that did not show consistent directions of effect. For example, when using the Kettunen data, positive directions of effect were found for BMI and WHR with very large VLDL. But when using the INTERVAL data, a mix of positive, negative, and null effects were found for BMI and WHR with very large VLDL. \par

On the whole, MR results revealed WHR to have a larger effect across metabolites compared to BMI and BF. However, confidence intervals for WHR and BMI mostly overlapped with one another. Evidence for an association between BF and the metabolic profile was weak. Generally, directions of effect for BF conflicted with those of BMI and WHR. Analyses using BF resulted in larger effect estimates and wider confidence intervals which spanned the null for the majority of metabolites. For example, when using Kettunen data, BF was negatively associated with metabolites in the medium VLDL subclass, while BMI and WHR were positively associated. When using INTERVAL data, BF was negatively associated with creatinine while BMI and WHR were positively associated. \par 

Consistent directions of effect across all adiposity measures were found for the amino acids tyrosine, phenylalanine, and valine only. There were no other metabolites across all three measures of adiposity with a consistent direction of effect. All three metabolites showed an increase as a result of adiposity. The only other amino acids associated with adiposity were decreased as a result of BMI and WHR (glutamine) and by BF (histidine). Results for tyrosine, phenylalanine, and valine are consistent with previous findings, including those by Wurtz et al. (2014)[@Wurtz2014]. Although confidence intervals were much wider than results here, Wurtz et al. (2014) found glutamine was increased in MR and decreased in observational analyses. Weak evidence for an increase in histidine was found by Wurtz et al. (2016). Of these associated amino acids, all but glutamine and tyrosine are essential. Increased tyrosine, phenylalanine, valine, leucine, and isoleucine have been associated with increased risk of numerous diseases such as colorectal cancer[@Ritchie2010; @Ni2014; @Lin2016; @Brown2016a], pancreatic cancer[@Ritchie2010], preeclampsia[@Bahado-Singh2012; @Bahado-Singh2014], irritable bowel syndrome[@LeGall2011; @Hong2011], and crohns and ulcerative colitis[@Bjerrum2015]. Reductions in glutamine and Histidine meanwhile have been associated with colorectal [@Lin2016; @Ikeda2012; @Ni2014] and pancreatic cancer[@Ikeda2012; @Zhang2012]. There is some conflicting evidence however, with higher histidine levels also found to be associated with colorectal cancer[@Goedert2014]. A recent prospective analysis found associations between 12 metabolites and increased risk of endometrial cancer, including, tyrosine, phenylalanine, leucine, and isoleucine. However, after adjusting for BMI only the association between glycine and serine with endometrial cancer remained[@Dossus2021]. Given many of these metabolites were associated with increased adiposity it is possible they may be intermediates in the adiposity relationship with endometrial cancer. \par

For BMI and WHR, directions of effect were the same across all metabolites. BMI and WHR were both associated with reductions of metabolites in medium HDL, large HDL, and very large HDL subclasses, as well as reductions in apolipoprotein A1, mean diameter for HDL particles, and total cholesterol in HDL. In addition, BMI and WHR were both associated with increases in numerous LDL subclasses (e.g., IDL and very small VLDL) as well as apolipoprotein B. These results are supported by the underlying relationship between HDL and LDL metabolites as well as similar results in numerous MR[@Wurtz2014; @Bull2020; @Bell2021a] and observational[@Wurtz2014; @Moore2014; @SantosFerreira2017; @Cirulli2019; @Lau2020; @Brachem2020; @Stevens2020; @OKeeffe2020; @Wulaningsih2019; @Frigerio2021; @Neeland2019; @Bachlechner2016; @Rangel-Huerta2019] analyses. Apolipoprotein A1 is the major component of HDL particles and enables uptake of lipids by HDL from cells. HDL particles primarily transport lipids away from the cells, in what is known as reverse cholesterol transport. These lipids end up at the liver where they are recycled and excreted[@Feingold2000; @Marz2017]. Apolipoprotein B on the other hand is the major component of VLDL, IDL, and LDL particles, enabling lipid uptake, and thus increased transport of lipids around the body[@Feingold2000; @Venugopal2021]. \par

Observational studies have highlighted increased HDL to be associated with a protective effect on cardiovascular disease (CVD)[@Kosmas2018] while increased LDL is shown to increase CVD risk[@Ference2017]. MR studies support observational results for LDL[@Ference2017], but are conflicting for HDLs protective effect[@White2016]. Randomised controlled trials have also not found strong evidence for an effect of HDL lowering drugs on CVD risk[@Holmes2017]. Some focus has been given to a measure of HDLs contribution to reverse cholesterol transport, HDL cholesterol efflux capacity (HDL-CEC), which is shown to be protective for CVD[@Rohatgi2014; @Kuusisto2019]. Estimation of HDL-CEC however was not available in the current analyses. There are also associations between reduced HDL and many cancers, however there is some evidence to suggest this may be bi-directional[@Pirro2018]. Recent work has suggested that increased HDL does not confer a protective benefit on mortality[@Ala-Korpela2021], and instead there is evidence that apolipoprotein B may underlie the association between many lipids and diseases such as CHD[@Richardson2020]. Apolipoprotein B and its associated lipids, such as VLDLs, may therefore be potential intermediates in disease associations. \par

Sensitivity analyses were broadly consistent with the main analysis when investigating the impact of each exposure on metabolites from both the Kettunen and INTERVAL datasets. There was considerable heterogeneity in the effect estimates derived from the different genetic instruments used in the main analysis. In single SNP MR analysis, a number of SNPs showed disproportionately larger effect estimates across many metabolites. Leave-one-out analysis did not highlight any individual SNP driving the effect for any one metabolite. \par

Steiger directionality tests can provide an estimate as to whether the "true" causal direction of an MR analyses has been tested. That is, if we perform the analysis of exposure $A$ on outcome $B$ and our hypothesis is that $A$ causes $B$, the Steiger test will estimate whether this direction ($A$ to $B$) is "true". The test relies on using effect size to estimate variance explained in the exposure and outcome. Results from the Steiger test varied across analyses using Kettunen and INTERVAL data. For analyses using the Kettunen data, a majority of tests did not reflect the "true" causal direction. This was primarily a result of almost all tests using BMI and WHR not reflecting a causal direction. When using the INTERVAL data, a majority of tests reflected the "true" causal direction. As the majority of analyses using additional instrument lists for BMI and WHR, for which all instruments were included in the larger main analysis instruments, reflected a "true" causal direction these results should be interpreted with caution. As both BMI and WHR used large SNP lists, it is likely that many of those SNPs will have larger effect sizes for metabolites given their effects on BMI and WHR are very small - effect sizes are small for BMI and WHR for numerous SNPs because they have been identified in large well powered GWAS which enable identification of SNPs with small effect size. For BF, results of the Steiger test are likely more complex, given that many tests found to be "true" when using the Kettunen data were also found to be "true" when using the INTERVAL data even though directions of effect were different. It is unclear why there is a difference here, but it may be due to the instrumentation of BF, which, unlike BMI and WHR, has been linked with 'favourable adiposity'. \par

### Limitations
MR relies upon the use of an instrument to model the effect of an exposure on an outcome. This relationship is dependent upon the genetic architecture of the instruments and the traits under investigation and power (predominantly sample size of the instrument-outcome effect), both of which influence the utility of the various MR methods. The first MR assumption, that the instrument is robustly associated with the exposure, is generally measured via an $F$ statistic, for which an arbitrary threshold of > 10 denotes a *strong instrument*. All of the instruments used here exceeded this threshold. In addition, the variance explained in an exposure by a GWAS can indicate the power afforded in the MR analysis; the variance explained for exposures used in the main analysis varied from 6% for BMI, to 3% for WHR, and ~0.4% for BF. The considerably lower variance explained for BF may have impacted on results; when using additional instrument lists for BMI and WHR which explained a lesser percentage of variance confidence intervals became wider. Winners curse is unlikely given summary statistics used here are well powered and underwent replication by the authors of the original GWAS publications. \par

The consistent directions of effect observed across BMI and WHR measures adds weight to their associations with metabolites. For BF, conflicting directions of effect were observed when comparing to BMI and WHR. However, confidence intervals were wider and, although they spanned the null, included the estimates and confidence intervals for BMI and WHR in a majority of cases. There was considerable difference in the sample sizes used in the GWAS for BMI, WHR, and BF. In addition, whereas BMI and WHR were measured in only one way for their respective GWAS, the BF GWAS included measures of BF from DXA and impedance devices. Though, as shown in Chapter \@ref(chapter4), DXA and impedance measures of BF are highly correlated, the additional analysis for BF which used only impedance measures in the GWAS showed much greater directional consistency with BMI and WHR and also included a number of metabolites which reached the multiple testing threshold. Given the highly correlated nature of the exposures, and the consistency in observational estimates with metabolites, results for BF here appear counter-intuitive. Additional analyses failed to appropriately account for the differences in results and further investigation of instrumentation is warranted. \par

The instrumentation approach used in the main analysis followed that used by the majority of analyses reviewed in Chapter \@ref(systematic-review). That is, instruments were obtained directly from the most recently published GWAS with the largest sample size. As GWAS sample sizes increase so does the likelihood of sample overlap which can bias estimates towards the confounded observational effect. Additionally, there is the concern of population structure within all GWAS, both large[@Haworth2019; @Lawson2020; @Sarmanova2020; @Sanderson2021] and small[@Sohail2019], which if not accounted for appropriately can lead to violation of the independence assumption and bias estimates towards the observational confounded estimate. Additional analyses, which used a number of different instrumentation practices, including using GWAS which did not include UK Biobank participants and using a standardised clumping strategy, aimed to reduce the impact of population structure that has been demonstrated in UK Biobank and that independence of SNPs was the same within and between exposure instrument lists. Additional BMI and WHR exposures, which did not include UK Biobank participants, showed consistent results with the main analysis. For BF there was little difference between the main analysis and the additional analysis which used 5 SNPs from Lu et al. (2015)[@Lu2015] having removed two SNPs that likely have different biological function. However, there was considerable difference in directions of effect when using the additional SNP list from Hubel et al. (2019)[@Hubel2019]. \par

There is clearly complexity in the choice of instrument, especially with complex traits such as adiposity. This complexity is perhaps well demonstrated through the BF analysis. For BF, there was considerable difference in effect estimates compared to the BMI and WHR results, with a large proportion of estimates in the opposite direction to those of BMI and WHR. This is counter-intuitive given the strong correlation between BMI, WHR, and BF, and the consistent results obtained in observational analyses in Chapter \@ref(observational) between BMI, WHR, and BF. Removal of two SNPs previously associated with 'favourable adiposity'[@Yaghootkar2014; @Yaghootkar2016] resulted in a global tightening of confidence intervals and a number of effect estimates changing direction to be more consistent with BMI and WHR. A number of these effects subsequently reached the multiple testing threshold. However, the majority of MR results remained opposite to that of BMI and WHR. To further investigate the difference in effects observed for BMI and WHR, and BF post-hoc interrogation of results from a single, well characterised, adiposity SNP (rs1558902) was undertaken. Measures of adiposity have been associated with numerous mutations in the *FTO* locus, with studies highlighting major roles in neural signalling of appetite suppression, alongside roles in adipocyte browning[@Loos2018]. Single SNP MR analysis using the *FTO* locus (rs1558902; BF beta = 0.051; BMI beta = 0.082) and Kettunen metabolites showed highly consistent results for BF instrumented using the Lu et al. (2016) estimate and BMI instrumented using the Locke et al. (2015) estimate. These results differed only in their effect estimate and standard error which was in line with the difference in the SNP beta for the traits - BMI effect estimates were 62% larger than BF effect estimates. Leave-one-out analysis for BF instrumented using the 7 genetic variants identified in Lu et al. (2016) did not indicate a single SNP that could be driving a pleiotropic association. However, median effect estimates for rs6857 (rs6857 is associated with *NECTIN2* and has previously been associated with a number of diseases including Alzheimers[@Ferrari2017]) were much larger than those for other SNPs, both with and without exclusion of the two 'favourable adiposity' SNPs. In many cases, rs6857 did not span the null. It is possible that tests for pleiotropy were underpowered however. The unexpected results for BF instrumented using the Lu et al. (2016) genetic variants may be due to a variety of reasons, not least measurement error, sample size differences between BF (up-to 89,297), BMI (up-to 795,624), and WHR (up-to 697,702), and the variance explained by the respective instruments: BF = 0.416%, WHR = 3%, BMI = 6%. The complexity of instrumentation is discussed further in the discussion Chapter \@ref(discussion) as this is also relevant to Chapter \@ref(mediation). \par

As discussed in the limitations section of Chapter \@ref(observational), there is no standardised approach, nor a gold standard, for performing metabolomics quality control. In Chapter \@ref(observational), quality control, including outlier detection and removal, was performed using the `metaboprep` `R` package. Metabolite data here however were from summary statistics, the data that went into these GWAS will have undergone different quality control procedures for the metabolomics data. Additionally, the metabolomics data were inverse rank normally transformed in the Kettunen data and inverse normally log transformed in the INTERVAL data, they were also adjusted for different covariables, it was therefore not appropriate to meta-analyse using effect estimates. Future work should look to use a standardised method for pre-analysis quality control of metabolomics data, such as `metaboprep` while ensuring study specific adjustments are made where appropriate. On the latter point, a centralised approach to analysis of metabolomics data, such as that being employed by the Consortium of Metabolomics Studies, will allow for more efficient comparisons and meta-analyses of studies. \par

MR analyses are subject to a number of assumption the main three being: (i) the instrumental variable ($Z$) is robustly associated with the exposure ($X$), (ii) there is no independent association of the instrumental variable with the outcome ($Y$) other than through the exposure, (iii) the instrumental variable is independent of common causes of the instrument-outcome association ($U$). In this work, instruments were obtained from large well-powered GWAS, and the $F$ statistic for each instrument was above a nominal threshold of 10 meaning the first assumption was likely met. In regards the other two assumptions, formal testing is not possible. However, sensitivity analyses can provide an indication of pleiotropy and, though not comprehensive, sensitivity analyses conducted here were concordant with the main analysis, suggesting that there is no large impact of pleiotropy in these results. However, as discussed, there may be additional cautions to be taken with the BF analysis, where instrumentation is currently likely to be inadequate. \par

An additional consideration with regards to instrumentation is the possibility that SNPs associated with adiposity traits may also be associated directly with metabolites or metabolic pathways. The Steiger directionality test can be used to test whether the "true" causal direction is the one under investigation, i.e. the effect of adiposity measures on metabolites. There are a number of limitations associated with the Steiger test[@Hemani2017] one of which is that it assumes that there are no pleiotropic effects. When using large SNP lists, the likelihood of an association between any one of the exposure associated SNPs with the outcome is likely to increase. The Steiger test uses effect size to estimate the variance explained by each SNP in the exposure and outcome respectively. In a well powered study, as more SNPs are included in an instrument there is a greater chance of including SNPs with small effect sizes for the exposure, as such the likelihood of the test returning a "true" causal direction will decrease. Although the additional analyses for BMI and WHR, which used smaller SNP lists compared to the main analysis, were found to reflect the "true" causal direction more often than the main analysis, the high correlation between the two analyses both in terms of effect size and direction of effect would suggest that any associations between the SNPs used in the main analysis with the outcomes did not drastically affect results. This does not fully address the potential for direct associations between exposure SNPs and the outcome however, and future work should look at exploring the genetic correlation between the traits. \par

<!--
The effects of population structure are becoming more apparent in large GWAS[@Hartwig2018; @Brumpton2019] and this poses an issue for MR studies which use their data. Analyses involving genetic instruments obtained from UK Biobank were repeated using instruments obtained from smaller studies that did not include UK Biobank and results were similar. This goes some-way to address concerns that large-scale studies may bias analyses due to sample overlap and population structure[@Haworth2019; @Lawson2020; @Sarmanova2020; @Sanderson2021]. That being said, it is still possible that smaller studies may also present structural concerns[@Sohail2019]. \par
-->

### Conclusion
Adiposity exerts a systemic impact on the metabolome, which is consistent across multiple measures of adiposity (namely, BMI and WHR). Results here highlight a number of metabolites associated with adiposity which have consistent directions of effect and effect sizes with previous MR analyses[@Wurtz2014; @Bull2020]. Some of these metabolites have previously been shown to be associated with adiposity-related diseases such as colorectal[@Bull2020] and endometrial cancer[@Kliemann2021], meaning that they could play an intermediary role in these relationships. However, this possible intermediary role of adiposity-associated metabolites on various health outcomes are yet to be explored systematically, especially in an MR context. Although adiposity instrumentation has been well established, and shown here to give consistent results across many different instrument lists and exposures, results for the main BF analysis presented here remain unexpected and inconsistent with BMI and WHR. Therefore, proper consideration of the appropriate use of genetic variants when compiling instruments for use in MR analyses is required, especially when instrumenting complex traits such as those presented here. By extension, instrumentation is of particular importance when investigating associations between adiposity-associated metabolites identified here and diseases identified in Chapter \@ref(systematic-review). In Chapter \@ref(mediation), investigation of the intermediary role of these adiposity-related metabolites and an exemplar outcome perturbed by adiposity as identified in meta-analysis in Chapter \@ref(systematic-review), endometrial cancer, will be explored. \par

\newpage


<!---
Steiger test
con
  still liable to measurement error
  doesn't account for winners curse (inflated estimates of variance explained in under-powered studies)
  simulations used a single SNP exposure
    extension to multiple instruments requires instruments to be independent
  simulations used one sample setting
  implies no horizontal pleiotropy
  assume no measurement error in IV - but this is unlikely
  doesn't account for exposure misspecification - an instrument could be associated with several related outcomes, with only one actually having a causal effect on the outcome - this is illustrated for lipid fractions[47, 48].
  inferring the causal direction using an instrument of unknown biology is highly sensitive to measurement error
  can distinguish between x → y or y → x but not x ← g → y
  assume all genetic effects are additive
  
  
pro
  uses a formal statistical framework to test for the reliability of the assumed direction of causality. 
  less likely to infer the wrong direction of causality compared to CIT, while substantially improving power over CIT in the cases where d > 0.
  An advantage of using the Steiger test in the two sample context is that it can compare correlations in independent samples where sample sizes are different



assumes that there is a causal relationship between the two variables, 
assumes that the SNP is a valid instrument for one of two variables

liable to give incorrect causal directions under some other circumstances. 
  some levels of horizontal pleiotropy, where the SNP influences the outcome through some pathway other than the exposure, could induce problems because this is a means by which the instrument is invalid. 
  some differential values of measurement error between the exposure and the outcome could lead to incorrect inference of the causal direction (S2 Text)
  some levels of unmeasured confounding between the exposure and the outcome could lead to inference of the wrong causal direction (S3 Text).
  
  
overview figure
  are there any benefits/limitations to using SNPs from large gwas
  why were only 110 metabolites shared - same reason as observational

must comment on the difference in variance explained of the SNP lists  
-->



