---
output: html_document
editor_options: 
  chunk_output_type: console
---
`r if(knitr:::is_latex_output()) '\\appendix'`

`r if(!knitr:::is_latex_output()) '# (APPENDIX) Appendix {-}'` 

<!--
If you feel it necessary to include an appendix, it goes here.
-->


# 

\newpage
## Chapter 1 {#chapter1-appendix}
```{r include=FALSE}
setwd(Sys.getenv("000_thesis"))
source("index/data/chapter1/chapter1_source.R")
```

### Literature search
Manual literature searching is prone to bias. Literature mining tools, though susceptible to publication and other biases, provide an alternative approach enabling a large number of articles to be assessed in a semi-systematic way. MELODI[@Elsworth2018], a literature mining tool, was used to identify intermediate diseases between BMI and mortality. <br>

Briefly, MELODI creates individual article sets based on search terms ('body mass index' and 'mortality') and looks for enriched overlapping terms. MELODI uses PubMed and SemMedDB to identify enriched terms. PubMed is a data base of health and biomedical research literature, and SemMedDB is a semantic predications repository built from PubMed citations. Identifying enriched overlapping terms is a two-step process. First, overlapping terms are identified. Second, the degree of overlap given the observed and expected frequency of terms across all articles not just those included in the article sets is quantified. <br>

All articles published from 01/01/2000--09/12/2019 (maximum number of articles per article set is 1,000,000) using the terms 'body mass index' as the source and 'mortality' as the outcome were included. Raw results are available on [GitHub](https://github.com/mattlee821/000_thesis/index/data/chapter1). A total of 187,951 and 787,451 articles were retrieved and included in the source and outcome article sets. Using the SemMedDB Triple results a total of `r nrow(MELODI)` enriched overlapping terms were identified. This included similar terms to 'body mass index' as the source, which were removed (n = `r nrow(MELODI_bmi_clna)`). A Bonferroni corrected p-value (`r signif(0.05/nrow(MELODI_bmi_clna), 2)`) removed `r nrow(MELODI_bmi_clna) - nrow(MELODI_bmi_clna_p)` terms. Terms were filtered for uniqueness (n = `r nrow(MELODI_bmi_clna_p_unique)`) and presence in the following categories: Age Group (n = `r nrow(MELODI_intermediates_unique_aggp)`), Bacterium (included because of confounding with pneumonia; n = `r nrow(MELODI_intermediates_unique_bact)`), finding (not an official MELODI category; n = `r nrow(MELODI_intermediates_unique_finding)`), general (not an official MELODI category; n = `r nrow(MELODI_intermediates_unique_general)`), Fungus (included because of confounding with pneumonia; n = `r nrow(MELODI_intermediates_unique_fngs)`), Health Care Activity (n = `r nrow(MELODI_intermediates_unique_hlca)`), Human (n = `r nrow(MELODI_intermediates_unique_humn)`), Injury or Poisoning (n = `r nrow(MELODI_intermediates_unique_inpo)`), Mammal (animal studies; n = `r nrow(MELODI_intermediates_unique_mamm)`), Patient or Disabled Group (n = `r nrow(MELODI_intermediates_unique_podg)`), Population Group (n = `r nrow(MELODI_intermediates_unique_popg)`), Sign or Symptom (n = `r nrow(MELODI_intermediates_unique_sosy)`), Virus (included because of confounding; n = `r nrow(MELODI_intermediates_unique_virs)`). <br>

The following terms were removed and merged into immunocompromised host: infections, hospital and opportunistic infections. The following terms were removed because of duplication under different names: cardiac death, sudden death, and sudden cardiac death (merged into cessation of life); coronary arteriosclerosis (atherosclerosis);  cardiovascular morbidity, cardiac event, heart diseases, vascular diseases (cardiovascular diseases); depressed mood (depressive disorder); diabetes mellitus and insulin-dependent (diabetic); metabolic diseases (metabolic syndrome). The following terms were removed because they were top-level categories that could either include a wide variety of terms already included or were duplicated by other terms: chronic disease, critical illness, disability, pregnancy complications and perinatal morbidity (included in pregnancy category), pathogenesis. As a result of filtering a total of `r nrow(MELODI_intermediates_unique_final)` terms remained. These terms were combined into `r nlevels(MELODI_final_table$Category)` categories: `r levels(MELODI_final_table$Category)`. The 'other' category included traits that did not fit into one of any of the other categories and which did not have aligned traits to form a separate category (Table \@ref(tab:chapter1-table-MELODI)). The majority of intermediates were cardiovascular related. <br> 

It should be noted that the search did not include articles prior to the year 2000 and focussed only on those archived by PubMed. Enrichment aims to reduce the noise introduced when searching hundreds of thousands of articles, however manual curation, which will hold its own biases, is needed in order to obtain an informative list of enriched terms. <br>

```{r chapter1-table-MELODI, echo=FALSE}
data <- MELODI_final_table
data <- data[order(data$Category),]
# make table
if (doc.type == "docx") { 
  x <- flextable(data = data) %>%
    set_caption('Intermediates between body mass index and mortality identified using the literature mining tool MELODI')
  } else { 

if (doc.type != "docx") { 
  x <- knitr::kable(
    (data), longtable = TRUE, booktabs = TRUE,
    caption = 'Intermediates between body mass index and mortality identified using the literature mining tool MELODI', row.names = F)
  x <- kable_styling(x, full_width = F) 
} else {

}
}
x

# Identify number of traits in each factor
# data %>% 
  # group_by(Category) %>%
   # summarise(Intermediate = length(Category))
```


\newpage
## Chapter 2 {#chapter2-appendix}

\newpage
## Chapter 3 {#chapter3-appendix}

\newpage
## Chapter 4 {#chapter4-appendix}

### Metabolites {#chapter4-appendix-metabolites}
\blandscape
```{r echo=FALSE, warning=FALSE, error=FALSE}
library(tidyr)
data <- read.table("data/chapter4/data/analysis/results/combined/combined.txt", header = T, sep = "\t")
data <- data[,c("metabolite","raw.label","class","subclass","derived_features")]
data <- unique(data[ , 1:5 ] )
# make table,
if (doc.type == "docx") { 
x <- data
  } else { 

if (doc.type != "docx") { 
  x <- knitr::kable(
    (data), longtable = T, booktabs = T,
    caption = 'Metabolites available after quality control across all age groups', row.names = F,
    col.names = c("Metabolite","Label", "Class","Subclass", "Derived"))
  x <- kable_styling(x, full_width = T) %>%
    collapse_rows(columns = 1:5)
} else {

}
}
x

```
\elandscape


### Tables of results
\blandscape
```{r echo=FALSE, warning=FALSE, error=FALSE}
library(tidyr)
data <- read.table("data/chapter4/data/analysis/results/combined/combined.txt", header = T, sep = "\t")
data <- subset(data, class == "Amino acids")
data <- data[,c("label","subclass","model","group","exposure","b", "lower_ci", "upper_ci","p")]
# make table,
if (doc.type == "docx") { 
x <- data
  } else { 

if (doc.type != "docx") { 
  x <- knitr::kable(
    (data), longtable = T, booktabs = T,
    caption = 'Results of linear regression analysis of measures of increased adiposity and metabolites among different age groups', row.names = F,
    col.names = c("Metabolite","Subclass", "Model","Group", "Exposure", "Beta", "Lower CI", "Upper CI", "p-value"),
    digits = 4)
  x <- kable_styling(x, full_width = F) %>%
    collapse_rows(columns = 3:5)
} else {

}
}
x

```
\elandscape

### Correlations {#chapter4-appendix-correlations}
Due to the large number of tests performed, concordance between models, exposures, and age groups were investigated using Spearman's correlations.

```{r echo=FALSE, warning=FALSE, error=FALSE}
library(tidyr)
data <- read.table("data/chapter4/data/analysis/results/combined/combined.txt", header = T, sep = "\t")

# models within groups within exposures
## children ====
a <- subset(data, group == "children")
bmi <- subset(a, exposure == "bmi")
bmi <- bmi[,c(1,3,16)]
bmi <- spread(bmi, key = model, value = b)
bmi <- bmi[,c(2:3)]
children_bmi_cor <- cor(bmi, use = "complete.obs", method = "spearman")

whr <- subset(a, exposure == "whr")
whr <- whr[,c(1,3,16)]
whr <- spread(whr, key = model, value = b)
whr <- whr[,c(2:3)]
children_whr_cor <- cor(whr, use = "complete.obs", method = "spearman")

bf <- subset(a, exposure == "bf")
bf <- bf[,c(1,3,16)]
bf <- spread(bf, key = model, value = b)
bf <- bf[,c(2:3)]
children_bf_cor <- cor(bf, use = "complete.obs", method = "spearman")

## adolescents ====
a <- subset(data, group == "adolescents")
bmi <- subset(a, exposure == "bmi")
bmi <- bmi[,c(1,3,16)]
bmi <- spread(bmi, key = model, value = b)
bmi <- bmi[,c(2:4)]
adolescents_bmi_cor <- cor(bmi, use = "complete.obs", method = "spearman")
whr <- ""

bf <- subset(a, exposure == "bf")
bf <- bf[,c(1,3,16)]
bf <- spread(bf, key = model, value = b)
bf <- bf[,c(2:4)]
adolescents_bf_cor <- cor(bf, use = "complete.obs", method = "spearman")

## young_adults ====
a <- subset(data, group == "young_adults")
bmi <- subset(a, exposure == "bmi")
bmi <- bmi[,c(1,3,16)]
bmi <- spread(bmi, key = model, value = b)
bmi <- bmi[,c(2:4)]
young_adults_bmi_cor <- cor(bmi, use = "complete.obs", method = "spearman")

whr <- subset(a, exposure == "whr")
whr <- whr[,c(1,3,16)]
whr <- spread(whr, key = model, value = b)
whr <- whr[,c(2:4)]
young_adults_whr_cor <- cor(whr, use = "complete.obs", method = "spearman")

bf <- subset(a, exposure == "bf")
bf <- bf[,c(1,3,16)]
bf <- spread(bf, key = model, value = b)
bf <- bf[,c(2:4)]
young_adults_bf_cor <- cor(bf, use = "complete.obs", method = "spearman")

## adults ====
a <- subset(data, group == "adults")
bmi <- subset(a, exposure == "bmi")
bmi <- bmi[,c(1,3,16)]
bmi <- spread(bmi, key = model, value = b)
bmi <- bmi[,c(2:4)]
adults_bmi_cor <- cor(bmi, use = "complete.obs", method = "spearman")

whr <- subset(a, exposure == "whr")
whr <- whr[,c(1,3,16)]
whr <- spread(whr, key = model, value = b)
whr <- whr[,c(2:4)]
adults_whr_cor <- cor(whr, use = "complete.obs", method = "spearman")

bf <- subset(a, exposure == "bf")
bf <- bf[,c(1,3,16)]
bf <- spread(bf, key = model, value = b)
bf <- bf[,c(2:4)]
adults_bf_cor <- cor(bf, use = "complete.obs", method = "spearman")

# model 1 within group across exposures
## children ====
a <- subset(data, group == "children")
a <- subset(a, model == "model2")
a <- a[,c(1,3,14)]
a <- spread(a, key = exposure, value = b)
a <- a[,c(2:4)]
children_m1_cor <- cor(a, use = "complete.obs", method = "spearman")
## adolescents ====
a <- subset(data, group == "adolescents")
a <- subset(a, model == "model2")
a <- a[,c(1,3,14)]
a <- spread(a, key = exposure, value = b)
a <- a[,c(2:3)]
adolescents_m1_cor <- cor(a, use = "complete.obs", method = "spearman")
## young_adults ====
a <- subset(data, group == "young_adults")
a <- subset(a, model == "model2")
a <- a[,c(1,3,14)]
a <- spread(a, key = exposure, value = b)
a <- a[,c(2:4)]
young_adults_m1_cor <- cor(a, use = "complete.obs", method = "spearman")
## adults ====
a <- subset(data, group == "adults")
a <- subset(a, model == "model2")
a <- a[,c(1,3,14)]
a <- spread(a, key = exposure, value = b)
a <- a[,c(2:4)]
adults_m1_cor <- cor(a, use = "complete.obs", method = "spearman")

# model 1 within exposure across groups
## bmi ====
a <- subset(data, exposure == "bmi")
a <- subset(a, model == "model2")
a <- a[,c(1,3,15)]
a <- spread(a, key = group, value = b)
a <- a[,c(2:5)]
bmi_m1_cor <- cor(a, use = "complete.obs", method = "spearman")
## whr ====
a <- subset(data, exposure == "whr")
a <- subset(a, model == "model2")
a <- a[,c(1,3,15)]
a <- spread(a, key = group, value = b)
a <- a[,c(2:4)]
whr_m1_cor <- cor(a, use = "complete.obs", method = "spearman")
## bf ====
a <- subset(data, exposure == "bf")
a <- subset(a, model == "model2")
a <- a[,c(1,3,15)]
a <- spread(a, key = group, value = b)
a <- a[,c(2:5)]
bf_m1_cor <- cor(a, use = "complete.obs", method = "spearman")
rm(a,bf,bmi,data,whr)

# show correlations for each model within exposures ====
print("Correlations across models within exposures and age groups")
print("Children BMI")
children_bmi_cor
print("Children WHR")
children_whr_cor
print("Children BF")
children_bf_cor

print("Adolescents BMI")
adolescents_bmi_cor
print("Adolescents BF")
adolescents_bf_cor

print("Young adults BMI")
young_adults_bmi_cor
print("Young adults WHR")
young_adults_whr_cor
print("Young adults BF")
young_adults_bf_cor

print("Adults BMI")
adults_bmi_cor
print("Adults WHR")
adults_whr_cor
print("Adults BF")
adults_bf_cor

# show correlations for model 1 across exposures ====
print("Correlations across exposures within age groups for model 2")
print("Children")
children_m1_cor

print("Adolescents")
adolescents_m1_cor

print("Young adults")
young_adults_m1_cor

print("Adults")
adults_m1_cor

# show correlaion for model 1 within exposures across age groups ====
print("Correlations within exposures across age groups for model 2")
print("BMI")
bmi_m1_cor

print("WHR")
whr_m1_cor

print("BF")
bf_m1_cor
```

### Communications {#chapter4-appendix-communications}
```{r appendix-chapter4-communications-tanita, echo=FALSE, out.height='100%', fig.cap="Email communication with Tanita RE calculation of body fat percentage from raw impedance"}
knitr::include_graphics("data/chapter4/supplement/Tanita_communication.pdf")
```

### Figures

#### Forestplots {#chapter4-appendix-figures-forestplots}
\blandscape
```{r chapter4-appendix-figure-forestplot-main, echo=FALSE, out.width='100%', fig.cap="Forestplot of effect estimates from a linear regression"}
knitr::include_graphics("data/chapter4/Supplement/forestplot_main.pdf")
```
\noindent 
\bsmall
*Figure \@ref(fig:chapter4-appendix-figure-forestplot-main) shows effect estimates and 95% confidence intervals from model 2 for all exposures and age groups. Effect estimates are per-standard deviation increase in metabolite per-standard deviation increase in exposure.*
\esmall
\elandscape

\blandscape
```{r chapter4-appendix-figure-forestplot-supplement, echo=FALSE, out.width='100%', fig.cap="Forestplot of effect estimates from a linear regression, derived metabolites"}
knitr::include_graphics("data/chapter4/Supplement/forestplot_supplement.pdf")
```
\noindent 
\bsmall
*Figure \@ref(fig:chapter4-appendix-figure-forestplot-supplement) shows effect estimates and 95% confidence intervals from model 2 for all exposures and age groups. Effect estimates are per-standard deviation increase in metabolite per-standard deviation increase in exposure.*
\esmall
\elandscape



Forestplots show the effect estimate and 95% confidence interval with multiple testing thresholds (solid points) set as the number of independent metabolites for each age group.



#### Circos plots
Circos plots show effect estimates and 95% confidence intervals with multiple testing thresholds (solid points) set as the number of independent metabolites for each age group (set as the lowest number of independent metabolites for combined age group plots).

##### Comparison of metabolic profile across exposures for each age group {#chapter4-appendix-circosplot-supplement2}
```{r plot-circos-supplement-children, echo=FALSE, out.width='100%', fig.cap="Circos plot of effect estimates from a linear regression in ALSPAC children, supplementary metabolites"}
knitr::include_graphics("data/chapter4/figures/circosplot_supplement_children.pdf")
```
\noindent 
\bsmall
*Figure \@ref(fig:plot-circos-supplement-children) shows effect estiamtes and 95% confidence intervals from a linear regression. Each point represents a single result, with the metabolites labelled around the outside and each track representing an exposure; the outer track is BMI, the middle track is WHR, the inner track is BF. Solid points indicate a multiple testing threshold has been reached. BMI = body mass index; WHR = waist hip ratio; FFM = fat free mass.*
\esmall

\newpage
```{r plot-circos-supplement-adolescents, echo=FALSE, out.width='100%', fig.cap="Circos plot of effect estimates from a linear regression in ALSPAC adolescents, supplementary metabolites"}
knitr::include_graphics("data/chapter4/figures/circosplot_supplement_adolescents.pdf")
```
\noindent 
\bsmall
*Figure \@ref(fig:plot-circos-supplement-adolescents) shows effect estiamtes and 95% confidence intervals from a linear regression. Each point represents a single result, with the metabolites labelled around the outside and each track representing an exposure; the outer track is BMI, the middle track is BF. Solid points indicate a multiple testing threshold has been reached. BMI = body mass index; BF = body fat percentage.*
\esmall

\newpage
```{r plot-circos-supplement-young-adults, echo=FALSE, out.width='100%', fig.cap="Circos plot of effect estimates from a linear regression in ALSPAC young adults, supplementary metabolites"}
knitr::include_graphics("data/chapter4/figures/circosplot_supplement_young_adults.pdf")
```
\noindent 
\bsmall
*Figure \@ref(fig:plot-circos-supplement-young-adults) shows effect estiamtes and 95% confidence intervals from a linear regression. Each point represents a single result, with the metabolites labelled around the outside and each track representing an exposure; the outer track is BMI, the middle track is WHR, the inner track is BF. Solid points indicate a multiple testing threshold has been reached. BMI = body mass index; WHR = waist hip ratio; BF = body fat percentage.*
\esmall

\newpage
```{r plot-circos-supplement-adults, echo=FALSE, out.width='100%', fig.cap="Circos plot of effect estimates from a linear regression in ALSPAC adults, supplementary metabolites"}
knitr::include_graphics("data/chapter4/figures/circosplot_supplement_adults.pdf")
```
\noindent 
\bsmall
*Figure \@ref(fig:plot-circos-supplement-adults) shows effect estiamtes and 95% confidence intervals from a linear regression. Each point represents a single result, with the metabolites labelled around the outside and each track representing an exposure; the outer track is BMI, the middle track is WHR, the inner track is BF. Solid points indicate a multiple testing threshold has been reached. BMI = body mass index; WHR = waist hip ratio; BF = body fat percentage.*
\esmall

















## Chapter 5 {#chapter5-appendix}

\blandscape
```{r appendix-chapter5-table-metabolites, echo=FALSE}
data <- read.table("../../002_adiposity_metabolites/data/kettunen_metabolites.txt", header = T, sep = "\t")
data <- data[,c(7,8,9,10)]
data <- data[order(data[,3], data[,1]),]

# make table
if (doc.type == "docx") { 
x <- data
  } else { 

if (doc.type != "docx") { 
  x <- knitr::kable(
    (data), longtable = T, booktabs = T,
    caption = 'Metabolites from Kettunen et al (2016) used as outcomes in MR analysis', row.names = F,
    col.names = c("Abbreviation", "Label", "Class","subclass"))
  x <- kable_styling(x, full_width = T) 
} else {

}
}
x
```
\elandscape

\newpage
\blandscape
```{r appendix-chapter5-table-exposures, echo=FALSE}
data <- read.table("../../002_adiposity_metabolites/data/exposure_SNPs.txt", header = T, sep = "\t")
data <- data[order(data$Exposure, data$Ref, data$Note, data$Chr, data$BP),]
data <- subset(data, Exposure != "WHRadjBMI")
data$P <- as.character(data$P)
colnames(data)[7] <- c("B")
colnames(data)[10] <- c("Exp")
colnames(data)[13] <- c("Clump")

# make table ====
if (doc.type == "docx") {
x <- data
  } else {

if (doc.type != "docx") {
  x <- knitr::kable(
    (data), longtable = T, booktabs = T, digits = 3,
    caption = 'Genetic instrumental variables', row.names = F)
  x <- kable_styling(x, full_width = F)
} else {

}
}
x
```
\elandscape

#### F-statistics
```{r appendix-chapter5-figure-fstatistics, echo=FALSE, out.width='100%', fig.cap="f-statistics for all exposures used in chapter 5"}
knitr::include_graphics("../../002_adiposity_metabolites/analysis/all_exposures_fstatistics.pdf")
```
\noindent 
\bsmall
*Mean f-statistic for each exposure indicated by the pink diamond. The blue line indicates a nominal threshold of 10. Exposures used in the main analysis are highlighted with coloured points. The name after each exposure trait represents the authors last name of the original GWAS publication for which each exposure was obtained. The number following the first authors last name represents the number of SNPs obtained from the original GWAS and included; clumped refers to this original number of SNPs having been pruned based on an LD R^2^ of $0.001$. BMI = body mass index, WHR = wasit hip ratio, WHRadjBMI = wasit hip ratio adjusted for BMI, BF = body fat percentage.*
\esmall


#### Correlations {#chapter5-appendix-correlations}
Consistency in effect direction and size was investigated using Spearman's correlations.

```{r echo=FALSE, warning=FALSE, error=FALSE}
library(tidyr)
data <- read.table("../../002_adiposity_metabolites/analysis/step1/combined/001_combined_mr_results_kettunen.txt", header = T, sep ="\t")
ivw <- subset(data, method == "Inverse variance weighted (multiplicative random effects)")
main_exposures <- c("BMI_Yengo_941", "WHR_Pulit_316", "BF_Lu_7")
main <- ivw[ivw$exposure %in% main_exposures, ]
main <- main[,c(2,4,7)]
main <- spread(main, key = exposure, value = b)

# correlations ====
all <- main[,c(2,3,4)]
all_cor <- cor(all, use = "complete.obs", method = "spearman")

# show correlations for each model within exposures ====
print("Correlations between exposures")
all_cor
```




#### Sensitivity analysis
##### Method comparison
For all method comparison Circos plots the outer track is the main analysis (IVW-MRE) result, and is coloured for the exposure for consistency with the main text of the thesis (BMI = yellow, WHR = red, BF = blue). Purple = MR Egger, green = Weighted median, orange = Weighted mode. Solid points represent a *p-value* threshold of $0.05/22$ having been met; however only directional consistency between the methods is of interest when comparing the IVW-MRE model to the sensitivity models. <br>

```{r appendix-chapter5-figure-circosplot-sensitivity-main-BMI, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE, out.width='100%', fig.cap="Circosplot of main and sensitivity analysis for BMI"}
knitr::include_graphics("data/chapter5/figures/sensitivity_analysis_BMI.pdf")
```

```{r appendix-chapter5-figure-circosplot-sensitivity-main-WHR, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE, out.width='100%', fig.cap="Circosplot of main and sensitivity analysis for WHR"}
knitr::include_graphics("data/chapter5/figures/sensitivity_analysis_WHR.pdf")
```

```{r appendix-chapter5-figure-circosplot-sensitivity-main-BF, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE, out.width='100%', fig.cap="Circosplot of main and sensitivity analysis for BF"}
knitr::include_graphics("data/chapter5/figures/sensitivity_analysis_WHR.pdf")
```


##### Representative figures
```{r appendix-chapter5-figure-singlesnp-representative-figure, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE, out.width='100%', fig.cap="Single SNP MR results for BMI and glycerol - representative figure"}
knitr::include_graphics("data/chapter5/figures/singlesnp_representative_figure.pdf")
```

```{r appendix-chapter5-figure-funnelplot-BMI-representative-figure, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE, out.width='100%', fig.cap="Funnel plot of single SNP MR results for BMI and Apolipoprotein B - representative figure"}
knitr::include_graphics("data/chapter5/figures/funnelplot_BMI_APOB_representative_figure.pdf")
```

```{r appendix-chapter5-figure-funnelplot-BF-representative-figure, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE, out.width='100%', fig.cap="Funnel plot of single SNP MR results for BF and Apolipoprotein B - representative figure"}
knitr::include_graphics("data/chapter5/figures/funnelplot_BF_APOB_representative_figure.pdf")
```

```{r appendix-chapter5-figure-leaveoneout-BF-representative-figure, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE, out.width='100%', fig.cap="Leave-one-iut results for BF and Acetoacetate - representative figure"}
knitr::include_graphics("data/chapter5/figures/leaveoneout_BF_representative_figure.pdf")
```

```{r appendix-chapter5-figure-leaveoneout-BMI-representative-figure, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE, out.width='100%', fig.cap="Leave-one-iut results for BMI and Acetate - representative figure"}
knitr::include_graphics("data/chapter5/figures/leaveoneout_BMI_representative_figure.pdf")
```




## Chapter 6

#

This first appendix includes all of the R chunks of code that were hidden throughout the document (using the `include = FALSE` chunk tag) to help with readibility and/or setup.  

**In the main Rmd file**  

This thesis was created using R Markdown. The template used here I have adapted from the R Markdown template here (<https://github.com/ismayc/thesisdown>), which is adapted from the Markdown template (<https://github.com/matlipson/phd_thesis_markdown>), which is an extension of Tom Pollards original Markdown for thesis work (<https://github.com/tompollard/phd_thesis_markdown>).  



#

Appendix B contains all of the code used throughout the thesis that is hidden from the reader using the `include = FALSE` chunk tag. This code is not necessary for reading, but is included here for completeness and transparency, and consists mainly of package installs and code to produce tables and figures included in the text.

To create the finished thesis file in R Markdown an index.Rmd file is used to compile multiple .Rmd files together. The index.Rmd file contains the YAML header, along with the following R code that installs and loads the neccessary packages to compile the thesis: 
```{r ref.label=c("include_packages"), results='hide', echo = TRUE}
```


<!--
To include hidden R code used in Markdown files run the below code. Use =c("references", "for", "your", "code") to include multiple sections of hidden code.
```{r ref.label="include_packages", results='hide', echo = TRUE}
```
-->

## Code from Chapter \@ref(chapter1)

## Code from Chapter \@ref(chapter2)

## Code from Chapter \@ref(chapter3)

## Code from Chapter \@ref(chapter4)

## Code from Chapter \@ref(chapter5)

## Code from Chapter \@ref(chapter6)

## Code from Chapter \@ref(chapter7)

## Code from Chapter \@ref(chapter8)

