---
output: 
    thesisdown::thesis_pdf: default
bibliography: bib/thesis.bib
csl: csl/nature.csl
space_betwee_paragraphs: true
fig_caption: true
always_allow_html: yes
link-citations: true
toc-depth: 2
lot: true
lof: true
editor_options: 
  chunk_output_type: console
---

`r if(knitr:::is_latex_output()) '\\appendix'`

`r if(!knitr:::is_latex_output()) '# (APPENDIX) Appendix {-}'` 

<!--
If you feel it necessary to include an appendix, it goes here.
-->

```{r include=FALSE}
# word count = 
doc.type <- knitr::opts_knit$get('rmarkdown.pandoc.to') # this will tell the .Rmd file what output you are knitting to (word, pdf, html) so that you cna use if else statements when making tables - html/pdf output tables do not work well in word. For word we need to use the 'flextable' packaged to make tables.
setwd(Sys.getenv("main"))
## packages
library(dplyr)
library(kableExtra)
library(flextable)
library(data.table)
library(ggplot2)
library(cowplot)
library(tidyverse)
options(tinytex.verbose = TRUE)
```

# 

All tables and figures presented here are also available on [GitHub](https://github.com/mattlee821/000_thesis/tree/master/index/data) within each chapter's respective folder - [github.com/mattlee821/000_thesis/tree/master/index/data](https://github.com/mattlee821/000_thesis/tree/master/index/data). \par

#### Acknowledgements {-}
Work in this thesis would not have been possible without the contribution of all the families who took part in ALSPAC, the midwives for their help in recruiting them, and the whole ALSPAC team, which includes interviewers, computer and laboratory technicians, clerical workers, research scientists, volunteers, managers, receptionists and nurses. Thanks to: Adam Butterowtrh (Univeristy of Cambridge) for letting me use summary statistics from INTERVAL, Carolina Borges (University of Bristol) for UK Biobank nuclear magnetic resonance genome-wide association study summary statistics, and Emma Anderson (University of Bristol) for diet trajectory data in ALSPAC. \par


\newpage
## Chapter 1: Introduction {#appendix-introduction}
```{r include=FALSE}
setwd(Sys.getenv("main"))
load("index/data/introduction/data/data.RData")
load("index/data/introduction/data/melodi_data.RData")
```

#### Literature search {-}
In Chapter 1, a literature search was conducted to provide a summary of the relationships between adiposity and health outcomes. However, manual literature searching is prone to bias. Literature mining tools, though susceptible to publication and other biases, provide an alternative approach enabling a large number of articles to be assessed in a semi-systematic way. Mining Enriched Literature Objects to Derive Intermediates (MELODI)[@Elsworth2018], a literature mining tool, was used to identify intermediate diseases between body mass index (BMI) and mortality. \par

Briefly, MELODI creates individual article sets based on the search terms 'body mass index' and 'mortality', and looks for enriched overlapping terms (i.e., terms found in both article sets). MELODI uses PubMed and SemMedDB to identify enriched terms. PubMed is a database of health and biomedical research literature, and SemMedDB is a semantic predications repository built from PubMed citations. Identifying enriched overlapping terms is a two-step process. First, overlapping terms are identified. Second, the degree of overlap given the observed and expected frequency of terms across all articles (not just those included in the article sets) is quantified. These overlapping terms are considered potential intermediates and may highlight novel pathways for future experimentation. In this situation, MELODI was used to identify what intermediates may link body mass index (BMI) and mortality in an effort to summarise the literature. \par

All articles published from 01/01/2000--09/12/2019 (maximum number of articles per article set is 1,000,000) using the terms 'body mass index' as the source and 'mortality' as the outcome were included. Raw results are available on [GitHub](https://github.com/mattlee821/000_thesis/index/data/chapter1). A total of 187,951 and 787,451 articles were retrieved and included in the source and outcome article sets, respectively. Using the SemMedDB results, a total of 10,828 enriched overlapping terms were identified. This included similar terms to 'body mass index' as the source, which were removed (n = `r nrow(MELODI_bmi_clna)`). A Bonferroni corrected p-value < `r signif(0.05/nrow(MELODI_bmi_clna), 2)` (i.e., 0.05/`r nrow(MELODI_bmi_clna)`) removed `r nrow(MELODI_bmi_clna) - nrow(MELODI_bmi_clna_p)` terms. Terms were filtered for uniqueness (n = `r nrow(MELODI_bmi_clna_p_unique)`) and presence in the following categories: "age group" (n = `r nrow(MELODI_intermediates_unique_aggp)`), "bacterium" (n = `r nrow(MELODI_intermediates_unique_bact)`), "finding" (not an official MELODI category; n = `r nrow(MELODI_intermediates_unique_finding)`), "general" (not an official MELODI category; n = `r nrow(MELODI_intermediates_unique_general)`), "fungus" (n = `r nrow(MELODI_intermediates_unique_fngs)`), "health care activity" (n = `r nrow(MELODI_intermediates_unique_hlca)`), "human" (n = `r nrow(MELODI_intermediates_unique_humn)`), "injury or poisoning" (n = `r nrow(MELODI_intermediates_unique_inpo)`), "mammal" (animal studies; n = `r nrow(MELODI_intermediates_unique_mamm)`), "patient or disabled group" (n = `r nrow(MELODI_intermediates_unique_podg)`), "population group" (n = `r nrow(MELODI_intermediates_unique_popg)`), "sign or symptom" (n = `r nrow(MELODI_intermediates_unique_sosy)`), "virus" (n = `r nrow(MELODI_intermediates_unique_virs)`). \par

The following terms were removed and merged into "immunocompromised host" category: infections, hospital and opportunistic infections. The following terms were removed because of duplication under different names: cardiac death, sudden death, and sudden cardiac death (merged into "cessation of life"); coronary arteriosclerosis ("atherosclerosis");  cardiovascular morbidity, cardiac event, heart diseases, vascular diseases ("cardiovascular diseases"); depressed mood (depressive disorder); diabetes mellitus and insulin-dependent ("diabetic"); metabolic diseases ("metabolic syndrome"). The following terms were removed because they were top-level categories that could either include a wide variety of terms already included or were duplicated by other terms: "chronic disease", "critical illness", "disability", "pregnancy complications and perinatal morbidity" (included in "pregnancy" category), "pathogenesis." As a result of filtering, a total of `r nrow(MELODI_intermediates_unique_final)` terms remained. These terms were combined into 9 categories: cancer, cardiovascular, immune, kidney, liver, neurological/behavioural, pregnancy, respiratory, other. The 'other' category included traits that did not fit into one of any of the other categories and which did not have aligned traits to form a separate category (Table \@ref(tab:appendix-observational-table-MELODI)). The majority of intermediates were cardiovascular related. \par

It should be noted that the search did not include articles prior to the year 2000 and focussed only on those archived by PubMed. Enrichment aims to reduce the noise introduced when searching hundreds of thousands of articles, however manual curation, which will hold its own biases, is needed in order to obtain an informative list of enriched terms. \par

```{r appendix-observational-table-MELODI, echo=FALSE}
data <- MELODI_final_table
data <- data[order(data$Category),]
knitr::kable(data, longtable = T, booktabs = T, format = "latex", escape = T,
    caption = 'Mining enriched literature objects to derive intermediates for "body mass index" and "mortality"', 
    row.names = F,
    col.names = c("Intermediate", "Category")) %>%
  kable_styling(full_width = F, latex_options = c("striped", "HOLD_poistion", "repeat_header"), position = "center") %>%
  column_spec(1, width = "18em") %>%
  column_spec(2, width = "10em")
```

\newpage

## Chapter 2: Systematic review: What has the application of Mendelian randomization told us about the causal relevance of adiposity and health outcomes? {#appendix-SR}
In Chapter \@ref(systematic-review), quality assessment of individual studies was performed using a modified version of the quality assessment tool used by Mamluk et al., (2020)[@Mamluk2020]. This tool is presented in Table \@ref(tab:appendix-SR-table-rob) on the next page, and the results of this analysis are presented in Table \@ref(tab:appendix-SR-table-quality-assessment-results) on the following pages. Results form the meta-analyses are given in Table \@ref(tab:appendix-SR-table-meta-analyses-results). \par

\newpage
\thispagestyle{empty}
\vspace*{-3cm}

```{r appendix-SR-table-rob, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
data <- data.frame(title = c("Weak instrument bias",
                             "Genetic confounding bias",
                             "Other Confounding bias",
                             "Additional direct effects between IV and outcome",
                             "Bias due to selection of participants (exposure)",
                             "Bias due to selection of participants (outcome)",
                             "IV association",
                             "Sample overlap",
                             "Sensitivity analyses",
                             "Descriptive data",
                             "Data availability",
                             "Statistical parameters"),
                   Question = c("Strength of association between instrument and exposure",
                                "Reported test on association between confounders and IV",
                                "Included confounders in the IV analysis",
                                "Presence of pleiotropy for IV tested",
                                "Homogenous population or similar ancestry?",
                                "Homogenous population or similar ancestry?",
                                "Provide information on the similarity of IV in exposure/outcome. Two-sample MR: same allele frequency, same ancestry etc.",
                                "Provide information on sample overlap",
                                "Have they performed sensitivity analyses.",
                                "Is information on the methodology missing/incorrect.",
                                "Is the  methodology reproducible?",
                                "Are the statistical parameters for the analysis given"),
                   High = c("F less than 10",
                            "Yes AND there is an obvious association",
                            "Yes (lifestyle factors)",
                            "No test/investigation performed or not discussed",
                            "non-homogenous population",
                            "non-homogenous population",
                            "Different ancestry",
                            "Not reported/assumed overlap",
                            "Not reported / not performed",
                            "Majority missing or incorrect - key info such as MR estimator and/or N SNPs missing",
                            "No code provided and no software/packages referenced",
                            "No"),
                   Moderate = c("F missing",
                                "Not presented or presented and there is some degree of association",
                                "Not discussed (one-sample/meta-analysis)",
                                "Post-hoc sensitivitiy analysis",
                                "population described as homogenous BUT no ancestry covariate included",
                                "population described as homogenous BUT no ancestry covariate included",
                                "Not reported/Assumed same underlying ancestry",
                                "Not reported/assumed no overlap",
                                "Reported",
                                "Some information missing such as N for exposure/outcome",
                                "Software/packages referenced",
                                "Some"),
                   Low = c("F greater than 10",
                           "Presented and no obvious association",
                           "No",
                           "Excluded/re-ran after exlcusion of pleiotropic SNPs / sensitivity analysis consistent with main analysis",
                           "population described as homogenous AND ancestry covariate included",
                           "population described as homogenous AND ancestry covariate included",
                           "Tested and same ancestry",
                           "Reported/no overlap",
                           "Reported in the context of main analysis / can't do it (e.g. one SNP used)",
                           "Majority of information presented and correct",
                           "Code is provided and all software/packages are referenced",
                           "Yes/NA because 1 SNP/directly genotyped"))

data <- tibble::rownames_to_column(data) %>%  
   pivot_longer(-rowname) %>% 
   pivot_wider(names_from=rowname, values_from=value) 
names(data) <- data[1,]
data <- data[-1,]

knitr::kable(data, longtable = T, booktabs = T, format = "latex", escape = T,
    caption = 'Quality assessment tool', 
    row.names = F,
    col.names = c(" ", "Weak instrument bias",
                  "Genetic confounding bias",
                  "Other Confounding bias",
                  "Additional direct effects between IV and outcome (exclusion restriction assumption)",
                  "Bias due to selection of participants (exposure)",
                  "Bias due to selection of participants (outcome)",
                  "IV association",
                  "Sample overlap",
                  "Senisitivity analyses",
                  "Descriptive data",
                  "Data availability",
                  "Statistical parameters")) %>%
  kable_styling(full_width = F, latex_options = c("striped", "HOLD_poistion"), position = "center", font_size = 6) %>%
  column_spec(1, width = "3em") %>%
  column_spec(2, width = "7em") %>%
  column_spec(3, width = "7em") %>%
  column_spec(4, width = "7em") %>%
  column_spec(5, width = "7em") %>%
  column_spec(6, width = "7em") %>%
  column_spec(7, width = "7em") %>%
  column_spec(8, width = "7em") %>%
  column_spec(9, width = "7em") %>%
  column_spec(10, width = "7em") %>%
  column_spec(11, width = "7em") %>%
  column_spec(12, width = "7em") %>%
  column_spec(13, width = "7em") %>%
  landscape()
```

\newpage
\blandscape
\thispagestyle{empty}
\vspace*{-3cm}

```{r appendix-SR-table-quality-assessment-results, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
data <- read.csv("../../005_systematic_review/analysis/quality_assessment/quality_assessment_results.csv", header = T)
data <- data[,c(2:18)]
data <- data[order(data$year, data$author),]
knitr::kable(data, longtable = T, booktabs = T, format = "latex", escape = T,
    caption = 'Quality assessment results for studies included in meta-analyses', 
    row.names = F,
    col.names = c("Meta-analysis", "Author", "Year", "doi", "1","2","3","4","5","6","7","8","9","10","11","12","Total")) %>%
    kable_styling(full_width = F, latex_options = c("striped", "HOLD_poistion", "repeat_header"), position = "center", font_size = 8) %>%
    kableExtra::footnote(general = "Meta-analysis indicates the meta-analysis the study contributed to. Columns 2, 3, and 4 give the first authors last name, year of publication and doi of the article. The remaining columns give the quality assessment tool question number and the total score for that study. BMI = body mass index; WHR = waist hip ratio; WHRadjBMI = WHR adjusted for BMI; SD = standard deviation; HOMA IR = Homoeostatic Model Assessment for Insulin Resistance; HbA1c = Hemoglobin A1C; HDL = high density lipoprotein; LDL = low density liporpotein; SBP = systolic blood pressure; ER- = oestrogen reseptor negative; CAD = coronary artery disease; PCOS = polycystic ovary syndrome", threeparttable = T, general_title = "", footnote_as_chunk = T) 

```
\elandscape

\newpage
\blandscape
\thispagestyle{empty}
\vspace*{-3cm}

```{r appendix-SR-table-meta-analyses-results, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
data <- read.csv("../../005_systematic_review/analysis/meta_analysis/results/meta_analysis_results_formatted_manual_appendix_latex.csv")
data$effect_estimate <- as.character(data$effect_estimate)
data$CI_lower <- as.character(data$CI_lower)
data$CI_upper <- as.character(data$CI_upper)
data$p <- as.character(data$p)
data$q <- as.character(data$q)
data$q_p <- as.character(data$q_p)
data$tau2 <- as.character(data$tau2)
data$tau2_se <- as.character(data$tau2_se)
data$tau <- as.character(data$tau)
data$h <- as.character(data$h)
data$i1 <- as.character(data$i1)

knitr::kable(data, longtable = T, booktabs = T, format = "latex", escape = T,
    caption = 'Meta-analyses results from Mendelian randomization analyses using adiposity as an exposure', 
    row.names = F, 
    col.names = c("Meta-analysis", "Study reference", "B/OR", "Lower CI", "Upper CI", "p-value", "q", "q_df", "q_p", "tau2", "se", "tau", "h","i2")) %>%
    kable_styling(full_width = F, latex_options = c("striped", "HOLD_poistion", "repeat_header"), position = "center", font_size = 6) %>%
  column_spec(1, width = "15em") %>%
  column_spec(2, width = "20em") %>%
  column_spec(3, width = "4em") %>%
  column_spec(4, width = "4em") %>%
  column_spec(5, width = "4em") %>%
  column_spec(6, width = "4em") %>%
  column_spec(7, width = "4em") %>%
  column_spec(8, width = "4em") %>%
  column_spec(9, width = "4em") %>%
  column_spec(10, width = "4em") %>%
  column_spec(11, width = "4em") %>%
  column_spec(12, width = "4em") %>%
  column_spec(13, width = "4em") %>%
  column_spec(14, width = "4em") %>%
  kableExtra::footnote(general = "Column 1 gives the exposure and outcome with units. Column 2 gives the studies used in the meta-analysis. B/OR = beta is the change in outcome per unit increase in exposure, OR is the odds ratio of the outcome per unit change in the exposure; CI = 95% confidence interval; q = heterogeneity statistic; q_df = degrees of freedom of Q; tau_2 = between study variance; se = standard error of tau_2; h = heterogeneity statistic within subgroups; i1 = heterogeneity statistic within subgroups; BMI = body mass index; WHR = waist hip ratio; WHRadjBMI = WHR adjusted for BMI; SD = standard deviation; HOMA IR = Homoeostatic Model Assessment for Insulin Resistance; HbA1c = Hemoglobin A1C; HDL = high density lipoprotein; LDL = low density liporpotein; SBP = systolic blood pressure; ER- = oestrogen reseptor negative; CAD = coronary artery disease; PCOS = polycystic ovary syndrome", threeparttable = T, general_title = "", footnote_as_chunk = T)
```

\elandscape

## Chapter 4: Associations between multiple measures of adiposity and metabolites: observational analysis {#appendix-observational}
In Chapter \@ref(observational), data from the Avon Longitudinal Study of Parents and Children (ALSPAC) was used to investgate the association between adiposity measures and nuclear magnetic resonance (NMR) derived metabolites and ratios. The following sections provide detail on ALSPAC, validation of Equation \@ref(eq:BF) with raw bioeletrical impedance values, and correlation coefficients (Searmans Rho) of the comparison of effect estimates from different models. \par

### Methods
#### Data overview
ALSPAC[@Fraser2013; @Boyd2013; @Northstone2019] is a prospective cohort study that invited women resident in Avon, UK with expected dates of delivery between 1st April 1991 and 31st December 1992 to participate. The initial number of pregnancies enrolled was 14,541 (for these at least one questionnaire was returned or a “Children in Focus” clinic has been attended by 19/07/99). Of these initial pregnancies, a total of 14,676 foetuses, resulted in 14,062 live births and 13,988 children alive at one year of age. The mothers and fathers associated with each pregnancy are referred to as generation 0 (G0) while the children of each eligible pregnancy (including individuals from subsequent recruitment drives) are referred to as generation 1 (G1). \par

Data in ALSPAC are split by clinic visits. For work in Chapter \@ref(obsevational), metabolomic data were available for G1 individuals from the following clinics: Focus at 7 (~8 years old), Focus at 8 (~9 years old), Before Breakfast Study (~8 years old), Teen Focus 3 (~18 years old), Teen Focus 4 (~17 years old), and Focus at 24 (~24 years old). Metabolomic data for G0 individuals were available from: Focus on Mothers 1 (~48 years old), Focus on Mothers 2 (~51 years old), and Focus on Fathers 1 (~53 years old). In order to maximize the sample size at each metabolomic clinic, data were combined where clinics were within a similar age range. Data were combined in the following groups for G1 individuals: Focus at 7 and Before Breakfast Study (children), and Teen Focus 3 and Teen Focus 4 (adolescents). The Focus at 24 clinic is referred to as young adults. For G0 individuals data from Focus on Mothers 1, Focus on Mothers 2, and Focus on Fathers 1 were combined into an adults group. For these combined data sets, duplicate individuals (i.e., those attending both clinics) were identified, and the measurement from the most recent clinic was dropped. \par

#### Validation of raw impedance as a measure of body fat percentage 
In children, a measure for BF was not available. Instead, impedance values were used to estimates BF using equation \@ref(eq:BF). The calculated BF resulted in negative BF values for some children (Figure \@ref(fig:appendix-observational-figure-BF-validation-distribution)). However, the calculated BF was positively correlated with weight, height and BMI in children. In adolescents, the calculated BF did not produce negative values and was positively correlated with impedance and dual energy X-ray absorptiometry (DXA) derived BF values (Figure \@ref(fig:appendix-observational-figure-BF-validation-correlation-2-adolescents)), as well as with weight, height, and BMI (Figure \@ref(fig:appendix-observational-figure-BF-validation-correlation)). Child calculated BF positively correlated with adolescent measures of BF (Figure \@ref(fig:appendix-observational-figure-BF-validation-correlation-2-children)). \par

Given that the equation was derived from adult (i.e., there is a volumetric difference adults and children[@Stergiakouli2014]), differences in the range of values is unsurprising. There is however strong correlation between calculated-BF and impedance-derived and DXA-derived values of BF in adolescents. As the estimates derived in linear models are in reference to the per unit increase in an exposure (rather than the range), the absolute value of the exposure does not need to positive. Thus, the negative values of BF will not impact the inference of a linear regression between BF and any metabolite. As such, BF calculated using equation \@ref(eq:BF) was used in subsequent analyses as a measure of BF in children. \par

(ref:appendix-observational-figure-BF-validation-distribution-cap) **Distribution of raw impedance and calculated body fat percentage in children and adolescents**. Plots give the distribution of the raw impedance value measured in ohms and body fat percentage (BF) derived using equation \@ref(eq:BF) for children and adolescents. Data is presented for complete cases across: raw metabolomics, body mass index, waist hip ratio (children only), impedance, height, weight, age, sex, and BF derived by impedance and dual energy x-ray absorptiometry in adolescents. Red is female data, grey is male data.

(ref:appendix-observational-figure-BF-validation-distribution-scap) Distribution of raw impedance and calculated body fat percentage in children and adolescents

```{r appendix-observational-figure-BF-validation-distribution, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-observational-figure-BF-validation-distribution-cap)', fig.scap='(ref:appendix-observational-figure-BF-validation-distribution-scap)'}
knitr::include_graphics("data/observational/figures/bf_validation_histogram.pdf")
```

(ref:appendix-observational-figure-BF-validation-correlation-cap) **Correlation between different body fat percentage measures in children and adolescents**. Scatter plots are presented alongside Pearson's correlation coefficients for each sex; data for males are in grey and females in red. Plots are arranged in 4 columns and 4 rows. Column 1 gives raw impedance value in children (y axis) with body fat percentage (BF)-calculated in children and adolescents using equation \@ref(eq:BF), and BF derived from dual energy x-ray absorptiometry (DXA) in adolescents. Column 2 gives child-calculated BF using equation \@ref(eq:BF) (y axis) with adolescent-calculated BF using equation \@ref(eq:BF) and DXA-derived BF. Column 3 gives adolescent raw impedance (y axis) with adolescent BF-calculated using equation \@ref(eq:BF) and DXA-derived BF. Column 4 gives adolescent calculated BF using equation \@ref(eq:BF) with adolescent raw impedance, DXA-derived BF, and adolescent raw impedance with DXA-derived BF. Data is presented for complete cases across: raw metabolomics, body mass index, waist hip ratio (children only), impedance, height, weight, age, sex, and BF derived by impedance and DXA in adolescents. 

(ref:appendix-observational-figure-BF-validation-correlation-scap) Correlation between different body fat percentage measures in children and adolescents

```{r appendix-observational-figure-BF-validation-correlation, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-observational-figure-BF-validation-correlation-cap)', fig.scap='(ref:appendix-observational-figure-BF-validation-correlation-scap)'}
knitr::include_graphics("data/observational/figures/bf_validation_correlation_bf.pdf")
```

(ref:appendix-observational-figure-BF-validation-correlation-2-children-cap) **Correlation between body fat percentage and anthropometric measures in children.** Scatter plots are presented alongside Pearson's correlation coefficients for each sex; data for males are in grey and females in red. Column 1 = impedance; column 2 = body fat percentage (BF). Impedance is given as ohms. Data is presented for complete cases across: QC'd metabolomics, body mass index, waist hip ratio (children only), impedance, height, weight, age, sex, and BF derived by impedance and DXA in adolescents. 

(ref:appendix-observational-figure-BF-validation-correlation-2-children-scap) Correlation between body fat percentage and anthropometric measures in children and adolescents

```{r appendix-observational-figure-BF-validation-correlation-2-children, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-observational-figure-BF-validation-correlation-2-children-cap)', fig.scap='(ref:appendix-observational-figure-BF-validation-correlation-2-children-scap)'}
knitr::include_graphics("data/observational/figures/bf_validation_correlation_children.pdf")
```

(ref:appendix-observational-figure-BF-validation-correlation-2-adolescents-cap) **Correlation between body fat percentage and anthropometric measures in adolescents.** Scatter plots are presented alongside Pearson's correlation coefficients for each sex; data for males are in grey and females in red. Column 1 = impedance; column 2 = BF calculated using equation \@ref(eq:BF); column 3 = BF derived from impedance device; column 4 = BF derived from dual energy x-ray absorptiometry (DXA). Impedance is given as ohms. Data is presented for complete cases across: QC'd metabolomics, body mass index, waist hip ratio (children only), impedance, height, weight, age, sex, and BF derived by impedance and DXA in adolescents. 

(ref:appendix-observational-figure-BF-validation-correlation-2-adolescents-scap) Correlation between body fat percentage and anthropometric measures in adolescents

```{r appendix-observational-figure-BF-validation-correlation-2-adolescents, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-observational-figure-BF-validation-correlation-2-adolescents-cap)', fig.scap='(ref:appendix-observational-figure-BF-validation-correlation-2-adolescents-scap)'}
knitr::include_graphics("data/observational/figures/bf_validation_correlation_adolescents.pdf")
```

<!-- #### Metabolites {#appendix-observational-metabolites} -->

<!-- ```{r appendix-observational-table-metabolites, echo=FALSE} -->
<!-- data <- read.table("data/observational/tables/metabolites.txt", header = T, sep = "\t") -->
<!-- data <- data[order(data$subclass, data$raw.label),] -->

<!-- knitr::kable(data, longtable = T, booktabs = T, format = "latex", escape = T, digits = 3, -->
<!--     caption = 'List of metabolites available in the Avon Longitudinal Study of Parents and Children', -->
<!--     row.names = F, -->
<!--     col.names = c("Abbreviation", "Metabolite", "Class", "Subclass", "Derived")) %>% -->
<!--   kable_styling(full_width = T, latex_options = c("striped", "HOLD_poistion", "repeat_header"), position = "center", font_size = 8) %>% -->
<!--   kableExtra::footnote(general = "Metabolites are given with the originally measured units along with the Class and Subclass provided by the metabolomics platform and whether the metabolite is directly measured (no) or a derived metabolite measure.", threeparttable = T, general_title = "", footnote_as_chunk = T) -->
<!-- ``` -->

\newpage

#### Outcomes
```{r appendix-observational-table-metabolites, echo=FALSE}
data <- read.table("data/observational/tables/metabolites.txt", header = T, sep = "\t")
data <- data[order(data$class, data$subclass, data$raw.label),]
knitr::kable(data, longtable = T, booktabs = T, format = "latex", escape = T,
    caption = 'Metabolites used in Mendelian randomization analyses and meta-analyses', 
    row.names = F,
    col.names = c("Metabolite","Label", "Class","Subclass", "Derived")) %>%
  kable_styling(full_width = T, latex_options = c("striped", "HOLD_poistion", "repeat_header"), position = "center", font_size = 8) %>%
  column_spec(column = 1, width = "5em") %>%
  column_spec(column = 2, width = "20em") %>%
  column_spec(column = 3, width = "12em") %>%
  column_spec(column = 4, width = "15em") %>%
  column_spec(column = 5, width = "3em") %>%
  kableExtra::footnote(general = "Table gives the metabolite abbreviation, label with originally measured units, class and subclass the metabolites were grouped in, and whether the metabolite is a derived measure such as a ratio. HDL = high density lipoprotein; LDL = low density lipoprotein; IDL = intermediate density lipoprotein; VLDL = very large low density lipoprotein.", threeparttable = T, general_title = "", footnote_as_chunk = T) 
```

### Results
#### Correlations {#appendix-observational-correlations} 
Due to the large number of tests performed, concordance between and within models, exposures, and age groups was investigated using Spearman's correlations. Results are separated into three sections (i) for correlations across models within exposures and age groups (e.g., correlation between model 1 and 2 for the effect of BMI on metabolites in children), (ii) for correlations across exposures within age groups for model 2 (e.g., correlation between the effect of BMI on metabolites and WHR on metabolites in children), and (iii) for correlations across exposures within age groups (e.g., correlation between the effect of BMI on metabolites in children and the effect of BMI on metabolites in adolescents).

##### Correlation results across models within exposures (BMI, WHR, and BF) and within age groups (children, adolescents, young adults, and adults):
```{r echo=FALSE, warning=FALSE, error=FALSE}
library(tidyr)
data <- read.table("data/observational/data/analysis/results/combined/combined.txt", header = T, sep = "\t")

# models within groups within exposures
## children ====
a <- subset(data, group == "children")
bmi <- subset(a, exposure == "bmi")
bmi <- bmi[,c(1,3,16)]
bmi <- spread(bmi, key = model, value = b)
bmi <- bmi[,c(2:3)]
children_bmi_cor <- cor(bmi, use = "complete.obs", method = "spearman")

whr <- subset(a, exposure == "whr")
whr <- whr[,c(1,3,16)]
whr <- spread(whr, key = model, value = b)
whr <- whr[,c(2:3)]
children_whr_cor <- cor(whr, use = "complete.obs", method = "spearman")

bf <- subset(a, exposure == "bf")
bf <- bf[,c(1,3,16)]
bf <- spread(bf, key = model, value = b)
bf <- bf[,c(2:3)]
children_bf_cor <- cor(bf, use = "complete.obs", method = "spearman")

## adolescents ====
a <- subset(data, group == "adolescents")
bmi <- subset(a, exposure == "bmi")
bmi <- bmi[,c(1,3,16)]
bmi <- spread(bmi, key = model, value = b)
bmi <- bmi[,c(2:4)]
adolescents_bmi_cor <- cor(bmi, use = "complete.obs", method = "spearman")
whr <- ""

bf <- subset(a, exposure == "bf")
bf <- bf[,c(1,3,16)]
bf <- spread(bf, key = model, value = b)
bf <- bf[,c(2:4)]
adolescents_bf_cor <- cor(bf, use = "complete.obs", method = "spearman")

## young_adults ====
a <- subset(data, group == "young_adults")
bmi <- subset(a, exposure == "bmi")
bmi <- bmi[,c(1,3,16)]
bmi <- spread(bmi, key = model, value = b)
bmi <- bmi[,c(2:4)]
young_adults_bmi_cor <- cor(bmi, use = "complete.obs", method = "spearman")

whr <- subset(a, exposure == "whr")
whr <- whr[,c(1,3,16)]
whr <- spread(whr, key = model, value = b)
whr <- whr[,c(2:4)]
young_adults_whr_cor <- cor(whr, use = "complete.obs", method = "spearman")

bf <- subset(a, exposure == "bf")
bf <- bf[,c(1,3,16)]
bf <- spread(bf, key = model, value = b)
bf <- bf[,c(2:4)]
young_adults_bf_cor <- cor(bf, use = "complete.obs", method = "spearman")

## adults ====
a <- subset(data, group == "adults")
bmi <- subset(a, exposure == "bmi")
bmi <- bmi[,c(1,3,16)]
bmi <- spread(bmi, key = model, value = b)
bmi <- bmi[,c(2:4)]
adults_bmi_cor <- cor(bmi, use = "complete.obs", method = "spearman")

whr <- subset(a, exposure == "whr")
whr <- whr[,c(1,3,16)]
whr <- spread(whr, key = model, value = b)
whr <- whr[,c(2:4)]
adults_whr_cor <- cor(whr, use = "complete.obs", method = "spearman")

bf <- subset(a, exposure == "bf")
bf <- bf[,c(1,3,16)]
bf <- spread(bf, key = model, value = b)
bf <- bf[,c(2:4)]
adults_bf_cor <- cor(bf, use = "complete.obs", method = "spearman")

# model 1 within group across exposures
## children ====
a <- subset(data, group == "children")
a <- subset(a, model == "model2")
a <- a[,c(1,3,14)]
a <- spread(a, key = exposure, value = b)
a <- a[,c(2:4)]
children_m1_cor <- cor(a, use = "complete.obs", method = "spearman")
## adolescents ====
a <- subset(data, group == "adolescents")
a <- subset(a, model == "model2")
a <- a[,c(1,3,14)]
a <- spread(a, key = exposure, value = b)
a <- a[,c(2:3)]
adolescents_m1_cor <- cor(a, use = "complete.obs", method = "spearman")
## young_adults ====
a <- subset(data, group == "young_adults")
a <- subset(a, model == "model2")
a <- a[,c(1,3,14)]
a <- spread(a, key = exposure, value = b)
a <- a[,c(2:4)]
young_adults_m1_cor <- cor(a, use = "complete.obs", method = "spearman")
## adults ====
a <- subset(data, group == "adults")
a <- subset(a, model == "model2")
a <- a[,c(1,3,14)]
a <- spread(a, key = exposure, value = b)
a <- a[,c(2:4)]
adults_m1_cor <- cor(a, use = "complete.obs", method = "spearman")

# model 1 within exposure across groups
## bmi ====
a <- subset(data, exposure == "bmi")
a <- subset(a, model == "model2")
a <- a[,c(1,3,15)]
a <- spread(a, key = group, value = b)
a <- a[,c(2:5)]
bmi_m1_cor <- cor(a, use = "complete.obs", method = "spearman")
## whr ====
a <- subset(data, exposure == "whr")
a <- subset(a, model == "model2")
a <- a[,c(1,3,15)]
a <- spread(a, key = group, value = b)
a <- a[,c(2:4)]
whr_m1_cor <- cor(a, use = "complete.obs", method = "spearman")
## bf ====
a <- subset(data, exposure == "bf")
a <- subset(a, model == "model2")
a <- a[,c(1,3,15)]
a <- spread(a, key = group, value = b)
a <- a[,c(2:5)]
bf_m1_cor <- cor(a, use = "complete.obs", method = "spearman")
rm(a,bf,bmi,data,whr)

# show correlations for each model within exposures ====
print("Children BMI")
children_bmi_cor
print("Children WHR")
children_whr_cor
print("Children BF")
children_bf_cor

print("Adolescents BMI")
adolescents_bmi_cor
print("Adolescents BF")
adolescents_bf_cor

print("Young adults BMI")
young_adults_bmi_cor
print("Young adults WHR")
young_adults_whr_cor
print("Young adults BF")
young_adults_bf_cor

print("Adults BMI")
adults_bmi_cor
print("Adults WHR")
adults_whr_cor
print("Adults BF")
adults_bf_cor

```

##### Correlation results across exposures (BMI, WHR, and BF) within age groups (children, adolescents, young adults, and adults) for model 2:
```{r echo=FALSE, warning=FALSE, error=FALSE}
# show correlations for model 1 across exposures ====
print("Children")
children_m1_cor

print("Adolescents")
adolescents_m1_cor

print("Young adults")
young_adults_m1_cor

print("Adults")
adults_m1_cor
```

##### Correlation results within exposures (BMI, WHR, and BF) across age age groups (children, adolescents, young adults, and adults) for model 2:
```{r echo=FALSE, warning=FALSE, error=FALSE}
# show correlaion for model 1 within exposures across age groups ====
print("BMI")
bmi_m1_cor

print("WHR")
whr_m1_cor

print("BF")
bf_m1_cor

```

\newpage

#### Distribution of adiposity measures

(ref:appendix-observational-figure-adiposity-distribution-cap) **Distribution of adiposity measures across all age groups**. Plots give the distribution of all adiposity measures across age groups by sex. Body fat percentage (BF) in children is derived from equation \@ref(eq:BF). BMI = body mass index; WHR = waist hip ratio. Data is presented for all individuals given in Table \@ref(tab:observational-table-ALSPAC-adiposity). The interquartile range and median are also shown. Raincloud plots produced using the `RainCloudPlots` `R` package[@Allen2019].

(ref:appendix-observational-figure-adiposity-distribution-scap) Distribution of adiposity measures across all age groups

```{r appendix-observational-figure-adiposity-distribution, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:observational-figure-adiposity-distribution-cap)', fig.scap='(ref:observational-figure-adiposity-distribution-scap)'}
knitr::include_graphics("data/observational/figures/plot_histogram_adiposity_measures.pdf")
```

#### Linear regression results 
The following two forest plots show the results from linear regression analysis for all exposures and age groups for model 2 (age, sex, maternal or own education, smoking status, alcohol consumption, diet (calories consumed per day)). The first plot shows all directly measured metabolites; the second plot shows derived metabolite measures such as ratios. Four Circos plots are presented after this; each Circos plot shows the results from linear regression analysis for all exposures within an age group for model 2. \par

###### Forest plots {#appendix-observational-figures-forestplots}
(ref:appendix-observational-figure-forestplot-main-cap) **Linear regression analysis: association between adiposity measures and directly measured metabolites**. Forest plot shows effect estimates and 95% confidence intervals from model 2 (age, sex, maternal or own education, smoking status, alcohol consumption, diet (calories consumed per day)) for all exposures and age groups. Units represent the unit change in each metabolite per standard deviation change in the exposure.

(ref:appendix-observational-figure-forestplot-main-scap) Linear regression analysis: association between adiposity measures and metabolites

\blandscape
```{r appendix-observational-figure-forestplot-main, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-observational-figure-forestplot-main-cap)', fig.scap='(ref:appendix-observational-figure-forestplot-main-scap)'}
knitr::include_graphics("data/observational/appendix/forestplot_main.pdf")
```
\elandscape

(ref:appendix-observational-figure-forestplot-derived-cap) **Linear regression analysis: association between adiposity measures and derived metabolites**. Forest plot shows effect estimates and 95% confidence intervals from model 2 (age, sex, maternal or own education, smoking status, alcohol consumption, diet (calories consumed per day)) for all exposures and age groups. Units represent the unit change in each metabolite per standard deviation change in the exposure.

(ref:appendix-observational-figure-forestplot-derived-scap) Linear regression analysis: association between adiposity measures and metabolites

\blandscape
```{r appendix-observational-figure-forestplot-derived, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-observational-figure-forestplot-derived-cap)', fig.scap='(ref:appendix-observational-figure-forestplot-derived-scap)'}
knitr::include_graphics("data/observational/appendix/forestplot_supplement.pdf")
```
\elandscape

##### Circos plots {#appendix-observational-figures-circosplots}

(ref:appendix-observational-figure-circos-children-cap) **Effect estimates from linear regression of adiposity measures on derived supplementary metabolites in children**. Circos plot shows effect estimates and 95% confidence intervals from model 2 for all exposures in children. Effect estimates are absolute change in metabolite per-standard deviation increase in exposure. The outer track is body mass index (BMI), the middle track is waist hip ratio (WHR), the inner track is body fat percentage (BF). Solid points indicate a multiple testing threshold has been met; multiple testing threshold (0.05/54) set as the number of independent metabolite features as calculated by `metaboprep`.

(ref:appendix-observational-figure-circos-children-scap) Effect estimates from linear regression of adiposity measures on derived supplementary metabolites in children

```{r appendix-observational-figure-circos-children, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-observational-figure-circos-children-cap)', fig.scap='(ref:appendix-observational-figure-circos-children-scap)'}
knitr::include_graphics("data/observational/figures/circosplot_derived_metabolites_children.pdf")
```

(ref:appendix-observational-figure-circos-adolescents-cap) **Effect estimates from linear regression of adiposity measures on derived supplementary metabolites in adolescents**. Circos plot shows effect estimates and 95% confidence intervals from model 2 for all exposures in adolescents Effect estimates are absolute change in metabolite per-standard deviation increase in exposure. The outer track is body mass index (BMI), the inner track is body fat percentage (BF). Solid points indicate a multiple testing threshold has been met; multiple testing threshold (0.05/48) set as the number of independent metabolite features as calculated by `metaboprep`.

(ref:appendix-observational-figure-circos-adolescents-scap) Effect estimates from linear regression of adiposity measures on derived supplementary metabolites in adolescents

```{r appendix-observational-figure-circos-adolescents, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-observational-figure-circos-adolescents-cap)', fig.scap='(ref:appendix-observational-figure-circos-adolescents-scap)'}
knitr::include_graphics("data/observational/figures/circosplot_derived_metabolites_adolescents.pdf")
```

(ref:appendix-observational-figure-circos-young-adults-cap) **Effect estimates from linear regression of adiposity measures on derived supplementary metabolites in young adults**. Circos plot shows effect estimates and 95% confidence intervals from model 2 for all exposures in young adults Effect estimates are absolute change in metabolite per-standard deviation increase in exposure. The outer track is body mass index (BMI), the middle track is waist hip ratio (WHR), the inner track is body fat percentage (BF). Solid points indicate a multiple testing threshold has been met; multiple testing threshold (0.05/46) set as the number of independent metabolite features as calculated by `metaboprep`.

(ref:appendix-observational-figure-circos-young-adults-scap) Effect estimates from linear regression of adiposity measures on derived supplementary metabolites in young adults

```{r appendix-observational-figure-circos-young-adults, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-observational-figure-circos-young-adults-cap)', fig.scap='(ref:appendix-observational-figure-circos-young-adults-scap)'}
knitr::include_graphics("data/observational/figures/circosplot_derived_metabolites_young_adults.pdf")
```


(ref:appendix-observational-figure-circos-adults-cap) **Effect estimates from linear regression of adiposity measures on derived supplementary metabolites in adults**. Circos plot shows effect estimates and 95% confidence intervals from model 2 for all exposures in adults Effect estimates are absolute change in metabolite per-standard deviation increase in exposure. The outer track is body mass index (BMI), the middle track is waist hip ratio (WHR), the inner track is body fat percentage (BF). Solid points indicate a multiple testing threshold has been met; multiple testing threshold (0.05/53) set as the number of independent metabolite features as calculated by `metaboprep`.

(ref:appendix-observational-figure-circos-adults-scap) Effect estimates from linear regression of adiposity measures on derived supplementary metabolites in adults

```{r appendix-observational-figure-circos-adults, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-observational-figure-circos-adults-cap)', fig.scap='(ref:appendix-observational-figure-circos-adults-scap)'}
knitr::include_graphics("data/observational/figures/circosplot_derived_metabolites_adults.pdf")
```


## Chapter 5: Associations between multiple measures of adiposity and metabolites: Mendelian randomization analysis {#appendix-MR}
In Chapter \@ref(MR), the association between adiposity measures and NMR derived metabolites and ratios was investigated using MR. Summary statistics from multiple genome-wide association studies (GWAS) were used to obtain genetic variants (single nucleotide polymorphisms; SNPs). The following sections on exposures detail the GWAS used in additional analyses in Chapter \@ref(MR) which aimed to test the validity of instruments used in the main analysis. The outcomes section gives a table of metabolites used in these analyses. The results sections present figures for the sensitivity and additional analyses described and summarised in the main text of Chapter \@ref(MR). \par

### Methods
#### Exposure instruments {#appendix-MR-exposures}
In the main analysis, BMI, WHR, and BF were instrumented using SNPs from Yengo et al. (2018)[@Yengo2018], Pulit et al. (2019)[@Pulit2019], and Lu et al. (206)[@Lu2016], respectively. Additional instrument lists were obtained for each of the adiposity measures. A list of all instruments used is available on [GitHub](https://github.com/mattlee821/000_thesis/blob/master/index/data/MR/tables/exposure_SNPs.txt). The following details the GWAS from which summary statistics for these additional SNP lists were obtained. \par

```{r appendix-MR-table-exposures, echo=FALSE}
data <- read.table("../../002_adiposity_metabolites/data/exposure_SNPs.txt", header = T, sep = "\t")
# data <- data[order(data$Exposure, data$Ref, data$Note, data$Chr, data$BP),]
# data$P <- as.character(data$P)
# 
# knitr::kable(data, longtable = T, booktabs = T, format = "latex", escape = T, digits = 3,
#     caption = 'Genetic instrumental variables used in Mendelian randomization analyses', 
#     row.names = F,
#     col.names = c("SNP", "Chr", "BP", "EA", "NEA", "EAF", "Beta", "SE", "P", "Exposure", "Ref", "Note", "Clumped")) %>%
#   kable_styling(full_width = T, latex_options = c("striped", "HOLD_poistion", "repeat_header"), position = "center", font_size = 8) %>%
#   kableExtra::footnote(general = "Single nucleotide polymorphisms (SNP) were used as genetic instrumental variables in Mendelian randomization analyses. Each SNP, and associated data, were extracted from published genome wide association studies (Ref). Data is ordered for Exposure, Ref, Chr, BP. Chr = chromosome; BP = bas position; EA = effect allele; NEA = non effect allele; EAF = effect allele frequency; Beta = effect of SNP on GWAS phenotype; SE = standard error of the Beta; P = p-value; Exposure = GWAS phenotype and the exposure used in the MR analysis for that SNP and correspondonding data; Ref = surname of first author of GWAS; Note = is applicable to the BMI Yengo SNPs and indicates whether the SNP was from the conditional joint (COJO) analysis or non-conditional joint (non-COJO) analysis; Clumped = whether that SNP was excluded after clumping.", threeparttable = T, general_title = "", footnote_as_chunk = T)
```

##### Body mass index
Two additional instrument lists were used for BMI. The first came from the same study as used in the main analysis (Yengo et al. (2019)) but did not use the conditional and joint analysis (COJO) GWAS summary statistics. COJO analyses attempt to identify secondary signals (SNPs) at each genome-wide association study (GWAS) identified loci after controlling for linkage disequilibrium (LD). That is, the presence of a secondary signal is conditional on the presence of the first[@Yang2012]. As such, COJO analyses may increase the number of available genetic instruments for MR analysis without introducing additional biases as a result of LD, such as falsely narrow confidence intervals (CIs). Few studies have investigated the use of COJO-GWAS summary statistics in MR analyses. The study by Yew et al., (2020)[@Yew2020], which used the Yengo et al. (2019) COJO-GWAS for BMI, found little evidence for a difference in resulting effect estimates of the association between BMI and atopic dermatitis when using the 941-SNP genetic instrument compared to instruments with different LD (thresholds included r^2^ 0.001, 0.001, 0.1, 0.2, and 0.5) pruning thresholds. They did not however compare results to the Yengo et al., (2019) non-COJO-GWAS for BMI. The GWAS for the non-COJO and COJO genetic instruments were the same, and are presented in the main text. Briefly, data were available from 515,509--795,624 individuals of European ancestries from two sources, the Genetic Investigation of Anthropometric Traits (GIANT) consortium[@Locke2015] and UK Biobank. In both sources, BMI was calculated as $\displaystyle \frac{weight\ (kg)}{height\ (m^2)}$. Yengo et al., (2018)[@Yengo2018] performed a fixed effects inverse variance weighted meta-analysis of BMI using GWAS results generated from UK Biobank (N = 456,426) and results from the GIANT consortium, Locke et al., (2015)[@Locke2015] (N = 322,154) using METAL[@Willer2010]. A total of 656 primary associations (i.e., non-COJO SNPs) reaching a genome wide significance threshold of p-value < 5 x 10^-8^ were identified and used for the additional analysis here. \par

The second instrument list came from Locke et al. (2015), in which 322,154 individuals of European ancestries were included in an inverse variance weighted fixed effects meta-analysis. This GWAS was the GIANT component of the Yengo et al., meta-analysis, and is described in detail in the main text of Chapter \@ref(MR). Briefly, a total of 82 GWAS and 43 studies using the Metabochip array were included in the meta-analysis by Locke et al. Individual GWAS were adjusted for age, age-squared, and study specific covariates with residuals inverse rank normally transformed. Imputation was performed using the HapMap phase II Utah residents of Northern and Western European ancestries (CEU) reference panel. Each study used a linear regression model assuming an additive genetic model with quality control following procedures outlined previously[@Winkler2014]. A fixed effect inverse variance weighted meta-analysis was performed using METAL for the 82 GWAS and 43 studies using the Metabochip array separately. The final meta-analysis combined the GWAS meta-analysis and Metabochip meta-analysis results using an inverse variance weighted fixed effects model. A total of 77 loci reaching genome-wide significance (p-value < 5 x 10^-8^) and separated by at least 500 kilobases were identified and used for the addional analysis here. \par

##### Waist hip ratio
In the main MR analysis, summary statistics for waist hip ratio (WHR) from Pulit et al. (2019) were used. Pulit et al., performed a fixed effects inverse variance weighted meta analysis from two sources, UK Biobank and the GIANT consortium. For the UK Biobank GWAS, the second release (June 2017) of UK Biobank data, which did not have corrected imputation at non-human reference consortium (HRC) sites, was used. Briefly, data were available from 485,486–-697,702 individuals of European ancestries from two sources, UK Biobank and the GIANT consortium[@Shungin2015]. In both sources, WHR was calculated as $\displaystyle \frac{waist\ circumference\ (cm)} {hip\ circumference\ (cm)}$.  Pulit et al., (2019)[@Pulit2019] performed a fixed effects inverse variance weighted meta-analysis of WHR using GWAS results generated from UK Biobank (N up to 485,486 (men up to 263,148; women up to 222,338)) and results from the GIANT consortium (N up to 212,248 (men up to 94,434; women up to 118,004))[@Shungin2015] using METAL. The GIANT component of this meta-analysis was used in the additional analysis here. \par

In GIANT, Shungin et al., (2015)[@Shungin2015] performed an inverse variance weighted fixed effects meta-analysis of up to 212,248 individuals of European ancestries. Individual studies recruited participants and undertook sample and SNP quality control. These studies used genome-wide or Metabochip arrays for sequencing. In studies using genome-wide arrays, imputation was performed with CEU haplotypes from HapMap. Metabochip arrays target specific genetic variants and imputation was not performed. For both study types, an additive genetic model was assumed, with each study running a linear regression GWAS. Sex-specific summary statistics were corrected for population structure using the genomic control inflation factor. Prior to meta-analysis of the genome-wide array GWAS and Metabochip array GWAS, SNPs were removed if they had a minor allele count $\leq$ 3, were not in Hardy-Weinberg equilibrium (p-value < 10^-6^), had a call rate < 95% or an imputation quality score < 0.3 for MACH, < 0.4 for IMPUTE, and < 0.8 for PLINK. In step 1, a meta-analysis of each array was performed using a fixed effects inverse variance weighted model and corrected for genomic control to account for structure between cohorts. In step 2, summary statistics from the meta-analysis of genome-wide array and Metabochip array GWAS were meta-analysed using a fixed effects inverse variance weighted model using METAL. In this second step genomic correction was not performed. In total, 210,088 individuals of European ancestries were included. A total of 26 loci reaching genome-wide significance (p-value < 5 x 10^-8^) and separated by at least 500 kilobases were identified and used in additional analysis here. \par

##### Body fat percentage
For BF, two additional instruments were used. The first used summary statistics from the sudy by Lu et al. (2016)[@Lu2016], used in the main analysis, after removal of two SNPs which had been previously associated with 'favourable adiposity'[@Yaghootkar2014; @Yaghootkar2016]. The second came from summary statistics from Hubel et al. (2019)[@Hubel2019] who analysed data from up to 155,961 (female = 70,700; male = 85,261) individuals of European ancestries from UK Biobank. Body composition was assessed using bioelectrical impedance (Tanita BC-418 MA; Tanita Corporation, Arlington Heights, IL); fat mass and fat free mass was calculated from raw impedance data adjusting for age, sex, height and athleticism (variable not described). BF was calculated as $\displaystyle \frac{fat\ mass\ (kg)} {weight\ (kg)}\ *\ 100$. Hubel et al., performed an inverse variance weighted fixed effects meta-analysis of two sex-specific GWAS using METAL. The sex-specific GWAS were performed using a linear regression model assuming an additive genetic model using BGENIE (V1.2). BF residuals were adjusted for age, socioeconomic status (measured by Townsend deprivation index), assessment centre, genotyping batch, smoking status, alcohol consumption, menopause and the first 6 PCs. Genotypes were imputed to the Human Reference Consortium. A total of 76 loci reaching a genome wide significance threshold of p-value < 5 x 10^-8^ and separated by at least 3,000 kilobases were identified and used in additional analysis here. \par

\newpage

#### Outcomes
```{r appendix-MR-table-metabolites, echo=FALSE}
data <- read.table("data/MR/tables/metabolites.txt", header = T, sep = "\t")
data <- data[order(data$class, data$subclass, data$raw.label),]
knitr::kable(data, longtable = T, booktabs = T, format = "latex", escape = T,
    caption = 'Metabolites used in Mendelian randomization analyses and meta-analyses', 
    row.names = F,
    col.names = c("Metabolite","Label", "Class","Subclass", "Derived", "Study")) %>%
  kable_styling(full_width = T, latex_options = c("striped", "HOLD_poistion", "repeat_header"), position = "center", font_size = 8) %>%
  column_spec(column = 1, width = "5em") %>%
  column_spec(column = 2, width = "20em") %>%
  column_spec(column = 3, width = "10em") %>%
  column_spec(column = 4, width = "10em") %>%
  column_spec(column = 5, width = "3em") %>%
  column_spec(column = 6, width = "3em") %>%
  kableExtra::footnote(general = "Table gives the metabolite abbreviation, label with originally measured units, class and subclass the metabolites were grouped in, whether the metabolite is a derived measure such as a ratio, and the study in which they were available. HDL = high density lipoprotein; LDL = low density lipoprotein; IDL = intermediate density lipoprotein; VLDL = very large low density lipoprotein.", threeparttable = T, general_title = "", footnote_as_chunk = T) 
```

\newpage


### Results

#### F-statistics

(ref:appendix-MR-figure-fstatistics-bmi-cap) **F-statistics for individual genetic instrumental variables and mean F-statistic for each body mass index instrument used in Mendelian randomization analyses**. Mean F-statistic for each exposure indicated by the pink diamond. The blue line indicates a nominal threshold of 10. Exposures used in the main analysis are highlighted with coloured points. The name after each exposure trait represents the first authors last name from the original GWAS publication for which each exposure was obtained. The number following the first authors last name represents the number of SNPs obtained from the original GWAS; clumped refers to this original number of SNPs having been pruned based on an LD R^2^ of $0.001$. BMI = body mass index..

(ref:appendix-MR-figure-fstatistics-bmi-scap) F-statistics for individual genetic instrumental variables and mean F-statistic for each body mass index instrument used in Mendelian randomization analyses

```{r appendix-MR-figure-fstatistics-bmi, echo=FALSE, fig.align='center', out.width='70%', fig.cap='(ref:appendix-MR-figure-fstatistics-bmi-cap)', fig.scap='(ref:appendix-MR-figure-fstatistics-bmi-scap)'}
knitr::include_graphics("../../002_adiposity_metabolites/analysis/bmi_f_statistics.pdf")
```

(ref:appendix-MR-figure-fstatistics-whr-cap) **F-statistics for individual genetic instrumental variables and mean F-statistic for each waist hip ratio instrument used in Mendelian randomization analyses**. Mean F-statistic for each exposure indicated by the pink diamond. The blue line indicates a nominal threshold of 10. Exposures used in the main analysis are highlighted with coloured points. The name after each exposure trait represents the first authors last name from the original GWAS publication for which each exposure was obtained. The number following the first authors last name represents the number of SNPs obtained from the original GWAS; clumped refers to this original number of SNPs having been pruned based on an LD R^2^ of $0.001$. WHR = waist hip ratio.

(ref:appendix-MR-figure-fstatistics-whr-scap) F-statistics for individual genetic instrumental variables and mean F-statistic for each waist hip ratio instrument used in Mendelian randomization analyses

```{r appendix-MR-figure-fstatistics-whr, echo=FALSE, fig.align='center', out.width='70%', fig.cap='(ref:appendix-MR-figure-fstatistics-whr-cap)', fig.scap='(ref:appendix-MR-figure-fstatistics-whr-scap)'}
knitr::include_graphics("../../002_adiposity_metabolites/analysis/whr_f_statistics.pdf")
```

(ref:appendix-MR-figure-fstatistics-bf-cap) **F-statistics for individual genetic instrumental variables and mean F-statistic for each body fat percentage instrument used in Mendelian randomization analyses**. Mean F-statistic for each exposure indicated by the pink diamond. The blue line indicates a nominal threshold of 10. Exposures used in the main analysis are highlighted with coloured points. The name after each exposure trait represents the first authors last name from the original GWAS publication for which each exposure was obtained. The number following the first authors last name represents the number of SNPs obtained from the original GWAS; clumped refers to this original number of SNPs having been pruned based on an LD R^2^ of $0.001$. BF = body fat percentage.

(ref:appendix-MR-figure-fstatistics-bf-scap) F-statistics for individual genetic instrumental variables and mean F-statistic for each body fat percentage instrument used in Mendelian randomization analyses

```{r appendix-MR-figure-fstatistics-bf, echo=FALSE, fig.align='center', out.width='70%', fig.cap='(ref:appendix-MR-figure-fstatistics-bf-cap)', fig.scap='(ref:appendix-MR-figure-fstatistics-bf-scap)'}
knitr::include_graphics("../../002_adiposity_metabolites/analysis/bmi_f_statistics.pdf")
```



#### Sensitivity analyses {#appendix-MR-sensitivity-analysis}
Assumptions of no pleiotropy were explored using MR-Egger[@Bowden2015], weighted median[@Burgess2017a] and weighted mode[@Hartwig2017] based estimators. Globally, sensitivity analysis was visually reflective of the main analysis for each exposure, though with wider CIs. CIs for sensitivity analyses tended to cross the null and were widest for MR Egger, which is in keeping with the lower power afforded with this model. Sensitivity results for WHR appeared to show most consistency with the main analysis for both the Kettunen and INTERVAL data; CIs for weighted median and mode models did not cross the null in a majority of results for subclasses. \par

\newpage

(ref:appendix-MR-figure-kettunen-circosplot-bmi-sensitivity-cap) **Mendelian randomization sensitivity analysis: association between body mass index and metabolites using Kettunen data et al. (2016)[@Kettunen2016]**. Circos plot shows each track as one MR model; the outer track is inverse variance weighted multiplicative random effects (IVW-MRE), the second track is MR Egger, the third track is weighted median, the inner track is weighted mode. Solid points indicate a multiple testing threshold (p-value < `r round(0.05/22,4)`) has been reached. Effect estimates represent a standard deviation (SD) change in the outcome per SD unit increase in the exposure; 95% confidence intervals are shown and may be hidden by the point estimate if very tight. Metabolites are grouped by subclass and arranged alphabetically within each subclass.

(ref:appendix-MR-figure-kettunen-circosplot-bmi-sensitivity-scap) Mendelian randomization sensitivity analysis: association between body mass index and metabolites using Kettunen data et al. (2016)[@Kettunen2016]

```{r appendix-MR-figure-kettunen-circosplot-bmi-sensitivity, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-kettunen-circosplot-bmi-sensitivity-cap)', fig.scap='(ref:appendix-MR-figure-kettunen-circosplot-bmi-sensitivity-scap)'}
knitr::include_graphics("data/MR/figures/circosplot_kettunen_sensitivity_analysis_BMI.pdf")
```

\newpage

(ref:appendix-MR-figure-interval-circosplot-bmi-sensitivity-cap) **Mendelian randomization sensitivity analysis: association between body mass index and metabolites using INTERVAL data**. Circos plot shows each track as one MR model; the outer track is inverse variance weighted multiplicative random effects (IVW-MRE), the second track is MR Egger, the third track is Weighted median, the inner track is Weighted mode. Solid points indicate a multiple testing threshold (p-value < `r round(0.05/28,4)`) has been reached. Effect estimates represent a standard deviation (SD) change in the outcome per SD unit increase in the exposure; 95% confidence intervals are shown and may be hidden by the point estimate if very tight. Metabolites are grouped by subclass and arranged alphabetically within each subclass.

(ref:appendix-MR-figure-interval-circosplot-bmi-sensitivity-scap) Mendelian randomization sensitivity analysis: association between body mass index and metabolites using INTERVAL data

```{r appendix-MR-figure-interval-circosplot-bmi-sensitivity, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-interval-circosplot-bmi-sensitivity-cap)', fig.scap='(ref:appendix-MR-figure-interval-circosplot-bmi-sensitivity-scap)'}
knitr::include_graphics("data/MR/figures/circosplot_interval_sensitivity_analysis_BMI.pdf")
```

\newpage

(ref:appendix-MR-figure-kettunen-circosplot-whr-sensitivity-cap) **Mendelian randomization sensitivity analysis: association between waist hip ratio and metabolites using Kettunen data et al. (2016)[@Kettunen2016]**. Circos plot shows each track as one MR model; the outer track is inverse variance weighted multiplicative random effects (IVW-MRE), the second track is MR Egger, the third track is weighted median, the inner track is weighted mode. Solid points indicate a multiple testing threshold (p-value < `r round(0.05/22,4)`) has been reached. Effect estimates represent a standard deviation (SD) change in the outcome per SD unit increase in the exposure; 95% confidence intervals are shown and may be hidden by the point estimate if very tight. Metabolites are grouped by subclass and arranged alphabetically within each subclass.

(ref:appendix-MR-figure-kettunen-circosplot-whr-sensitivity-scap) Mendelian randomization sensitivity analysis: association between waist hip ratio and metabolites using Kettunen data et al. (2016)[@Kettunen2016]

```{r appendix-MR-figure-kettunen-circosplot-whr-sensitivity, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-kettunen-circosplot-whr-sensitivity-cap)', fig.scap='(ref:appendix-MR-figure-kettunen-circosplot-whr-sensitivity-scap)'}
knitr::include_graphics("data/MR/figures/circosplot_kettunen_sensitivity_analysis_BMI.pdf")
```

\newpage

(ref:appendix-MR-figure-interval-circosplot-whr-sensitivity-cap) **Mendelian randomization sensitivity analysis: association between waist hip ratio and metabolites using INTERVAL data**. Circos plot shows each track as one MR model; the outer track is inverse variance weighted multiplicative random effects (IVW-MRE), the second track is MR Egger, the third track is Weighted median, the inner track is Weighted mode. Solid points indicate a multiple testing threshold (p-value < `r round(0.05/28,4)`) has been reached. Effect estimates represent a standard deviation (SD) change in the outcome per SD unit increase in the exposure; 95% confidence intervals are shown and may be hidden by the point estimate if very tight. Metabolites are grouped by subclass and arranged alphabetically within each subclass.

(ref:appendix-MR-figure-interval-circosplot-whr-sensitivity-scap) Mendelian randomization sensitivity analysis: association between waist hip ratio and metabolites using INTERVAL data

```{r appendix-MR-figure-interval-circosplot-whr-sensitivity, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-interval-circosplot-whr-sensitivity-cap)', fig.scap='(ref:appendix-MR-figure-interval-circosplot-whr-sensitivity-scap)'}
knitr::include_graphics("data/MR/figures/circosplot_interval_sensitivity_analysis_BMI.pdf")
```

\newpage

(ref:appendix-MR-figure-kettunen-circosplot-bf-sensitivity-cap) **Mendelian randomization sensitivity analysis: association between body fat percentage and metabolites using Kettunen data et al. (2016)[@Kettunen2016]**. Circos plot shows each track as one MR model; the outer track is inverse variance weighted multiplicative random effects (IVW-MRE), the second track is MR Egger, the third track is weighted median, the inner track is weighted mode. Solid points indicate a multiple testing threshold (p-value < `r round(0.05/22,4)`) has been reached. Effect estimates represent a standard deviation (SD) change in the outcome per SD unit increase in the exposure; 95% confidence intervals are shown and may be hidden by the point estimate if very tight. Metabolites are grouped by subclass and arranged alphabetically within each subclass.

(ref:appendix-MR-figure-kettunen-circosplot-bf-sensitivity-scap) Mendelian randomization sensitivity analysis: association between body fat percentage and metabolites using Kettunen data et al. (2016)[@Kettunen2016]

```{r appendix-MR-figure-kettunen-circosplot-bf-sensitivity, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-kettunen-circosplot-bf-sensitivity-cap)', fig.scap='(ref:appendix-MR-figure-kettunen-circosplot-bf-sensitivity-scap)'}
knitr::include_graphics("data/MR/figures/circosplot_kettunen_sensitivity_analysis_BMI.pdf")
```

\newpage

(ref:appendix-MR-figure-interval-circosplot-bf-sensitivity-cap) **Mendelian randomization sensitivity analysis: association between body fat percentage and metabolites using INTERVAL data**. Circos plot shows each track as one MR model; the outer track is inverse variance weighted multiplicative random effects (IVW-MRE), the second track is MR Egger, the third track is Weighted median, the inner track is Weighted mode. Solid points indicate a multiple testing threshold (p-value < `r round(0.05/28,4)`) has been reached. Effect estimates represent a standard deviation (SD) change in the outcome per SD unit increase in the exposure; 95% confidence intervals are shown and may be hidden by the point estimate if very tight. Metabolites are grouped by subclass and arranged alphabetically within each subclass.

(ref:appendix-MR-figure-interval-circosplot-bf-sensitivity-scap) Mendelian randomization sensitivity analysis: association between body fat percentage and metabolites using INTERVAL data

```{r appendix-MR-figure-interval-circosplot-bf-sensitivity, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-interval-circosplot-bf-sensitivity-cap)', fig.scap='(ref:appendix-MR-figure-interval-circosplot-bf-sensitivity-scap)'}
knitr::include_graphics("data/MR/figures/circosplot_interval_sensitivity_analysis_BMI.pdf")
```

\newpage

###### Analysis using Kettunen data
In single-SNP MR using Kettunen data, visual inspection of forest plots showed $S$ shaped distributions of effect estimates for all tests (Schematic illustration \@ref(fig:MR-figure-kettunen-singlesnp-representative-figure)). Effect estimates for some SNPs in the single-SNP MR analysis appeared to be outliers. For example, for the analysis of the association between BMI and glycoproteins, rs4673553 showed a disproportionately larger effect estimate of 22 SD units increase per SD higher BMI (standard error = 0.85; p-value = 5.66 x 10^-148^) when compared to other SNPs. \par

To further investigate SNPs with potentially outlying effect estimates, the median effect size across all metabolites for each SNP was investigated. A number of SNPs showed larger median effect sizes across a majority of metabolites. Funnel plots did not however highlight outlying SNPs across all metabolites. Instead, funnel plots were reflective of some SNPs having larger effect estimates more broadly (Representative Figure \@ref(fig:MR-figure-kettunen-funnelplot-BMI-representative-figure)). The low number of SNPs used for BF did not result in meaningfully interpretable funnel plots (Representative Figure \@ref(fig:MR-figure-kettunen-funnelplot-BF-representative-figure)). \par

Although a number of SNPs showed disproportionately larger effect sizes, in leave-one-out analysis, visual inspection of forest plots showed that no single-SNP altered the direction of effect for any metabolite across exposures. For BF, CIs for one or more SNPs crossed the null for every metabolite tested (Representative Figure \@ref(fig:appendix-MR-figure-kettunen-leaveoneout-BF-representative-figure)). This was not the case for BMI and WHR, where, for many metabolites, CIs did not cross the null for any SNPs (Representative Figure \@ref(fig:appendix-MR-figure-kettunen-leaveoneout-BMI-representative-figure)). \par

\newpage

(ref:appendix-MR-figure-kettunen-singlesnp-representative-figure-cap) **Representative figure: single-SNP Mendelian randomization analysis of the association between body mass index and glycerol using Kettunen data**. Forest plot shows the effect estimate and 95% confidence interval for each individual SNPs effect on glycerol. Effect estimates represent a standard deviation (SD) change in the outcome per SD unit increase in the exposure.

(ref:appendix-MR-figure-kettunen-singlesnp-representative-figure-scap) Representative figure: single-SNP Mendelian randomization analysis of the association between body mass index and glycerol using Kettunen data

```{r appendix-MR-figure-kettunen-singlesnp-representative-figure, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-kettunen-singlesnp-representative-figure-cap)', fig.scap='(ref:appendix-MR-figure-kettunen-singlesnp-representative-figure-scap)'}
knitr::include_graphics("data/MR/figures/singlesnp_representative_figure_kettunen.pdf")
```

\newpage

(ref:appendix-MR-figure-kettunen-funnelplot-BMI-representative-figure-cap) **Representative figure: funnel plot of effect estimates of the association between body mass index and glycoproteins using Kettunen data**. Funnel plot shows the results of a single-SNP Mendelian randomization analysis with effect estimate and standard error. Asymmetry in the funnel may indicate the presence of pleiotropy.

(ref:appendix-MR-figure-kettunen-funnelplot-BMI-representative-figure-scap) Representative figure: funnel plot of effect estimates of the association between body mass index and glycoproteins using Kettunen data

```{r appendix-MR-figure-kettunen-funnelplot-BMI-representative-figure, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-kettunen-funnelplot-BMI-representative-figure-cap)', fig.scap='(ref:appendix-MR-figure-kettunen-funnelplot-BMI-representative-figure-scap)'}
knitr::include_graphics("data/MR/figures/funnelplot_BMI_representative_figure_kettunen.pdf")
```

\newpage

(ref:appendix-MR-figure-kettunen-funnelplot-BF-representative-figure-cap) **Representative figure: funnel plot of effect estimates of the association between body mass index and histidine using Kettunen data**. Funnel plot shows the results of a single-SNP MR with effect estimate and standard error. Asymmetry in the funnel may indicate the presence of pleiotropy.

(ref:appendix-MR-figure-kettunen-funnelplot-BF-representative-figure-scap) Representative figure: Funnel plot of effect estimates of the association between body mass index and histidine using Kettunen data

```{r appendix-MR-figure-kettunen-funnelplot-BF-representative-figure, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-kettunen-funnelplot-BF-representative-figure-cap)', fig.scap='(ref:appendix-MR-figure-kettunen-funnelplot-BF-representative-figure-scap)'}
knitr::include_graphics("data/MR/figures/funnelplot_BF_representative_figure_kettunen.pdf")
```

\newpage

(ref:appendix-MR-figure-kettunen-leaveoneout-BF-representative-figure-cap) **Representative figure: leave-one-out Mendelian randomization analysis of the association between body fat percentage and acetoacetate using Kettunen data**. A leave-one-out analysis performs an Mendelian randomization analysis of an exposure and outcome for all SNPs excluding a different SNP each time. Forest plot shows the effect estimate and 95% confidence interval for each SNP exclusion with acetoacetate. Effect estimates represent an SD unit change in the metabolite per SD unit increase in the exposure.

(ref:appendix-MR-figure-kettunen-leaveoneout-BF-representative-figure-scap) Representative figure: leave-one-out Mendelian randomization analysis of the association between body fat percentage and acetoacetate using Kettunen data

```{r appendix-MR-figure-kettunen-leaveoneout-BF-representative-figure, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-kettunen-leaveoneout-BF-representative-figure-cap)', fig.scap='(ref:appendix-MR-figure-kettunen-leaveoneout-BF-representative-figure-scap)'}
knitr::include_graphics("data/MR/figures/leaveoneout_BF_representative_figure_kettunen.pdf")
```

\newpage

(ref:appendix-MR-figure-kettunen-leaveoneout-BMI-representative-figure-cap) **Representative figure: leave-one-out Mendelian randomization analysis of the association body mass index and acetate using Kettunen data**. A leave-one-out analysis performs an Mendelian randomization analysis of an exposure and outcome for all SNPs excluding a different SNP each time. Forest plot shows the effect estimate and 95% confidence interval for each SNP exclusion with acetoacetate. Effect estimates represent an SD unit change in the metabolite per SD unit increase in the exposure.

(ref:appendix-MR-figure-kettunen-leaveoneout-BMI-representative-figure-scap) Representative figure: leave-one-out Mendelian randomization analysis of the association body mass index and acetate using Kettunen data

```{r appendix-MR-figure-kettunen-leaveoneout-BMI-representative-figure, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-kettunen-leaveoneout-BMI-representative-figure-cap)', fig.scap='(ref:appendix-MR-figure-kettunen-leaveoneout-BMI-representative-figure-scap)'}
knitr::include_graphics("data/MR/figures/leaveoneout_BMI_representative_figure_kettunen.pdf")
```

\newpage

###### Analysis using INTERVAL data
Broadly speaking, sensitivity analyses using INTERVAL data were similar to that of the sensitivity analyses using the Kettunen data. In single-SNP MR, visual inspection of forest plots showed $S$ shaped distributions of effect estimates for all tests similar to analyses using the Kettunen data (Representative Figure \@ref(fig:appendix-MR-figure-interval-singlesnp-representative-figure), figure also shows outlier SNP with effect estimate close to -6). As with the Kettunen data, effect estimates for some SNPs in the single-SNP MR analysis were much greater than others. \par

As with the Kettunen analysis, to investigate whether there were many SNPs with potentially outlying effect estimates, the median effect size across all metabolites for each SNP was investigated. A number of SNPs showed larger median effect sizes across a majority of metabolites. Many of these SNPs had similarly large effect sizes across both the Kettunen and INTERVAL datasets. As an example, for BF, more often than not, rs6857 showed a greater effect estimate than the other 6 SNPs and CIs that did not overlap the null or the 6 other SNPs used as instruments. rs6857 was found in both the Kettunen and INTERVAL data to have a disproportionately larger effect estimate than other SNPs. For BMI and WHR, SNPs with disproportionally larger effect sizes tended to have CIs which overlapped other SNPs. The degree of overlap was minimal however and mostly at the tail-end of the CI. \par

Funnel plots did not highlight outlying SNPs, but did reflect some SNPs having larger effect estimates across the board (Representative Figure \@ref(fig:appendix-MR-figure-interval-funnelplot-BMI-representative-figure)). The low number of SNPs used for BF did not result in meaningfully interpretable funnel plots (Representative Figure \@ref(fig:appendix-MR-figure-interval-funnelplot-BF-representative-figure)). In leave-one-out analysis, visual inspection of forest plots showed that no single-SNP altered the direction of effect for any metabolite across exposures. For BF, CIs for one or more SNPs crossed the null for a majority of metabolites tested (Representative Figure \@ref(fig:appendix-MR-figure-interval-leaveoneout-BF-representative-figure)). This was not the case for BMI and WHR, where CIs for many metabolite estimates did not cross the null for any SNPs (Representative Figure \@ref(fig:appendix-MR-figure-interval-leaveoneout-BMI-representative-figure)). \par

\newpage

(ref:appendix-MR-figure-interval-singlesnp-representative-figure-cap) **Representative figure: single-SNP Mendelian rnadomisation analysis of the association between body mass index and glycine using INTERVAL data**. Forest plot shows the effect estimate and 95% confidence interval for each individual SNPs effect on glycerol. Effect estimates represent a standard deviation (SD) change in the outcome per SD unit increase in the exposure.

(ref:appendix-MR-figure-interval-singlesnp-representative-figure-scap) Representative figure: single-SNP Mendelian rnadomisation analysis of the association between body mass index and glycine using INTERVAL data

```{r appendix-MR-figure-interval-singlesnp-representative-figure, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-interval-singlesnp-representative-figure-cap)', fig.scap='(ref:appendix-MR-figure-interval-singlesnp-representative-figure-scap)'}
knitr::include_graphics("data/MR/figures/singlesnp_representative_figure_interval.pdf")
```

\newpage

(ref:appendix-MR-figure-interval-funnelplot-BMI-representative-figure-cap) **Representative figure: funnel plot of effect estimates of the association between body mass index and tyrosine using INTERVAL data**. Funnel plot shows the results of a single-SNP MR with effect estimate and standard error. Asymmetry in the funnel may indicate the presence of pleiotropy.

(ref:appendix-MR-figure-interval-funnelplot-BMI-representative-figure-scap) Representative figure: funnel plot of effect estimates of the association between body mass index and tyrosine using INTERVAL data

```{r appendix-MR-figure-interval-funnelplot-BMI-representative-figure, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-interval-funnelplot-BMI-representative-figure-cap)', fig.scap='(ref:appendix-MR-figure-interval-funnelplot-BMI-representative-figure-scap)'}
knitr::include_graphics("data/MR/figures/funnelplot_BMI_representative_figure_interval.pdf")
```

\newpage

(ref:appendix-MR-figure-interval-leaveoneout-BF-representative-figure-cap) **Representative figure: leave-one-out Mendelian randomization analysis of the association between body fat percentage and acetoacetate using INTERVAL data**. A leave-one-out analysis performs an Mendelian randomization analysis of an exposure and outcome for all SNPs excluding a different SNP each time. Forest plot shows the effect estimate and 95% confidence interval for each SNP exclusion with acetoacetate. Effect estimates represent an SD unit change in the metabolite per SD unit increase in the exposure.

(ref:appendix-MR-figure-interval-leaveoneout-BF-representative-figure-scap) Representative figure: leave-one-out Mendelian randomization analysis of the association between body fat percentage and acetoacetate using INTERVAL data

```{r appendix-MR-figure-interval-leaveoneout-BF-representative-figure, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-interval-leaveoneout-BF-representative-figure-cap)', fig.scap='(ref:appendix-MR-figure-interval-leaveoneout-BF-representative-figure-scap)'}
knitr::include_graphics("data/MR/figures/leaveoneout_BF_representative_figure_interval.pdf")
```

\newpage

(ref:appendix-MR-figure-interval-leaveoneout-BMI-representative-figure-cap) **Representative figure: leave-one-out Mendelian randomization analysis of the association body mass index and acetate using INTERVAL data**. A leave-one-out analysis performs an Mendelian randomization analysis of an exposure and outcome for all SNPs excluding a different SNP each time. Forest plot shows the effect estimate and 95% confidence interval for each SNP exclusion with acetoacetate. Effect estimates represent an SD unit change in the metabolite per SD unit increase in the exposure.

(ref:appendix-MR-figure-interval-leaveoneout-BMI-representative-figure-scap) Representative figure: leave-one-out Mendelian randomization analysis of the association body mass index and acetate using INTERVAL data

```{r appendix-MR-figure-interval-leaveoneout-BMI-representative-figure, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-interval-leaveoneout-BMI-representative-figure-cap)', fig.scap='(ref:appendix-MR-figure-interval-leaveoneout-BMI-representative-figure-scap)'}
knitr::include_graphics("data/MR/figures/leaveoneout_BMI_representative_figure_interval.pdf")
```

\newpage

#### Post-hoc additional analyses
```{r eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
# packages ====
library(tidyr)
library(dplyr)
# data ==== 
data <- read.table("../../002_adiposity_metabolites/analysis/step1/combined/001_combined_mr_results_kettunen.txt", header = T, sep ="\t")
data <- subset(data, method == "Inverse variance weighted (multiplicative random effects)")

## bmi
a <- c("BMI_Yengo_941", "BMI_Yengo_656", "BMI_Locke_77")
data1 <- data[data$exposure %in% a, ]
data1 <- data1[,c(3,4,7)]
data1 <- spread(data1, exposure, b)
data1 <- data1[,c(2:4)]
bmi_cor <- cor(data1, use = "complete.obs", method = "spearman")

## whr
a <- c("WHR_Pulit_316", "WHR_Shungin_26")
data1 <- data[data$exposure %in% a, ]
data1 <- data1[,c(3,4,7)]
data1 <- spread(data1, exposure, b)
data1 <- data1[,c(2:3)]
whr_cor <- cor(data1, use = "complete.obs", method = "spearman")

## bf
a <- c("BF_Lu_7", "BF_Lu_5", "BF_Hubel_76")
data1 <- data[data$exposure %in% a, ]
data1 <- data1[,c(3,4,7)]
data1 <- spread(data1, exposure, b)
data1 <- data1[,c(2:4)]
bf_cor <- cor(data1, use = "complete.obs", method = "spearman")
```

A number of additional SNP lists were used to instrument BMI, WHR, and BF due to potential limitations of instrument lists used in the main analysis. All analysis, including sensitivity and additional sensitvity analyses, were repeated for these additional SNP lists. 

Results from additional instruments for BMI showed broadly larger effect estimates but consistent directions of effect across metabolites. For the BMI SNPs obtained from a non-UK Biobank GWAS, effect estimates had much wider CIs (Figure \@ref(fig:appendix-MR-figure-kettunen-circosplot-bmi-additional) and \@ref(fig:appendix-MR-figure-interval-circosplot-bmi-additional)). Spearman's Rho correlataion of MR results was highest between the two SNP lists from Yengo et al. (2018) (`r round(bmi_cor[[2,3]],2)`) -- correlation between the Locke et al. (2015) SNP list and the COJO SNP list from Yengo et al (2018) (`r round(bmi_cor[[1,3]],2)`) and the non-COJO SNP list from Yengo et al. (2018) (`r round(bmi_cor[[1,2]],2)`) were also high. For WHR, the global pattern of association was similar between both the main and additional exposure (Figure \@ref(fig:appendix-MR-figure-kettunen-circosplot-whr-additional) and \@ref(fig:appendix-MR-figure-interval-circosplot-whr-additional)) with high correlation between MR results (`r round(whr_cor[[1,2]],2)`). Effect estimates were larger for the additional exposure from Shungin et al (2014), for which fewer results reached the multiple testing threshold with wider CIs which crossed the null more often than with the main analysis. \par

For BF, there was considerable similarity between the main analysis and the additional analysis from Lu et al (2016) which did not include two SNPs previously identified as being associated with favourable adiposity (Figure \@ref(fig:appendix-MR-figure-kettunen-circosplot-bf-additional) and \@ref(fig:appendix-MR-figure-interval-circosplot-bmi-additional)). More metabolites reached the multiple testing threshold when using the 5 SNPs from Lu et al (2016) as opposed to the full 7 SNPs, this included associations with apolipoprotein A-1, phenylalanine, tyrosine, glucose, and cholesterol esters in very large HDL. For the additional analysis which used 76 SNPs from Hubel et al (2016), MR results were considerbaly smaller and appeared to show conflicting directions of effect with that of the Lu et al. (2016) SNPs (both using 7 and 5 SNPs). CIs were much tighter and two metabolites (phenylalanine and glycoprotein acetyls) reached the multiple testing threshold. Correlation coefficients between the two Lu et al (2016) SNP lists was high (`r round(bf_cor[[3,2]],2)`), however both the 5 (`r round(bf_cor[[1,3]],2)`) and 7 (`r round(bf_cor[[1,2]],2)`) SNP lists from Lu et al. (2016) showed weaker inverse correaltions with the SNP list from Hubel et al (2016). \par

\newpage

(ref:appendix-MR-figure-kettunen-circosplot-bmi-additional-cap) **Mendelian randomization additional analysis: association between body mass index and metabolites using Kettunen data**. Circos plot shows each track as different body mass index (BMI) instruments: the outer track is the main analysis using Yengo et al. (2018)[@Yengo2018] using 941 COJO SNPs; the middle track is Yengo et al. (2018)[@Yengo2018] using 656 non-COJO SNPs; the inner track is Locke et al. (2015)[@Locke2015] 77 SNPs. Solid points indicate a multiple testing threshold (p-value < `r round(0.05/22,4)`) has been reached. Effect estimates represent a standard deviation (SD) change in the outcome per SD unit increase in the exposure; 95% confidence intervals are shown and may be hidden by the point estimate if very tight. Metabolites are grouped by subclass and arranged alphabetically within each subclass.

(ref:appendix-MR-figure-kettunen-circosplot-bmi-additional-scap) Mendelian randomization additional analysis: association between body mass index and metabolites using Kettunen data

```{r appendix-MR-figure-kettunen-circosplot-bmi-additional, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-kettunen-circosplot-bmi-additional-cap)', fig.scap='(ref:appendix-MR-figure-kettunen-circosplot-bmi-additional-scap)'}
knitr::include_graphics("data/MR/figures/circosplot_kettunen_additional_BMI.pdf")
```

\newpage

(ref:appendix-MR-figure-interval-circosplot-bmi-additional-cap) **Mendelian randomization additional analysis: association between body mass index and metabolites using INTERVAL data**. Circos plot shows each track as different BMI instrumentation: the outer track is the main analysis using Yengo et al. (2018)[@Yengo2018] using 941 COJO SNPs; the middle track is Yengo et al. (2018)[@Yengo2018] using 656 non-COJO SNPs; the inner track is Locke et al. (2015)[@Locke2015] 77 SNPs. Solid points indicate a multiple testing threshold (p-value < `r round(0.05/28,4)`) has been reached. Effect estimates represent a standard deviation (SD) change in the outcome per SD unit increase in the exposure; 95% confidence intervals are shown and may be hidden by the point estimate if very tight. Metabolites are grouped by subclass and arranged alphabetically within each subclass.

(ref:appendix-MR-figure-interval-circosplot-bmi-additional-scap) Mendelian randomization additional analysis: association between body mass index and metabolites using INTERVAL data

```{r appendix-MR-figure-interval-circosplot-bmi-additional, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-interval-circosplot-bmi-additional-cap)', fig.scap='(ref:appendix-MR-figure-interval-circosplot-bmi-additional-scap)'}
knitr::include_graphics("data/MR/figures/circosplot_interval_additional_BMI.pdf")
```

\newpage

(ref:appendix-MR-figure-kettunen-circosplot-whr-additional-cap) **Mendelian randomization additional analysis: association between waist hip ratio and metabolites using Kettunen datas**. Circos plot shows each track as different WHR instrumentation: the outer track is the main analysis using Pulit et al. (2020)[@Pulit2019] using 316 SNPs; the inner track is Shungin et al. (2015)[@Shungin2015] using 26 SNPs. Solid points indicate a multiple testing threshold (p-value < `r round(0.05/28,4)`) has been reached. Effect estimates represent a standard deviation (SD) change in the outcome per SD unit increase in the exposure; 95% confidence intervals are shown and may be hidden by the point estimate if very tight. Metabolites are grouped by subclass and arranged alphabetically within each subclass.

(ref:appendix-MR-figure-kettunen-circosplot-whr-additional-scap) Mendelian randomization additional analysis: association between body mass index and metabolites using Kettunen data

```{r appendix-MR-figure-kettunen-circosplot-whr-additional, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-kettunen-circosplot-whr-additional-cap)', fig.scap='(ref:appendix-MR-figure-kettunen-circosplot-whr-additional-scap)'}
knitr::include_graphics("data/MR/figures/circosplot_kettunen_additional_BMI.pdf")
```

\newpage

(ref:appendix-MR-figure-interval-circosplot-whr-additional-cap) **Mendelian randomization additional analysis: association between waist hip ratio and metabolites using INTERVAL data**. Circos plot shows each track as different WHR instrumentation: the outer track is the main analysis using Pulit et al. (2020)[@Pulit2019] using 316 SNPs; the inner track is Shungin et al. (2015)[@Shungin2015] using 26 SNPs. Solid points indicate a multiple testing threshold (p-value < `r round(0.05/28,4)`) has been reached. Effect estimates represent a standard deviation (SD) change in the outcome per SD unit increase in the exposure; 95% confidence intervals are shown and may be hidden by the point estimate if very tight. Metabolites are grouped by subclass and arranged alphabetically within each subclass.

(ref:appendix-MR-figure-interval-circosplot-whr-additional-scap) Mendelian randomization additional analysis: association between waist hip ratio and metabolites using INTERVAL data

```{r appendix-MR-figure-interval-circosplot-whr-additional, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-interval-circosplot-whr-additional-cap)', fig.scap='(ref:appendix-MR-figure-interval-circosplot-whr-additional-scap)'}
knitr::include_graphics("data/MR/figures/circosplot_interval_additional_BMI.pdf")
```

\newpage

(ref:appendix-MR-figure-kettunen-circosplot-bf-additional-cap) **Mendelian randomization additional analysis: association between body fat percentage index and metabolites using Kettunen data**. Circos plot shows each track as different BF instrumentation: the outer track is the main analysis using Lu et al. (2016)[@Lu2016] using 7 SNPs; the middle track is Lu et al. (2016)[@Lu2016] using 5 SNPs having removed two for being associated with favourable adiposity; the inner track is Hubel et al. (2019)[@Hubel2019] using 76 SNPs. Solid points indicate a multiple testing threshold (`r round(0.05/22,4)`) has been reached. Effect estimates represent a standard deviation (SD) change in the outcome per SD unit increase in the exposure; 95% confidence intervals are shown and may be hidden by the point estimate if very tight. Metabolites are grouped by subclass and arranged alphabetically within each subclass.

(ref:appendix-MR-figure-kettunen-circosplot-bf-additional-scap) Mendelian randomization additional analysis: association between body fat percentage and metabolites using Kettunen data

```{r appendix-MR-figure-kettunen-circosplot-bf-additional, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-kettunen-circosplot-bf-additional-cap)', fig.scap='(ref:appendix-MR-figure-kettunen-circosplot-bf-additional-scap)'}
knitr::include_graphics("data/MR/figures/circosplot_kettunen_additional_BMI.pdf")
```

\newpage

(ref:appendix-MR-figure-interval-circosplot-bf-additional-cap) **Mendelian randomization additional analysis: association between body fat percentage and metabolites using INTERVAL data**. Circos plot shows each track as different BF instrumentation: the outer track is Lu et al. (2016)[@Lu2016] using 7 SNPs; the middle track is Lu et al. (2016)[@Lu2016] using 5 SNPs having removed two for being associated with favourable adiposity; the inner track is Hubel et al. (2019)[@Hubel2019] using 76 SNPs. Solid points indicate a multiple testing threshold (`r round(0.05/28,4)`) has been reached. Effect estimates represent a standard deviation (SD) change in the outcome per SD unit increase in the exposure; 95% confidence intervals are shown and may be hidden by the point estimate if very tight. Metabolites are grouped by subclass and arranged alphabetically within each subclass.

(ref:appendix-MR-figure-interval-circosplot-bf-additional-scap) Mendelian randomization additional analysis: association between body fat percentage and metabolites using INTERVAL data

```{r appendix-MR-figure-interval-circosplot-bf-additional, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-MR-figure-interval-circosplot-bf-additional-cap)', fig.scap='(ref:appendix-MR-figure-interval-circosplot-bf-additional-scap)'}
knitr::include_graphics("data/MR/figures/circosplot_interval_additional_BMI.pdf")
```

\newpage


## Chapter 6: Associations between adiposity associated metabolites and endometrial cancer: Mendelian randomization analysis {#appendix-mediation}

In Chapter \@ref(mediation), the potential intermediary role of metabolites in the association between adiposity and endometrial cancer was investigated. The following sections provide a list of metabolites used in these analyses, 

```{r appendix-mediation-table-metabolites, echo=FALSE}
data <- read.table("data/mediation/tables/metabolites.txt", header = T, sep = "\t")
data <- data[order(data$Class, data$Subclass, data$Metabolite),]
knitr::kable(data, longtable = T, booktabs = T, format = "latex", escape = T,
    caption = 'Metabolites used in Mendelian randomization analyses and meta-analyses', 
    row.names = F,
    col.names = c("Metabolite","Label", "Class","Subclass", "Derived")) %>%
  kable_styling(full_width = T, latex_options = c("striped", "HOLD_poistion", "repeat_header"), position = "center", font_size = 8) %>%
  column_spec(column = 1, width = "5em") %>%
  column_spec(column = 2, width = "20em") %>%
  column_spec(column = 3, width = "12em") %>%
  column_spec(column = 4, width = "15em") %>%
  column_spec(column = 5, width = "3em") %>%
  kableExtra::footnote(general = "Table gives the metabolite abbreviation, label with originally measured units, class and subclass the metabolites were grouped in, and whether the metabolite is a derived measure such as a ratio. HDL = high density lipoprotein; LDL = low density lipoprotein; IDL = intermediate density lipoprotein; VLDL = very large low density lipoprotein.", threeparttable = T, general_title = "", footnote_as_chunk = T) 
```

### Results

#### Two-sample univariable Mendelian randomization: association between adiposity measures and endometrial cancer

(ref:appendix-mediation-figure-forestplot-adiposity-endometrial-cap) **Univariable Mendelian randomization analysis: association between adiposity measures and endometrial cancer**. Odds ratios and associated 95% confidence per SD unit higher body mass index (BMI), waist hip ratio (WHR), or body fat percentage (BF). Models shown are for the main analysis, inverse variance weighted multiplicative random effects, and sensitivity models (weighted median, weighted mode, and MR Egger). Solid points indicate a nominal p-value threshold of 0.05 has been reached. 

(ref:appendix-mediation-figure-forestplot-adiposity-endometrial-scap) Univariable Mendelian randomization analysis: association between adiposity measures and endometrial cancer

```{r appendix-mediation-figure-forestplot-adiposity-endometrial, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-mediation-figure-forestplot-adiposity-endometrial-cap)', fig.scap='(ref:appendix-mediation-figure-forestplot-adiposity-endometrial-scap)'}
knitr::include_graphics("../../007_metabolites_outcomes/analysis/001_adiposity_endometrial/figures/forestplot.pdf")
```

(ref:appendix-mediation-figure-representative-funnel-adiposity-endometrial-cap) **Representative figure: funnel plot of effect estimates of the association between body mass index and endometrial cancer**. Funnel plot shows the results of a single-SNP Mendelian randomisation with effect estimate and standard error. Asymmetry in the funnel may indicate the presence of pleiotropy. Effect estimates (log odds; x-axis) and standard error (y-axis) per SD unit higher body mass index (BMI) are shown. 

(ref:appendix-mediation-figure-representative-funnel-adiposity-endometrial-scap) Representative figure: funnel plot of effect estimates of the association between body mass index and endometrial cancer

```{r appendix-mediation-figure-representative-funnel-adiposity-endometrial, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-mediation-figure-representative-funnel-adiposity-endometrial-cap)', fig.scap='(ref:appendix-mediation-figure-representative-funnel-adiposity-endometrial-scap)'}
knitr::include_graphics("../../007_metabolites_outcomes/analysis/001_adiposity_endometrial/figures/representative_plot_mr_funnel.png")
```

(ref:appendix-mediation-figure-representative-leaveoneout-adiposity-endometrial-cap) **Representative figure: leave-one-out Mendelian randomization analysis of the association between body mass index and endometrial cancer**. A leave-one-out analysis performs an Mendelian randomization analysis of an exposure and outcome for all SNPs excluding a different SNP each time. Forest plot shows the effect estimate and 95% confidence interval for each SNP exclusion with endometrial cancer. Effect estimates (log odds) and associated 95% confidence per SD unit higher body mass index (BMI) are shown. 

(ref:appendix-mediation-figure-representative-leaveoneout-adiposity-endometrial-scap) Representative figure: leave-one-out Mendelian randomization analysis of the association between body mass index and endometrial cancer

```{r appendix-mediation-figure-representative-leaveoneout-adiposity-endometrial, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-mediation-figure-representative-leaveoneout-adiposity-endometrial-cap)', fig.scap='(ref:appendix-mediation-figure-representative-leaveoneout-adiposity-endometrial-scap)'}
knitr::include_graphics("../../007_metabolites_outcomes/analysis/001_adiposity_endometrial/figures/representative_plot_leaveoneout_forest.png")
```

(ref:appendix-mediation-figure-representative-singlesnp-adiposity-endometrial-cap) **Representative figure: single-SNP Mendelian randomization analysis of the association between body mass index and endometrial cancer**. Forest plot shows the effect estimate and 95% confidence interval for each individual SNPs effect on glycerol. Effect estimates (log odds) represent a standard deviation (SD) change in the outcome per SD unit higher body mass index (BMI) are show.

(ref:appendix-mediation-figure-representative-singlesnp-adiposity-endometrial-scap) Representative figure: single-SNP Mendelian randomization analysis of the association between body mass index and endometrial cancer

```{r appendix-mediation-figure-representative-singlesnp-adiposity-endometrial, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-mediation-figure-representative-singlesnp-adiposity-endometrial-cap)', fig.scap='(ref:appendix-mediation-figure-representative-singlesnp-adiposity-endometrial-scap)'}
knitr::include_graphics("../../007_metabolites_outcomes/analysis/001_adiposity_endometrial/figures/representative_plot_singlesnp_forest.png")
```

#### Two-sample univariable Mendelian randomization: association between adiposity measures and metabolites

\newpage
\thispagestyle{empty}
\vspace*{-3cm}

(ref:appendix-mediation-figure-forestplot-adiposity-metabolites-cap) **Univariable Mendelian randomization analysis: association between adiposity measures and metabolites using UK Biobank data**. Effect estimates and associated 95% confidence intervals shown for an standard deviation (SD) change in metabolite per SD unit higher body mass index (BMI), waist hip ratio (WHR), or body fat percentage (BF). Solid points indicate a nominal p-value threshold of 0.05 has been reached. 

(ref:appendix-mediation-figure-forestplot-adiposity-metabolites-scap) Two-sample MR: Effect of adiposity on metabolites

```{r appendix-mediation-figure-forestplot-adiposity-metabolites, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-mediation-figure-forestplot-adiposity-metabolites-cap)', fig.scap='(ref:appendix-mediation-figure-forestplot-adiposity-metabolites-scap)'}
knitr::include_graphics("../../007_metabolites_outcomes/analysis/002_adiposity_metabolite/figures/forestplot.pdf")
```

(ref:appendix-mediation-figure-representative-funnel-adiposity-metabolites-cap) **Representative figure: funnel plot of effect estimates of the association between body mass index and acetate using UK Biobank data**. Funnel plot shows the results of a single-SNP Mendelian randomisation with effect estimate and standard error. Asymmetry in the funnel may indicate the presence of pleiotropy. Effect estimates (log odds; x-axis) and standard error (y-axis) per SD unit higher body mass index (BMI) are shown. 

(ref:appendix-mediation-figure-representative-funnel-adiposity-metabolites-scap) Representative figure: Funnel plot of MR effect estimates from two-sample MR of adiposity on metabolites

```{r appendix-mediation-figure-representative-funnel-adiposity-metabolites, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-mediation-figure-representative-funnel-adiposity-metabolites-cap)', fig.scap='(ref:appendix-mediation-figure-representative-funnel-adiposity-metabolites-scap)'}
knitr::include_graphics("../../007_metabolites_outcomes/analysis/002_adiposity_metabolite/figures/representative_plot_mr_funnel.pdf")
```

(ref:appendix-mediation-figure-representative-leaveoneout-adiposity-metabolites-cap) **Representative figure: leave-one-out Mendelian randomization analysis of the association between body mass index and tyrosine using UK Biobank data**. A leave-one-out analysis performs an Mendelian randomization analysis of an exposure and outcome for all SNPs excluding a different SNP each time. Forest plot shows the effect estimate and 95% confidence interval for each SNP exclusion with tyrosine. Effect estimates and associated 95% confidence per standard deviation (SD) unit change in the exposure per SD unit higher body mass index (BMI) are shown. 

(ref:appendix-mediation-figure-representative-leaveoneout-adiposity-metabolites-scap) Representative figure: leave-one-out Mendelian randomization analysis of the association between body mass index and tyrosine using UK Biobank data

```{r appendix-mediation-figure-representative-leaveoneout-adiposity-metabolites, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-mediation-figure-representative-leaveoneout-adiposity-metabolites-cap)', fig.scap='(ref:appendix-mediation-figure-representative-leaveoneout-adiposity-metabolites-scap)'}
knitr::include_graphics("../../007_metabolites_outcomes/analysis/002_adiposity_metabolite/figures/representative_plot_leaveoneout_forest.pdf")
```

(ref:appendix-mediation-figure-representative-singlesnp-adiposity-metabolites-cap) **Representative figure: single-SNP Mendelian randomization analysis of the association between body mass index and phenylalanine using UK Biobank data**. Forest plot shows the effect estimate and 95% confidence interval for each individual SNPs effect on phenylalanine. Effect estimates represent a standard deviation (SD) change in the outcome per SD unit higher body mass index (BMI).

(ref:appendix-mediation-figure-representative-singlesnp-adiposity-metabolites-scap) Representative figure: single-SNP Mendelian randomization analysis of the association between body mass index and phenylalanine using UK Biobank data

```{r appendix-mediation-figure-representative-singlesnp-adiposity-metabolites, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-mediation-figure-representative-singlesnp-adiposity-metabolites-cap)', fig.scap='(ref:appendix-mediation-figure-representative-singlesnp-adiposity-metabolites-scap)'}
knitr::include_graphics("../../007_metabolites_outcomes/analysis/002_adiposity_metabolite/figures/representative_plot_singlesnp_forest.pdf")
```

#### Two-sample univariable Mendelian randomization: association between adiposity-associated metabolites and endometrial cancer

(ref:appendix-mediation-figure-representative-funnel-metabolites-endometrial-cap) **Representative figure: funnel plot of effect estimates of the association apolipoprotein B and endometrial cancer using UK Biobank data**. Funnel plot shows the results of a single-SNP Mendelian randomisation with effect estimate and standard error. Asymmetry in the funnel may indicate the presence of pleiotropy. Effect estimates (log odds; x-axis) and standard error (y-axis) per SD unit higher body mass index (BMI) are shown. 

(ref:appendix-mediation-figure-representative-funnel-metabolites-endometrial-scap) Representative figure: funnel plot of effect estimates of the association apolipoprotein B and endometrial cancer using UK Biobank data

```{r appendix-mediation-figure-representative-funnel-metabolites-endometrial, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-mediation-figure-representative-funnel-metabolites-endometrial-cap)', fig.scap='(ref:appendix-mediation-figure-representative-funnel-metabolites-endometrial-scap)'}
knitr::include_graphics("../../007_metabolites_outcomes/analysis/003_metabolite_endometrial/figures/representative_plot_mr_funnel.pdf")
```

(ref:appendix-mediation-figure-representative-leaveoneout-metabolites-endometrial-cap) **Representative figure: leave-one-out Mendelian randomization analysis of the association between phenylalanine and endometrial cancer using UK Biobank data**. A leave-one-out analysis performs an Mendelian randomization analysis of an exposure and outcome for all SNPs excluding a different SNP each time. Forest plot shows the effect estimate and 95% confidence interval for each SNP exclusion with endometrial cancer. Effect estimates (log odds) and associated 95% confidence per SD unit phenylalanine are shown. 

(ref:appendix-mediation-figure-representative-leaveoneout-metabolites-endometrial-scap) Representative figure: Leave-one-out plot of MR effect estimates from two-sample MR of metabolites on endometrial cancer

```{r appendix-mediation-figure-representative-leaveoneout-metabolites-endometrial, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-mediation-figure-representative-leaveoneout-metabolites-endometrial-cap)', fig.scap='(ref:appendix-mediation-figure-representative-leaveoneout-metabolites-endometrial-scap)'}
knitr::include_graphics("../../007_metabolites_outcomes/analysis/003_metabolite_endometrial/figures/representative_plot_leaveoneout_forest.pdf")
```

(ref:appendix-mediation-figure-representative-singlesnp-metabolites-endometrial-cap) **Representative figure: single-SNP Mendelian randomization analysis of the association between phenylalanine and endometrial cancer**. Forest plot shows the effect estimate and 95% confidence interval for each individual SNPs effect on glycerol. Effect estimates (log odds) represent a standard deviation (SD) change in the outcome per SD unit higher phenylalanine (BMI).

(ref:appendix-mediation-figure-representative-singlesnp-metabolites-endometrial-scap) Representative figure: single-SNP Mendelian randomization analysis of the association between phenylalanine and endometrial cancer

```{r appendix-mediation-figure-representative-singlesnp-metabolites-endometrial, echo=FALSE, fig.align='center', out.width='100%', fig.cap='(ref:appendix-mediation-figure-representative-singlesnp-metabolites-endometrial-cap)', fig.scap='(ref:appendix-mediation-figure-representative-singlesnp-metabolites-endometrial-scap)'}
knitr::include_graphics("../../007_metabolites_outcomes/analysis/003_metabolite_endometrial/figures/representative_plot_singlesnp_forest.pdf")
```

#### Two-sample multivariable Mendelian randomization: intermediate effects of adiposity-associatied metabolites in the association between adiposity measures and endometrial cancer
```{r appendix-mediation-table-mvmr-bmi, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
data <- read.table("../index/data/mediation/tables/mvmr_bmi.txt", header = T, sep = "\t")
data$p.value <- as.character(data$p.value)

knitr::kable(data, longtable = T, booktabs = T, format = "latex", escape = F,
    caption = 'Two-sample univariable and multivariable Mendelian randomization analysis of the association between body mass index and endometrial cancer.', row.names = F,
    col.names = c("Exposure", "Outcome", "Analysis", "Adjusted","OR",  "Lower CI", "Upper CI", "p-value")) %>%
  kable_styling(latex_options = c("striped", "HOLD_position"), position = "center", font_size = 10) %>%
  kableExtra::footnote(general = "Results for univariable Mendelian randomization (MR) are using an inverse variance weighted multiplicative random effects (IVW-MRE) model; results for multivariable MR are using an IVW model. Effect estimates are given as the odds (OR) of endometrial cancer per SD unit increase in exposure. BMI = body mass index; SVLDLTG = triglycerides in small VLDL; XSVLDLTG = triglycerides in very small VLDL.", threeparttable = T, general_title = "", footnote_as_chunk = T)
```

```{r appendix-mediation-table-mvmr-whr, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
data <- read.table("../index/data/mediation/tables/mvmr_whr.txt", header = T, sep = "\t")
data$p.value <- as.character(data$p.value)

knitr::kable(data, longtable = T, booktabs = T, format = "latex", escape = F,
    caption = 'Two-sample univariable and multivariable Mendelian randomization analysis of the association between waist hip ratio and endometrial cancer.', row.names = F,
    col.names = c("Exposure", "Outcome", "Analysis", "Adjusted","OR",  "Lower CI", "Upper CI", "p-value")) %>%
  kable_styling(latex_options = c("striped", "HOLD_position"), position = "center", font_size = 10) %>%
  kableExtra::footnote(general = "Results for univariable Mendelian randomization (MR) are using an inverse variance weighted multiplicative random effects (IVW-MRE) model; results for multivariable MR are using an IVW model. Effect estimates are given as the odds (OR) of endometrial cancer per SD unit increase in exposure. WHR = waist hip ratio; SVLDLTG = triglycerides in small VLDL; XSVLDLTG = triglycerides in very small VLDL.", threeparttable = T, general_title = "", footnote_as_chunk = T)
```

```{r appendix-mediation-table-mvmr-bf, eval=TRUE, include=TRUE, echo=FALSE, error=FALSE, message=FALSE, warning=FALSE}
data <- read.table("../index/data/mediation/tables/mvmr_bf.txt", header = T, sep = "\t")
data$p.value <- as.character(data$p.value)

knitr::kable(data, longtable = T, booktabs = T, format = "latex", escape = F,
    caption = 'Two-sample univariable and multivariable Mendelian randomization analysis of the association between body fat percentage and endometrial cancer.', row.names = F,
    col.names = c("Exposure", "Outcome", "Analysis", "Adjusted","OR",  "Lower CI", "Upper CI", "p-value")) %>%
  kable_styling(latex_options = c("striped", "HOLD_position"), position = "center", font_size = 10) %>%
  kableExtra::footnote(general = "Results for univariable Mendelian randomization (MR) are using an inverse variance weighted multiplicative random effects (IVW-MRE) model; results for multivariable MR are using an IVW model. Effect estimates are given as the odds (OR) of endometrial cancer per SD unit increase in exposure. BF = body fat percentage; SVLDLTG = triglycerides in small VLDL; XSVLDLTG = triglycerides in very small VLDL.", threeparttable = T, general_title = "", footnote_as_chunk = T)
```

